<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  <head>
    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <title>Charlene BOEHM : DNA Frequencies( vs Pathogens ) -- Articles
      &amp; US Patent</title>
  </head>
  <body>
    <blockquote><br>
      <img alt="" src="0logo.gif" width="124" height="82"><br>
      <a href="../index.htm"><b>rexresearch.com</b></a><b><br>
      </b>
      <hr width="100%" size="2">
      <div align="center"><big><big><b>Charlene BOEHM</b><br>
            <b>DNA Frequencies</b></big></big><br>
      </div>
      <b> </b>
      <hr width="100%" size="2"><b><br>
        <br>
      </b>
      <hr width="100%" size="2"> <b><br>
        <a href="#dnafreq">dnafrequencies.com List</a></b><br>
      <a href="#DNA_Path">Charlene Boehme : </a><a href="#DNA_Path"><b>Frequencies




          of Rife-related Plasma Emission Devices</b></a><br>
      <a href="#US7280874"><b><b>C. Boehme : </b>US7280874</b><b> --
          Methods for determining therapeutic resonant frequencies</b></a><small><br>
      </small><b><a href="#lawsuit">C. Boehm-et-al --&nbsp; Patent
          Infringement Lawsuit</a> <br>
      </b><b><b><a href="#forum">C. Boehm : Prostate Cancer DNA
            Frequencies -- The Rife Forum</a><br>
          <br>
        </b></b>
      <hr width="100%" size="2"><b><a name="dnafreq"></a><br>
      </b><a href="http://www.dnafrequencies.com" "><b>http://www.dnafrequencies.com</b></a><b><br>
      </b>
      <div align="center"><big><b>What we offer</b></big><br>
      </div>
      <br>
      Sets and packages of DNA-related pathogen frequencies are offered
      for sale on a consulting basis for experimental use. Because these
      life forms are sometimes in a state of evolution, or as our
      knowledge of them is widened, the sets of numbers will
      occasionally be expanded or updated.<br>
      <br>
      Many times, it's not initially known what strain of bacteria or
      virus may be involved in a pathogenic infectious situation. For
      that reason, some sets are offered which are designated "general".
      Those sets contain numbers that relate to more than one species or
      strain of an organism.<br>
      <br>
      If you would like to see something of interest to you included on
      these pages that you cannot currently find, please contact us.<br>
      <br>
      <div align="center"><big><b>Frequency sets for common human
            pathogens</b><br>
          <b>Available @ dnafrequencies.com</b></big><br>
      </div>
      <br>
      Bartonella general<br>
      Brucella general<br>
      Childhood viruses -- Includes items from measles (red measles)
      virus, mumps virus, Rubella (German measles) virus, and Chickenpox
      virus (human herpesvirus 3)<br>
      Dental pathogens general -- Includes items from Streptococcus
      mutans, Treponema denticola, Porphyromonas gingivalis, and
      Fusobacterium nucleatum<br>
      Ehrlichia general<br>
      HPV (Human papillomavirus) cervical, general<br>
      HPV (Human papillomavirus) skin, general<br>
      Respiratory viruses general, non-influenza<br>
      Acanthamoeba castellani<br>
      Achromobacter xylosoxidans<br>
      Acinetobacter baumanii general<br>
      Acinetobacter lwoffii<br>
      Actinobacillus actinomycetemcomitans -- Renamed: Aggregatibacter
      actinomycetemcomitans<br>
      Adenovirus type 14 (human)<br>
      Adenovirus type 36 (human)<br>
      Aeromonas hydrophila<br>
      Aeromonas veronii<br>
      Aggregatibacter actinomycetemcomitans -- Previously known as
      Actinobacillus actinomycetemcomitans<br>
      Alcaligenes faecalis<br>
      Anaerococcus prevotii<br>
      Anaplasma phagocytophilum<br>
      Ancylostoma duodenale &amp; Necator americanus <br>
      Angiostrongylus general<br>
      Aspergillus flavus<br>
      Aspergillus fumigatus<br>
      Aspergillus niger<br>
      Atopobium vaginae<br>
      Avian erythroblastosis virus<br>
      Avian leucosis virus<br>
      Avian myelocytomatosis virus<br>
      Babesia bovis<br>
      Babesia microti<br>
      Bacillus anthracis general<br>
      Bacillus cereus<br>
      Bacteroides distasonis<br>
      Bacteroides fragilis<br>
      Bacteroides vulgatus<br>
      Barmah forest virus<br>
      Bartonella bacilliformis<br>
      Bartonella clarridgeiae<br>
      Bartonella grahamii<br>
      Bartonella henselae<br>
      Bartonella quintana<br>
      Bartonella tribocorum<br>
      Bartonella vinsonii<br>
      Bayliascaris procyonis<br>
      BK virus general<br>
      Blastocystis general<br>
      Blastomyces dermatitidis<br>
      Bordetella pertussis &amp; parapertussis<br>
      Borna disease virus<br>
      Borrelia afzelii<br>
      Borrelia bissettii<br>
      Borrelia burgdorferi<br>
      Borrelia duttonii<br>
      Borrelia garinii<br>
      Borrelia hermsii<br>
      Borrelia miyamotoi<br>
      Borrelia parkeri<br>
      Borrelia recurrentis<br>
      Borrelia spielmanii<br>
      Borrelia turicatae<br>
      Borrelia valaisiana<br>
      Brucella abortus<br>
      Brucella canis<br>
      Brucella melitensis<br>
      Brucella suis<br>
      Brugia malayi<br>
      Burkholderia cepacia &amp; cenocepacia<br>
      Burkholderia multivorans<br>
      Campylobacter jejuni<br>
      Campylobacter rectus<br>
      Candida albicans general<br>
      Candida glabrata<br>
      Candida parapsilosis<br>
      Candida tropicalis<br>
      Capnocytophaga ochracea<br>
      Chikungunya virus<br>
      Chlamydia muridarum<br>
      Chlamydia trachomatis general<br>
      Chlamydophila caviae<br>
      Chlamydophila pecorum<br>
      Chlamydophila pneumoniae general<br>
      Chlamydophila psittaci<br>
      &nbsp;Citrobacter freundii<br>
      Citrobacter koseri<br>
      Clonorchis sinensis<br>
      Clostridium botulinum general<br>
      Clostridium difficile<br>
      Clostridium perfringens<br>
      Clostridium ramosum<br>
      Clostridium tetani<br>
      Coccidioides general<br>
      <b>Coronavirus, MERS</b><br>
      Corynebacterium diphtheriae<br>
      Corynebacterium urealyticum<br>
      Coxiella burnetti<br>
      Coxsackievirus A, types 1-12<br>
      Coxsackievirus A, types 13-24<br>
      Coxsackievirus B, type 4<br>
      Coxsackievirus B, types 1-6<br>
      Cryptococcus neoformans<br>
      Cryptosporidium hominis<br>
      Cryptosporidium parvum<br>
      Cytomegalovirus<br>
      Dengue virus, general<br>
      Dialister invisus<br>
      Dictyostelium discoideum<br>
      Dientamoeba fragilis<br>
      Diphyllobothrium latum &amp; nihonkaiense<br>
      Dirofilaria immitis<br>
      Eastern equine encephalitis virus<br>
      <big><b>Ebola virus</b></big><b><br>
      </b>Echinococcus general<br>
      Echovirus type 11<br>
      Echovirus type 7<br>
      Ehrlichia canis<br>
      Ehrlichia chaffeensis<br>
      Eikenella corrodens<br>
      Encephalitozoon cuniculi<br>
      Entamoeba histolytica<br>
      Enterobacter cloacae<br>
      Enterobacter sakazakii<br>
      Enterobius vermicularis<br>
      Enterococcus faecalis<br>
      Enterococcus faecium<br>
      Enterococcus general<br>
      Enterovirus 68 (human)<br>
      Enterovirus 71 (human)<br>
      Epidermophyton floccosum<br>
      Epstein Barr virus<br>
      Escherichia coli (E. coli), general<br>
      Fasciola hepatica<br>
      Finegoldia magna<br>
      Francisella tularensis<br>
      Fusobacterium necrophorum<br>
      Fusobacterium nucleatum<br>
      Gardnerella vaginalis<br>
      Giardia lamblia (aka Giardia intestinalis)<br>
      Gordonia bronchialis<br>
      Granulicatella adiacens<br>
      Haemonchus contortus<br>
      Haemophilus influenzae general<br>
      Hantavirus (aka Hantaan virus)<br>
      Helicobacter hepaticus<br>
      Helicobacter pylori general<br>
      Hendra virus<br>
      Hepatitis A<br>
      Hepatitis B virus general<br>
      Hepatitis C virus general<br>
      Hepatitis G<br>
      Herpes simplex 1 virus<br>
      Herpes simplex 2 virus<br>
      Histoplasma capsulatum<br>
      HIV-1 (Human immunodeficiency virus 1), general<br>
      HTLV-1 (Human T-cell lymphotropic virus type 1, aka Human T-cell
      leukemia virus type 1)<br>
      HTLV-2 (Human T-cell lymphotropic virus type 2)<br>
      Human endogenous retrovirus HRES-1<br>
      Human endogenous retrovirus K<br>
      Human endogenous retrovirus W<br>
      Human foamy virus (aka Human spumaretrovirus)<br>
      Human herpesvirus 1<br>
      Human herpesvirus 2<br>
      Human herpesvirus 3<br>
      Human herpesvirus 4<br>
      Human herpesvirus 5<br>
      Human herpesvirus 6A<br>
      Human herpesvirus 6B<br>
      Human herpesvirus 7<br>
      Human herpesvirus 8<br>
      Human immunodeficiency virus 1 (HIV-1), general<br>
      Human mouse mammary tumor virus<br>
      Human papillomavirus (HPV) type 16<br>
      Human papillomavirus (HPV) type 18<br>
      Human papillomavirus (HPV) type 3<br>
      Human Papillomavirus (HPV) type 31<br>
      Human papillomavirus (HPV) type 5<br>
      Human papillomavirus (HPV) types 1 &amp; 2<br>
      Human papillomavirus (HPV) types 3, 10, &amp; 28<br>
      Human papillomavirus (HPV) types 38, 41, &amp; 49<br>
      Human papillomavirus (HPV) types 4, 27, 29<br>
      Human papillomavirus (HPV) types 6 &amp; 11<br>
      Human parvovirus B19<br>
      Human spumaretrovirus (aka Human foamy virus)<br>
      Human T-cell leukemia virus type 1 (HTLV-1), aka Human T-cell
      lymphotropic virus type 1<br>
      Human T-cell lymphotropic virus type 1 (HTLV-1), aka Human T-cell
      leukemia virus type 1<br>
      Human T-cell lymphotropic virus type 2 (HTLV-2)<br>
      Hymenolepsis diminuta<br>
      Influenza A general<br>
      Influenza B general<br>
      Japanese encephalitis virus<br>
      JC virus<br>
      Kingella kingae<br>
      Klebsiella oxytoca<br>
      Klebsiella planticola<br>
      Klebsiella pneumoniae subspecies pneumoniae<br>
      Legionella pneumophila general<br>
      Leishmania braziliensis<br>
      Leptospira general<br>
      Listeria monocytogenes<br>
      Loa loa<br>
      Louping ill virus<br>
      Malassezia sympodialis<br>
      Measles virus (red measles, rubeola)<br>
      Merkel cell polyomavirus<br>
      MERS coronavirus<br>
      Methanobacter smithii<br>
      Micrococcus luteus<br>
      Microfilaria general<br>
      Microsporum canis<br>
      Mobiluncus curtisii<br>
      Molluscum contagiosum virus<br>
      Moraxella catarrhalis<br>
      Morganella morganii<br>
      Mortierella verticillata<br>
      Mumps virus<br>
      Mycobacterium abscessus<br>
      Mycobacterium avium<br>
      Mycobacterium avium subspecies paratuberculosis<br>
      Mycobacterium bovis<br>
      Mycobacterium general<br>
      Mycobacterium intracellulare<br>
      Mycobacterium kansasii<br>
      Mycobacterium leprae<br>
      Mycobacterium paratuberculosis<br>
      Mycobacterium tuberculosis general<br>
      Mycoplasma arthritidis<br>
      Mycoplasma fermentans<br>
      Mycoplasma general<br>
      Mycoplasma genitalium<br>
      Mycoplasma hominis<br>
      Mycoplasma penetrans<br>
      Mycoplasma pneumoniae<br>
      Necator americanus &amp; Ancylostoma duodenale<br>
      Neisseria gonorrhoeae<br>
      Neisseria mucosa<br>
      Neorickettsia sennetsu<br>
      Nocardia farcinica<br>
      Nocardia general<br>
      Norovirus / Norwalk virus, general<br>
      Novosphingobium aromaticivorans<br>
      Ochrobactrum anthropi<br>
      Onchocerca volvulus <br>
      Opisthorchis felineus - common name: cat liver fluke<br>
      Pantoea ananatis<br>
      Parachlamydia acanthamoeba<br>
      Paragonimus westermani -- &nbsp; Also known as Oriental lung fluke<br>
      Parvovirus B19, human<br>
      Pasteurella multocida<br>
      Pediculus humanus capitus -- Common name: human head louse<br>
      Penicillium marneffei<br>
      Peptostreptococcus anaerobius<br>
      Peptostreptococcus stomatis<br>
      Photorhabdus asymbiotica<br>
      Plasmodium falciparum<br>
      Pneumocystis jiroveci -- formerly called Pneumocystis carinii<br>
      Poliovirus general<br>
      Porphyromonas endodontalis<br>
      Porphyromonas gingivalis<br>
      Powassan virus<br>
      Prevotella intermedia<br>
      Prevotella melaninogenica<br>
      Prevotella nigrescens<br>
      Prevotella tannerae<br>
      Propionibacterium acnes<br>
      Propionibacterium propionicum<br>
      Proteus mirabilis<br>
      Protochlamydia amoebophila<br>
      Providencia alcalifaciens<br>
      Providencia stuartii<br>
      Pseudomonas aeruginosa general<br>
      Pseudomonas fluorescens<br>
      Ralstonia pickettii<br>
      Respiratory syncytial virus general<br>
      Rhizopus oryzae<br>
      Rhodococcus equi -- Formerly called Corynebacterium equi<br>
      Rickettsia africae<br>
      Rickettsia felis -- Causes spotted fever in humans; carried by
      fleas<br>
      Rickettsia prowazekii<br>
      Rickettsia rickettsii<br>
      Rickettsia typhi <br>
      Rift Valley fever virus<br>
      Ross River Virus<br>
      Rothia dentocarios<br>
      Rubella virus (German measles) -- This set is not related to
      measles virus (aka red measles or rubeola)<br>
      Salmonella enterica, serovars typhi &amp; paratyphi <br>
      Salmonella typhimurium -- This species associated with food
      poisoning<br>
      Schistosoma haematobium -- common name: human blood fluke<br>
      Schistosoma japonicum -- common name: Oriental blood fluke<br>
      Schistosoma mansoni -- common name: human blood fluke<br>
      Schistosoma mekongi<br>
      Serratia marcescens<br>
      Shigella general<br>
      Simian parainfluenza virus 5<br>
      Simian virus 40 (SV40)<br>
      Simkania negevensis<br>
      Chlamydia family organism<br>
      Solobacterium moorei<br>
      Spiroplasma mirum<br>
      Sporothrix schenckii<br>
      St. Louis encephalitis virus<br>
      Staphylococcus aureus general<br>
      Staphylococcus epidermidis general<br>
      Staphylococcus general<br>
      Staphylococcus haemolyticus<br>
      Staphylococcus saprophyticus<br>
      Stenotrophomonas maltophilia<br>
      Streptococcus agalactiae general<br>
      Streptococcus anginosus<br>
      Streptococcus constellatus subspecies pharyngitis<br>
      Streptococcus dysgalactiae subspecies equisimilis<br>
      Streptococcus equi subspecies zooepidemicus<br>
      Streptococcus gallolyticus<br>
      Streptococcus gordonii<br>
      Streptococcus intermedius<br>
      Streptococcus mitis<br>
      Streptococcus mutans<br>
      Streptococcus oralis<br>
      Streptococcus parasanguinis<br>
      Streptococcus pneumoniae<br>
      Streptococcus porcinus<br>
      Streptococcus pyogenes<br>
      Streptococcus salivarius<br>
      Streptococcus sanguinis<br>
      Streptococcus suis<br>
      Streptomyces griseus<br>
      Strongyloides stercoralis<br>
      Swine influenza A, 2009<br>
      Taenia general<br>
      Taenia hydatigena<br>
      Taenia solium -- common name: pork tapeworm<br>
      Tannerella forsythensis<br>
      Tick-borne encephalitis virus -- aka FSME virus ("Fruhsommer
      Meningoenzephalitis Virus"), or Early summer meningoencephalitis
      virus<br>
      Toxocara canis &amp; Toxocara felis -- Common names: dog roundworm
      and cat roundworm<br>
      Toxoplasma gondii<br>
      Treponema denticola<br>
      Treponema pallidum<br>
      Treponema vincentii<br>
      Trichinella spiralis<br>
      Trichomonas vaginalis<br>
      Trichophyton general <br>
      Trichuris trichiura -- common name: human whipworm<br>
      Tropheryma whipplei<br>
      Trypanosoma brucei gambiense<br>
      Trypanosoma cruzi<br>
      Ureaplasma parvum<br>
      Ureaplasma urealyticum<br>
      Varicella zoster virus -- aka Human herpesvirus 3<br>
      Venezuelan equine encephalitis virus<br>
      Waddlia chondrophila<br>
      Chlamydia family organism<br>
      West Nile virus<br>
      Western equine encephalitis virus<br>
      Wolbachia endosymbiont of Brugia malayi<br>
      Wuchereria bancrofti<br>
      XMRV (Xenotropic MuLV-related virus)<br>
      Yellow fever virus <br>
      Yersinia enterocolitica<br>
      &nbsp;&nbsp;&nbsp; <br>
      <hr width="100%" size="2"><a name="DNA_Path"></a><br>
      <div align="center"><big><b>DNA Pathogen Frequencies</b></big><br>
      </div>
      <br>
      An abridged version of the longer one published in 1999 by
      Charlene Boehm, the inventor of the DNA frequency method. Some
      text from that original paper has been removed from this version
      because it is outdated, redundant, or not specifically related to
      the DNA frequency method.<br>
      <br>
      <div align="center"><b>A Look At the Frequencies of Rife-related
          Plasma Emission Devices</b><br>
        <br>
        <b>by Charlene Boehm</b><br>
      </div>
      <br>
      This is a story of an exploration with numbers.<br>
      <br>
      The origin of the MORs (Mortal Oscillatory Rates of bacteria and
      viruses), originally discovered by Royal Rife during the first
      half of the twentieth century, has perplexed many people since
      that time. While it is generally acknowledged that some type of
      resonance phenomenon destroyed or debilitated the organisms, it
      has been difficult at best to pinpoint any association of specific
      frequency with what is physically affecting these life forms
      during the time of their debilitation or demise.<br>
      <br>
      What exactly might be the destructive mechanism that is affecting
      each organism? Is it a resonance related to its full size, or
      perhaps that of the nucleus, mitochondria, or capsid? Is it a
      correlation with some type of biochemical resonance? Why does each
      organism seem to need a specific frequency? Could the phenomenon
      be related to its DNA, and if so, what is the resonance
      relationship? These questions and more have kept folks that use or
      explore Rife-related technologies awake into the wee hours of the
      morning on many occasions, and have been the focus of endless
      animated discussions.<br>
      <br>
      This paper will explore some possibilities that might assist in
      shedding light on the resonance relationships.<br>
      <br>
      These mechanisms of action require that some type of physical
      parameter be available that can be converted into frequency. Two
      major physics relationships, that of converting a length into
      frequency (or wavelength, to be more accurate); and that of
      converting mass into frequency, will be looked at in some detail.<br>
      <br>
      While it is acknowledged that some of the concepts presented in
      this paper will be open to dispute, it was felt that the sheer
      number of correlations found with the audio frequencies currently
      being used begged a closer look. For that reason these ideas are
      being offered to the community of serious researchers as a
      springboard for further discussion. The concepts and frequencies
      discussed in this paper, and any materials eventually offered in
      conjunction with this paper, are in no manner intended to suggest
      treatment or cure for any disease or condition. Furthermore, this
      writer cannot assume any responsibility for enhancement of or
      degradation to physical health arising from use of the information
      presented in this paper.<br>
      <br>
      <b>The complete genome.</b><b><br>
      </b><br>
      The developments in the past thirty to forty years in the field of
      genetics and molecular biology has resulted in an explosion of
      information available to anyone that cares to take a look.
      Information is widely available in medical and scientific
      journals, and extensive databases can also be accessed on the
      internet.<br>
      <br>
      The length of any object can be thought of as having a resonant
      frequency by virtue of correlation with a wave-length. For
      instance, a person’s height has its own resonant wavelength and
      resultant frequency. Is it possible that an organism’s entire DNA
      genome could also possess a resonant wavelength and frequency
      related to its total length? Is there a way to calculate the
      entire length of an organism’s DNA genome? Thanks to explicit
      analysis of DNA structure, it is now accurately known how far
      apart the base pair molecules are spaced in that helix. If one
      knows exactly how many base pairs are contained in the complete
      genome, finding the entire length is a simple matter of
      multiplying the number of base pairs times the spacing. [For an
      explanation regarding structure and base pairs of DNA, see L.
      Stryer, Biochemistry, 4th ed., (W.H. Freeman, 1995), p. 75 ff.,
      ISBN 0-7167-2009-4]<br>
      <br>
      As a point of discussion, it must be pointed out that advanced
      x-ray analysis of crystallized DNA has shown that base pair
      spacing is not always consistent. There are some very localized
      areas that contain “squeezing” or “spreading” of the base pairs.
      However, for the purpose of this analysis, the classic
      Watson-Crick model of base pair spacing will be used, which is
      actually an average spacing over the entire length of the DNA
      genome. To use any other model for this discussion would make it
      hopelessly complex for these purposes. For further discussion on
      this subject, see Stryer, p. 788.<br>
      <br>
      The dimensions of the B-helix, which is by far the most common DNA
      form for bacterial and eukaryotic life forms, tells us that:<br>
      <br>
      a.One complete turn of the helix spans a distance of 35.4
      angstroms on its axis.<br>
      <br>
      b.There are 10.4 base pairs in each helical turn. [These
      measurements are given in Stryer, p. 791].<br>
      <br>
      Therefore, the spacing of the individual base pairs on the axis
      would be 35.4 angstroms divided by 10.4, which equals 3.403846
      angstroms. In scientific notation, this can be written as 3.403846
      e-10 meters. The use of meters will now make it possible to
      convert this total length (or wavelength) to frequency.<br>
      <br>
      Looking at an example from a real organism, the Rubella measles
      virus contains 9755 base pairs in its entire DNA genome. (For
      access to base pair information on viruses, go to
      http://www.ncbi.nlm.nih.gov/genomes/static/vis.html).<br>
      <br>
      9755 base pairs x the base pair spacing of 3.403846 e-10 meters =
      3.32045 e-06 meters total length. This is a figure that can be
      used as a possible wavelength for the Rubella viral DNA.<br>
      <br>
      To convert this wavelength to frequency, we turn to the physics
      formula:<br>
      velocity / wavelength = frequency<br>
      <br>
      [See J. Cutnell &amp; K. Johnson, Physics, 2nd ed., (John Wiley
      &amp; Sons, 1992), pg. 698, ISBN 0-471-52919-2, or any good
      physics text].<br>
      <br>
      In this instance we will use the speed of light: 299,792,458
      meters per second as a velocity. (Further comments regarding the
      use of this velocity follow shortly).<br>
      <br>
      Substituting the numbers into the forumla, we get 299,792,458
      meters/second divided by 3.32045 e-06 meters = 9.02866 e+13 hertz.<br>
      <br>
      This would be a possible theoretical resonant frequency for the
      Rubella DNA genome. It is interesting to note that this frequency
      falls at the high end of the infrared section of the
      electromagnetic spectrum (near visible light), and in the general
      area of the spectrum that Royal Rife had under consideration in
      his microscopic work.<br>
      <br>
      To access this frequency in the audio range, an accurate and
      resonant way to accomplish this it is to repeatedly divide the
      frequency by 2. In music, this would be called going to a lower
      octave. Because there is no comparable term to “octave” in
      electromagnetic frequency terminology, the word “octave” will be
      used from this point onward to designate this /2 relationship (or
      x2 for an upper octave). It is a calculation that will be used
      often. Furthermore, dividing a frequency by 2 (i.e., translating
      it into the immediate lower octave) can also be visualized as
      doubling its wavelength in an exact and exceedingly precise
      manner.<br>
      <br>
      Therefore, dividing the original Rubella resonant frequency of
      9.02866 e+13 hz down by many octaves (i.e., doubling the
      wavelength many times) eventually brings us to a frequency at a
      representative octave low in the audio range: 164.23045 hz. This
      could be a possible resonant frequency of the Rubella genome in
      this low audio range.<br>
      <br>
      To “debilitate” this frequency, the following mathematical
      relationship was considered: multiplying this resonant frequency
      by the square root of 2 (1.4142136).<br>
      <br>
      A note is perhaps in order to the general reader: while these
      ideas are being presented in a manner to reach as wide an audience
      as possible, a brief explanation follows (involving the square
      root of 2 relationship) which will get slightly technical. One can
      proceed to the section following the starred line (if desired),
      with no interruption in content.<br>
      <br>
      The general physics formula for the velocity of electromagnetic
      (EM) radiation through any medium equals the inverse of the square
      root of the product of the electrical permittivity and the
      magnetic permeability. The formula reads (in the case of EM
      velocity through a vacuum, and also a good approximation for air):<br>
      <br>
      velocity = 1/v (e0µ0)<br>
      <br>
      where e0 is the electrical permittivity, and µ0 is the magnetic
      permeability.<br>
      The permittivity and permeability are commonly known physics
      constants:<br>
      permittivity e0 = 8.85418782 e-12 farads/meter<br>
      permeability µ0 = 1.2566370614 e-6 henrys/meter<br>
      [D. Lide, ed., Handbook of Chemistry and Physics, 76th ed., (CRC
      Press, 1995), p. 1-1].<br>
      <br>
      Applying these constants in the above formula indeed results in
      the velocity of light through a vacuum: 299,792,458 meters per
      second. Having this velocity figure makes it possible to compute
      electromagnetic frequencies (if the wavelength is also a known
      factor).<br>
      <br>
      However, the next question arises: do electromagnetic waves travel
      through biological tissue at this velocity? Perhaps a new velocity
      can be computed from the formula above, using values for
      permittivity and permeability through biological media.<br>
      <br>
      A representative figure for permittivity (e) through body tissue
      is: 71 e-12 farads/meter. [See E. Hecht, Physics, Vol. 2,
      (Brooks/Cole Publishing Co., 1996), p. 664].<br>
      <br>
      And the permeability (µ) through body tissue is for all practical
      purposes, the same as that of a vacuum: 1.25663706144 e-06
      henrys/meter. [See R. T. Hitchcock &amp; R. Patterson,
      Radio-Frequency and ELF Electromagnetic Energies, A Handbook for
      Professionals, (Van Nostrand Reinhold, 1995), chart on page 27].<br>
      <br>
      Applying these numbers to the above physics formula, the result
      is: velocity = 1 / v [(71 e-12 F/m) x (1.2566370614 e-06 H/m)] =
      105,868,288.9 meters per second as a representative velocity of
      electromagnetic energy through body tissue.<br>
      <br>
      How does this figure compare with that of the speed of light
      through a vacuum?<br>
      Putting these two figures into a ratio gives:<br>
      299,792,458 meters per sec. / 105,868,288.9 meters per sec. =
      2.831749347<br>
      <br>
      If that ratio is divided in half, the result is 1.4158747,
      extremely close to 1.4142136, the value for the square root of 2.
      The next logical step would then be to explore the use of this
      ratio in computing possible frequencies for use in conjunction
      with body tissue (i.e., multiplying a frequency obtained with
      speed-of-light velocity by the square root of two).<br>
      <br>
      The possible low-octave DNA resonant frequency for the Rubella
      virus (using the speed of light velocity) was 164.23045 hz, and
      multiplying that number by v2 = 232.256 hz. (The frequencies that
      are arrived at using the v2 multiplier will henceforth be referred
      to as a “debilitating frequency”).<br>
      <br>
      Now if one uses the representative EM velocity through body tissue
      (105,868,288.9 meters per second), and recalculates the frequency
      associated with the Rubella viral genome wavelength (using the
      formula: velocity / wavelength = frequency), and then divides down
      by octaves as usual, one will come up with nearly the exact same
      frequency as would be arrived at by using the speed of light
      velocity, dividing the high frequency down by octaves, and
      multiplying the low octave by the square root of 2. (105,868,288.9
      meters per sec / 3.32045 E-06 meters = 3.188371724 E+13 hz, which
      divided down by many octaves comes to 231.9845 hz, and is
      extremely close to the 232.256 hz debilitating frequency using the
      speed of light and v2 method).<br>
      <br>
      Now, if we multiply the frequency 232.256 up by just one octave
      (x2), we get 464.5 hz. Interestingly, one of the frequencies used
      for Rubella with the plasma beam devices is 459 hz, only 4.5 hz
      away!<br>
      <br>
      Because the plasma beam devices present the frequencies using a
      square wave (which contains a very strong showing of odd-numbered
      harmonics), it was thought that perhaps some of the early odd
      harmonics (such as 3, 5, 7, 9, 11, etc.) of a currently used
      frequency might also show a mathematical correlation with the DNA
      debilitating frequency suggested above. Such correlations could
      easily be determined using a computer spreadsheet. Here is one
      such example.<br>
      <br>
      One of the frequencies used for “general” measles is 745 hz. Its
      5th harmonic falls at 3725 hz (745 x 5 = 3725), which when divided
      down by 4 octaves (divide by 16) gives 232.8 hz. This is extremely
      close to the above debilitating frequency of 232.256 hz.<br>
      <br>
      One could also look at it in this manner: multiplying the original
      DNA debilitating frequency up by four octaves, 232.256 hz x 16 =
      3716.1 hz. This is close to the fifth harmonic of 745 hz (3725
      hz). So at this juncture we might ask, is the fifth harmonic of
      745 hz hitting an octave of the DNA “debilitating frequency” as
      described above, or at least very close to it?<br>
      <br>
      The Rubella viral organism was used to present the basic concepts
      and procedures being used in this methodology. Another organism
      that gives even more information is Borrelia burgdorferi, which is
      associated with Lyme’s disease.<br>
      <br>
      For convenience however, the formula for finding the
      genome-related debilitating frequency is recapitulated:<br>
      <br>
      [299,792,458 m. per sec / (# of base pairs) x (3.403846154 E-10
      m.)] = frequency<br>
      which, when divided down by many octaves to the low audio range,
      and then multiplied by v2, yields a baseline “debilitating
      frequency”.<br>
      <br>
      The entire genome of Borrelia burgdorferi sains 910,724 base
      pairs. Using the spacing length of 3.403846 e-10 meters, this
      gives us a total genome length of 3.09996 e-04 meters, which
      converts to a frequency (using speed of light as velocity) of
      9.670835558 e+11 hz. Dividing this down by octaves into the low
      audio range gives us 112.58 hz, and then multiplying by v2 yields
      a debilitating frequency of 159.217 hz.<br>
      <br>
      Multiplying this number up by 2 octaves (x4) gives 636.87 hz. One
      of the frequencies currently being used for Lyme’s is 640 hz
      (under “hatchlings/eggs” in the frequency list website given
      above).<br>
      <br>
      Another frequency currently used for this condition is 254 hz, and
      its 5th harmonic is 1270 hz, which divided down by 3 octaves
      (divide by 8) = 158.75 hz, almost exactly falling at the Borrelia
      representative debilitating frequency (abbr. “df”) of 159.217 hz.
      Remember, it is possible that a debilitating frequency may occur
      for an organism at any octave location up and down the entire
      spectrum!<br>
      <br>
      Yet another frequency being used for Lyme’s is 432 hz and its
      upper octave 864 hz. The third harmonic of 432 hz = 1296 hz, which
      divided down by 3 octaves (divide by 8) gives 162 hz, also fairly
      close to the df of 159.217 hz.<br>
      <br>
      Once again these are two more examples of the odd harmonics of
      currently used frequencies correlating with an upper octave of the
      debilitating frequency. It could also help to initially explain
      why more than one audio frequency is effective at targeting an
      organism.<br>
      <br>
      At this point it also must be stated, there will always be
      variation in nature, now and forever. Organisms constantly adapt
      to their surroundings, and this is reflected in (or initiated by)
      changes in their DNA structure. Therefore, one can never assume
      that frequencies computed on the basis of genome wavelength will
      always and forever give accurate, hard and fast results. The
      numbers should be used only to guide us into the ballpark, so to
      speak.<br>
      <br>
      Another aspect of Borrelia burgdorferi that turns out to hold
      considerable interest is that of the plasmids that the organism
      harbors. Plasmids are small, freely-circulating independent pieces
      of usually circular DNA that often (but not always) program
      information relating to the pathogenicity or virulence of the
      organism, and are present in nearly all (if not all) types of
      bacteria. After looking at the base pair information of 11
      Borrelia plasmids thus far, the following frequency correlations
      have shown up (to save time and space, the entire mathematical
      procedure will be shortened):<br>
      <br>
      1.&nbsp; Plasmid cp26 containing 26,498 base pairs. Debilitating
      frequency (df) is at 171 hz, one octave up is at 342 hz, near
      currently used Lyme frequencies of 338 and 344 hz.<br>
      <br>
      2.&nbsp; Plasmid cp9 containing 9386 base pairs, df is at 241.4
      hz, one octave up is 482.8 hz, near currently used frequencies of
      484 and 485 hz.<br>
      <br>
      3.&nbsp; Plasmid lp28-1 containing 26,921 base pairs, df is at
      168.3 hz, one octave up is 336.6 hz, very near currently used
      frequency at 338 hz.<br>
      <br>
      4.&nbsp; Plasmid lp28-2 containing 29,766 base pairs, df is at
      152.2 hz, next 2 octaves up are at 304.5 and 608.9 hz, near the
      currently used frequencies of 306 &amp; 610 hz.<br>
      <br>
      5.&nbsp; Plasmid lp28-3 containing 28,601 base pairs, df is at
      158.4 hz, two octaves up falls at 633.6 hz, near the currently
      used frequency of 630 hz.<br>
      <br>
      6.&nbsp; Plasmid lp28-4 containing 27,323 base pairs, df is at
      165.8 hz, two octaves up falls at 663.4 hz, near the currently
      used frequency of 667 hz.<br>
      <br>
      7.&nbsp; Plasmid lp36 containing 36,849 base pairs, df is at 245.9
      hz, one octave up falls at 491.9 hz, near the currently used
      frequency of 495 hz.<br>
      <br>
      8.&nbsp; Plasmid lp54 containing 53,561 base pairs, df is at 169.2
      hz, one octave up falls at 338.4 hz, almost exactly the same as
      the currently used frequency of 338 hz.<br>
      <br>
      Charlene Boehm<br>
      August 6, 1999<br>
      <br>
      <hr width="100%" size="2"><a name="US7280874"></a><br>
      <a href="http://www.espacenet.com" "><b>http://www.espacenet.com</b></a><br>
      <div align="center"><big><b>US7280874</b><br>
          <b>Methods for determining therapeutic resonant frequencies</b></big><br>
      </div>
      <br>
      Methods are provided for readily and efficiently determining
      resonant frequencies that can be used therapeutically or
      beneficially, for debilitation of specific types of genomic
      materials, including DNA and/or RNA, genes, and gene sections. The
      methods can be used in a variety of circumstances related to
      various human and animal diseases and conditions. Methods allow
      determination of therapeutic resonant frequencies for use in
      various media having different refractivities. Therapeutic or
      beneficial resonance frequencies thus determined are adapted for
      use with currently available frequency-emitting devices by
      shifting resonant frequencies to electromagnetic ranges capable of
      generation by such devices.<br>
      <br>
      <b>CROSS REFERENCE TO RELATED APPLICATIONS</b><b><br>
      </b><br>
      [0001] This application claims priority to applicant's co-pending
      application having U.S. Ser. No. 60/181,460, filed Feb. 10, 2000.<br>
      <br>
      <b>FIELD OF THE INVENTION</b><b><br>
      </b><br>
      [0002] The present invention relates to methods for determining
      resonant frequencies having therapeutic uses in a variety of
      settings. In particular, the present invention provides methods
      for efficiently determining therapeutic resonant frequencies for
      complete genomes or partial genomic materials, for use in various
      media having different refractivities.<br>
      <br>
      <b>BACKGROUND OF THE INVENTION</b><b><br>
      </b><br>
      [0003] Resonant frequency therapy (RFT) is a non-invasive
      treatment that has been reported to offer significant relief to
      sufferers of a variety of ailments and medical conditions. The use
      of RFT for human and animal therapeutic purposes began in the
      early 1900's, and experienced accelerated development through the
      research of Royal Rife and his associates in the 1930's and
      afterward.<br>
      <br>
      [0004] Using new microscope technology he developed, Rife
      discovered that plasma waves could be used to transmit radio and
      audio frequencies, which were tuned to the frequencies of specific
      microorganisms, and that each microorganism responded to its
      unique frequencies. For example, Rife found that staphylococcus,
      streptococcus, microorganisms associated with tuberculosis,
      typhoid, and leprosy, as well as cancer particles, and other
      disease-causing agents succumbed when exposed to certain
      frequencies peculiar to each organism or particle. See, Siedel, R.
      E., and M. E. Winter, The New Microscopes, Smithsonian Annual
      Report 1944, pp. 193-200.<br>
      <br>
      [0005] Using the principles of Rife's discoveries, various
      researchers developed devices for emitting frequencies designed to
      treat a range of diseases and conditions. For example, Dr. Abraham
      Ginsberg used an apparatus which produced intermittent bursts of
      high energy in the short wave spectrum. Ginsberg's modality was
      found to stimulate the reticuloendothelial system without
      undesirably heating tissue. Using his device, Ginsberg reported
      successfully treating patients with various clinical conditions,
      including chronic Staphylococcus infections, acute inflammatory
      middle ear, chronic ulcerative colitis, bronchitis, rheumatoid
      arthritis, gout, flu, and thrombophlebitis, among others. See,
      Cominole, B., Clinical Impressions and Speculations on the Use of
      High-Frequency Pulsed Energy, The Dr. Abraham J. Ginsberg
      Foundation for Medical Research Symposium, Jun. 29, 1959.<br>
      <br>
      [0006] Research utilizing resonant frequencies and therapeutic
      modalities implementing such frequencies have proliferated over
      the past ten years. A recent example of the use of resonant
      frequency therapy is the Christchurch Resonant Frequency Therapy
      Centre in Dunedin, New Zealand. While the Centre emphasizes that
      resonant frequency therapy is not intended to replace treatment
      regimens and medication prescribed by physicians, it does report
      successful treatment of a range of clinical conditions, including
      arthritis, tinnitis, blood pressure, cataracts, headaches,
      shingles, and psoriasis. Arthritis patients report particular
      success with pain reduction and greater mobility. See The
      Christchurch Press, Frequency Therapy Offers Relief, Independent
      Newspapers Limited, Oct. 28, 1999.<br>
      <br>
      [0007] Thus, the use of audio, radio, and light waves to inhibit
      microbial growth and to treat diseases and affected tissue is well
      known in the art. Effective therapeutic resonant frequencies have
      been identified through various means. Trial and error approaches
      with resonant frequencies have been used to obtain therapeutic
      responses. Devices for applying electromagnetic energy to living
      tissue are disclosed, for example, in U.S. Pat. Nos. 3,876,373,
      4,524,079, and 5,091,152. Effective resonant frequencies have also
      been identified through the use of frequency scanning with
      electronic devices capable of detecting a frequency response from
      a bacterial, viral, and/or tissue sample. Such devices for
      detecting frequency response are disclosed, for example, in U.S.
      Pat. Nos. 5,552,274, 5,981,182, and 6,004,257. Thus, there exists
      a need for more efficient and accurate methodology than trial and
      error, to determine therapeutic resonant frequencies for specific
      target materials, such as microorganisms.<br>
      <br>
      [0008] Therapeutic resonant frequencies may be used to inhibit, or
      debilitate, and/or stimulate a biophysical event. The efficacy of
      such frequencies, whether for stimulation or for debilitation,
      depends to some extent on the type of frequency delivery system
      used, including variables such as power levels, waveform, harmonic
      content of the wave, and other factors. Once therapeutic resonant
      frequencies are determined, the user must choose which devices and
      delivery systems are most effectively used in conjunction with
      those frequencies. To increase efficacy, an easier, quicker, and
      more accurate way of determining therapeutic resonant frequencies
      is needed.<br>
      <br>
      [0009] Despite both historical and increasing recent interest in
      use of resonant frequency therapy, mechanism(s) of action
      underlying the use of known therapeutic resonant frequencies is
      not fully understood. While it is recognized that some type of
      resonance phenomenon debilitates or destroys microorganisms, the
      biophysical and/or biochemical mechanism(s) associated with use of
      specific resonant frequencies and that lead to microbial
      inhibition are not completely known.<br>
      <br>
      [0010] Before now, there has never existed a methodology that
      links effective therapeutic resonant frequencies to a biophysical
      or biochemical event, process, or structure. The electronic
      scanning devices and methods currently commercially available
      provide no explanation or insight regarding which physical
      structure or process is influenced by the frequencies used.<br>
      <br>
      [0011] There is a need for methodology to more readily and
      efficiently influence genomic materials, by more precisely and
      efficiently determining therapeutic resonant frequencies that can
      be easily and accurately adjusted to ranges used by currently
      available devices. It is to these perceived needs that the present
      invention is directed.<br>
      <br>
      <b>SUMMARY OF INVENTION</b><b><br>
      </b><br>
      [0012] The present invention provides methods for determining
      resonant frequencies having therapeutic uses in a variety of
      settings. In particular, the present invention provides methods
      for efficiently and accurately determining therapeutic resonant
      frequencies for complete genomes and partial genomic materials,
      for use in various media having different refractivities.<br>
      <br>
      [0013] Methods of the present invention utilize biophysical and
      biochemical properties of genomic materials to determine
      therapeutic resonant frequencies. For example, the length of any
      object can be considered as having a resonant frequency by virtue
      of correlation with a wavelength that manifests itself into a
      surrounding medium. On that basis, the length of biomolecular
      chains of DNA and RNA can be calculated, and thus can provide
      wavelength-matching information unique to a specific strand of
      genomic material.<br>
      <br>
      [0014] DNA or RNA chains are constructed in such a way that
      negatively-charged molecular ions (the PO4 groups) run the entire
      length of the molecule on the outer surface of the chain in a
      helical fashion, causing the molecule to contain a relatively
      large negative charge on its surface. Thus the chain is highly
      electro-sensitive to the influences of resonant oscillating
      electromagnetic fields. Resonance is defined as the increase in
      amplitude of the natural oscillation or frequency of a system,
      when exposed to a periodic force whose frequency is equal or very
      close to the natural frequency of the system. The natural
      oscillation of a system or part of a system is defined as its
      "natural resonant frequency".<br>
      <br>
      [0015] In radio science, the length of an antenna will largely
      determine how effectively the antenna responds to the wavelength
      energy of an incoming transmission. Methods for determining
      therapeutic resonant frequencies of the present invention utilize
      the principle that the length of a DNA or RNA helical chain can be
      electromagnetically resonated in similar fashion.<br>
      <br>
      [0016] Methods of the present invention allow precise correlations
      between resonant frequencies and the length of the genomic
      material under consideration. If a resonant frequency is generated
      in air (or a vacuum) while the target material resides in a
      different medium, in this invention's method a refractive
      adjustment is made to insure that the wavelength traveling from
      the air or vacuum medium transforms to the length of the target
      material in the surrounding medium. By accounting for an
      appropriate electromagnetic refractive index for the surrounding
      medium, such as water or tissue, methods of the present invention
      provide the advantage of determining a resonant frequency that
      would be more closely related to the length of the genomic
      material and its natural resonant frequency, and thus would be
      more appropriate, or therapeutic, for the genomic material in that
      specific medium.<br>
      <br>
      [0017] The natural electromagnetic resonant frequencies for
      genomes fall for the most part in the infrared region of the
      electromagnetic (EM) spectrum. The natural resonant frequencies
      for genes and smaller portions of DNA or RNA appear in the near
      infrared, visible, and near ultraviolet regions of the spectrum.
      For many currently available frequency-emitting devices, the
      natural resonant frequencies such as those associated with genomic
      material are not achievable due to the technical limitations of
      the device. Indeed, particular devices often are capable of
      generating frequencies in only narrow ranges. To overcome such
      limitations, methods of the present invention adjust resonant
      frequencies upward or downward. For example, to determine an
      appropriate lower range frequency in accordance with the present
      invention, the therapeutic resonant frequency is divided by the
      number 2, as many times as necessary, until a frequency in the
      frequency-generating range of a device is reached. The power of 2
      by which a therapeutic resonant frequency is factored will depend
      on the range of the electromagnetic spectrum within which a
      frequency delivery device operates.<br>
      <br>
      [0018] In music, a similar adjustment would be termed moving to a
      higher or lower octave. Moving to a higher octave would in effect
      cut the wavelength in half, while moving to a lower octave would
      double the wavelength. In accordance with methods of the present
      invention, therapeutic resonant frequencies of genomic material
      "shifted by octaves," to a lower octave in the electromagnetic
      spectrum, by dividing the therapeutic resonant frequency by some
      power of the number 2. The lower octave of a therapeutic resonant
      frequency, while having a much longer wavelength, will resonate
      with the first therapeutic resonant frequency, just as musical
      octaves resonate with and amplify each other, but only when the
      octave shift is exact.<br>
      <br>
      [0019] The present invention comprises methods for determining
      therapeutic resonant frequencies of electromagnetic radiation for
      influencing a target genomic material, where the genomic material
      is surrounded by a medium. Embodiments of these methods include
      the following steps: (1) determining a velocity of electromagnetic
      radiation through the medium surrounding the genomic material; (2)
      determining the length of the genomic material; (3) determining a
      first resonant frequency of the genomic material in one
      electromagnetic frequency range by dividing the velocity of the
      electromagnetic radiation through the surrounding medium by the
      length of the genomic material; (4) dividing or multiplying the
      first resonant frequency by a factor of a power of two to obtain
      at least one resonant frequency in another electromagnetic
      frequency range; (5) programming a frequency-emitting device to
      emit at least one resonant frequency in the other electromagnetic
      frequency range selected in step 4; and (6) selectively
      influencing the target genomic material with at least one resonant
      frequency in the selected electromagnetic frequency range, when
      the frequency-emitting device emits at least one resonant
      frequency in the selected electromagnetic frequency range into the
      medium surrounding the target genomic material.<br>
      <br>
      [0020] Methods of the present invention further comprise
      determining the length of the genomic material by determining the
      number of base pairs in the genomic material (in the case of
      single-stranded genomic material, this step would comprise
      determining the number of bases); using the spacing between
      adjacent base pairs or bases; and multiplying the number of base
      pairs or bases in the genomic material by the spacing between
      adjacent base pairs or bases. In a preferred embodiment, the base
      pairs or bases are spaced apart by an average spacing, which is a
      known value, and determining the length of the genomic material
      comprises determining the number of base pairs or bases in the
      genomic material, and then multiplying that number of base pairs
      or bases in the genomic material by the known value for the
      average spacing between base pairs or bases.<br>
      <br>
      [0021] In a typical environment, genomic material exists in
      living, or in-vivo, tissue. In methods of the present invention,
      the velocity of electromagnetic radiation through in-vivo tissue
      is determined by accounting for the electrical permittivity of
      in-vivo tissue in relation to velocity, such that the
      velocity=1/[square root of]([epsilon][mu]), where [epsilon] is the
      electrical permittivity of in-vivo tissue, and [mu] is the
      magnetic permeability of in-vivo tissue. With this measurement of
      in-vivo velocity, a refractive index of electromagnetic radiation
      through in-vivo tissue is determined by dividing the velocity of
      electromagnetic radiation, or the speed of light in a vacuum, by
      the speed of light in in-vivo tissue. Then by dividing a
      therapeutic resonant frequency determined for the genomic material
      in an air medium by the refractive index for in-vivo tissue, a
      therapeutic resonant frequency for the genomic material surrounded
      by in-vivo tissue is determined.<br>
      <br>
      [0022] In other embodiments, methods of the present invention
      include multiplying therapeutic resonant frequencies in a range
      adaptable for use in frequency-emitting devices by a positive
      integer to determine harmonic frequencies; or dividing therapeutic
      resonant frequencies in a range adaptable for use in
      frequency-emitting devices by a positive integer to determine
      subharmonic frequencies. By programming a frequency-emitting
      device to emit the harmonic and subharmonic frequencies, target
      genomic material is selectively influenced with the therapeutic
      resonant frequencies and the harmonic and subharmonic frequencies,
      when the frequency-emitting device emits these frequencies into
      the medium surrounding the target genomic material.<br>
      <br>
      [0023] Features of methods for determining therapeutic resonant
      frequencies of the present invention may be accomplished
      singularly, or in combination, in one or more of the embodiments
      of the present invention. As will be appreciated by those of
      ordinary skill in the art, the present invention has wide utility
      in a number of applications as illustrated by the variety of
      features and advantages discussed below.<br>
      <br>
      [0024] Methods of the present invention provide numerous
      advantages over prior efforts to identify therapeutic resonant
      frequencies. For example, the present invention advantageously
      provides methods for determining resonant frequencies effective
      for stimulation and/or debilitation of specific types of DNA
      and/or RNA genomes, genes and gene sections.<br>
      <br>
      [0025] Another advantage of the methods of the present invention
      is that they provide means for readily and efficiently determining
      therapeutic resonant frequencies using widely available data.<br>
      <br>
      [0026] Another advantage is that the present invention provides
      methods for readily and efficiently predicting resonant
      frequencies that can be used therapeutically or beneficially in a
      variety of circumstances related to treatment of various human and
      animal diseases and conditions.<br>
      <br>
      [0027] Another advantage is that the present invention provides
      methods for readily and efficiently determining therapeutic
      resonant frequencies that take into account an appropriate
      electromagnetic refractive index for a surrounding medium. In so
      doing, the present invention has the advantage of determining a
      more precise therapeutic resonant frequency for the genomic system
      in a particular medium.<br>
      <br>
      [0028] Still another advantage is that the present invention
      provides easier and more efficient methods for determining
      resonant frequencies that significantly enhance the therapeutic
      benefit and cost-effectiveness of currently existing
      electromagnetic, magnetic, plasma, audio, or other
      frequency-emitting devices.<br>
      <br>
      [0029] Another advantage over prior approaches to identifying
      resonant frequencies is that the present invention provides the
      advantage of methods that utilize a simple biophysical model for
      explaining and understanding why specific resonant frequencies are
      effective.<br>
      <br>
      [0030] As will be realized by those of skill in the art, many
      different embodiments of methods for determining therapeutic
      resonant frequencies according to the present invention are
      possible. Additional uses, objects, advantages, and novel features
      of the invention are set forth in the detailed description that
      follows and will become more apparent to those skilled in the art
      upon examination of the following or by practice of the invention.<br>
      <br>
      <b>DETAILED DESCRIPTION OF THE INVENTION</b><b><br>
      </b><br>
      [0031] The present invention comprises methods for determining
      resonant frequencies having therapeutic or beneficial uses in a
      variety of settings. In particular, the present invention includes
      methods for efficiently and accurately determining therapeutic
      resonant frequencies for specific complete genomes, or partial
      genomic materials. Methods of the present invention also comprise
      means for determining a more precise, and thus more therapeutic
      resonant frequency for the genomic system in a particular medium
      by accounting for an appropriate electromagnetic refractive index
      for the surrounding medium. Complete Genome<br>
      <br>
      [0032] As described above, an object has a natural resonant
      frequency by the correlation of the length of the object with a
      wavelength that manifests into its surrounding medium. For
      example, the length of a DNA or RNA chain provides a wavelength
      measurement that can be used to determine a resonant frequency. In
      embodiments of the present invention, the spacing of nucleotide
      base pairs in a DNA double helix is used in the mathematical
      process to determine frequency. The entire length of a piece of
      genomic material, is determined by multiplying the number of base
      pairs or bases in the genomic material times the spacing length
      between base pairs or bases.<br>
      <br>
      [0033] It is known that base pair spacing in strands of DNA is not
      always consistent. Localized areas contain "squeezing" or
      "spreading" of base pairs in various ways. In embodiments of the
      methods of the present invention, the classic Watson-Crick model
      of base pair spacing is used. The Watson-Crick model of base pair
      spacing is an average spacing over the entire length of the DNA
      molecule. Use of an average base pair spacing allows for accuracy
      sufficient to determine therapeutic resonant frequencies in
      accordance with the methods of the present invention.<br>
      <br>
      [0034] The B-helix is the most common in-vivo DNA form in
      bacterial and eukaryotic life forms, and is used herein as
      illustration in the methods of the present invention. In the
      B-helix, one complete turn of the helix spans a distance of 35.4
      angstroms on its axis; and there are 10.4 base pairs in each
      helical turn. Therefore, the spacing of individual base pairs on
      the axis would be 35.4 angstroms per turn divided by 10.4 base
      pairs per turn, which equals 3.403846 angstroms spacing between
      each base pair. In scientific notation using SI units, the base
      pair spacing length is expressed as 3.403846 e-10 meters. This use
      of meters allows conversion of the total length (treated as
      wavelength) into a frequency.<br>
      <br>
      [0035] By way of illustration using a pathogenic microorganism,
      the DNA genome of Borrelia burgdorferi strain B31 contains 910,724
      base pairs. To determine its length, 910,724 base pairs times the
      base pair spacing of 3.403846 e-10 meters=3.09996 e-4 meters total
      length of the genome. As described above, the length of an object
      can represent the object's wavelength; in this case, the length of
      the Borrelia genome represents its wavelength.<br>
      <br>
      [0036] To convert this wavelength to frequency, the following
      common physics relationship is used:<br>
      <br>
      velocity/wavelength=frequency (1)<br>
      <br>
      [0037] If the DNA under consideration was in a medium of air,
      velocity would be the speed of electromagnetic radiation, or
      light, in air. For purposes of comparison, if Borrelia burgdorferi
      was in an air medium, according to methods of the present
      invention, the velocity of electromagnetic radiation through air
      (299,792,458 m/s) would be used in determining a therapeutic
      resonant frequency. Dividing this velocity by the Borrelia
      burgdorferi genome wavelength: (299,792,458 m/s/3.09996 e-4
      meters)=9.6708492 e+11 Hz, the therapeutic resonant frequency for
      Borrelia burgdorferi in an air medium.<br>
      <br>
      [0038] However, genomic material including that of Borrelia
      burgdorferi, generally exists in a medium of living tissue. The
      velocity of electromagnetic radiation through a general in-vivo
      tissue medium is equal to the inverse of the square root of the
      product of the electrical permittivity and the magnetic
      permeability of the medium. The formula for velocity of
      electromagnetic radiation through a typical in-vivo tissue medium
      is given as:<br>
      <br>
      velocity =1/[square root of]([epsilon][mu]) (2)<br>
      <br>
      where [epsilon] is the electrical permittivity and [mu] is the
      magnetic permeability of the medium.<br>
      <br>
      [0039] The magnetic permeability ([mu]) through in-vivo tissue is
      known to be the same as that in air: 1.2566370614 e-6
      henrys/meter. However, electrical permittivity in live body tissue
      is not the same as for air. A representative value for electrical
      permittivity through in-vivo tissue is 71 e-12 farads/meter.
      Applying these figures to formula (2) above, the result is:
      velocity=1/[square root of][(71 e-12 F/m)*(1.2566370614 e&lt;-6
      &gt; H/m)]=105,868,288.9 meters per second, a representative
      velocity of electromagnetic radiation through in-vivo tissue.<br>
      <br>
      [0040] Thus, in this method of the present invention, to obtain an
      in-vivo therapeutic resonant frequency of the Borrelia burgdorferi
      DNA genome having a length of 3.09996 e-4 meters, formula (1)
      above (velocity/wavelength=frequency) is used: 105,868,288.9
      meters per second/3.09996 e-4 meters=3.41515016 e+11 Hz.<br>
      <br>
      [0041] Using the results of the above steps, a general refractive
      index of electromagnetic radiation through in-vivo tissue can be
      determined. A refractive index (n) is given by the ratio of the
      speed of light in a vacuum to the speed of light in the medium
      under consideration. This ratio is stated as:<br>
      <br>
      n=speed of light in a vacuum/speed of light in a medium. (3)<br>
      <br>
      According to the steps given above, a refractive index of
      electromagnetic radiation through in-vivo tissue would be:
      (299,792,458 m/s)/(105,868,288.9 m/s)=2.831749.<br>
      <br>
      [0042] Then, by dividing a therapeutic frequency determined for a
      particular genomic material in an air medium by the refractive
      index for in-vivo tissue, a therapeutic resonant frequency for the
      genomic material in an in-vivo tissue medium is quickly
      determined. Following the example above, dividing the resonant
      frequency of Borrelia in air (9.6708492 e+11 Hz) by the refractive
      index of electromagnetic radiation through in-vivo tissue
      (2.831749), gives the in-vivo resonant frequency for the Borrelia
      burgdorferi genome (3.41515016 e+11 Hz).<br>
      <br>
      [0043] The steps described above for the methods of the present
      invention can be adjusted to correlate with any medium surrounding
      the genomic material under consideration, as long as an accurate
      electromagnetic velocity through the medium is known or can be
      determined.<br>
      <br>
      [0044] The 3.41515016 e+11 Hz in-vivo therapeutic resonant
      frequency determined above for the Borrelia burgdorferi genome
      appears in the infrared range of the electromagnetic spectrum. In
      embodiments of the present invention, methods allow access to
      corresponding resonant frequencies in the lower audio range. For
      example, to determine an accurate resonant frequency in the audio
      range corresponding to first therapeutic resonant frequency, the
      first resonant frequency is divided by the number 2, as many times
      as necessary, to reach a frequency in the audio range. In musical
      terms, as described above, frequencies that are related by a
      factor of 2, or a power thereof, are known as octaves. In the
      example of the in-vivo Borrelia burgdorferi genome, a multi-octave
      shift to audio range can be reached by dividing the first
      therapeutic resonant frequency by 2&lt;29&gt; , which gives a
      corresponding second therapeutic resonant frequency of 636.12 Hz,
      which is in the audio range. This process of dividing (or
      multiplying) any resonant frequency transposes it into a different
      octave by doubling (or halving) its wavelength in an exact and
      precise manner, allowing a resonant correlation with the length
      under consideration in a specific medium. Thus, in the present
      invention, an octave-shifted therapeutic resonant frequency will
      have a precise correlation with the first therapeutic resonant
      frequency.<br>
      <br>
      [0045] In the example above, an in-vivo therapeutic resonant
      frequency of the Borrelia burgdorferi genome is 3.41515016 e+11
      Hz. Corresponding therapeutic useful resonant frequencies in a
      different electromagnetic range, determined by dividing by
      appropriate powers of 2, results in Borrelia burgdorferi in-vivo
      therapeutic resonant frequencies in the audio range at: 636.12 Hz,
      1272.24 Hz, 2544.5 Hz, 5088.9 Hz, etc.<br>
      <br>
      [0046] As another illustration, if Borrelia burgdorferi were in a
      different medium such as water at 40 degrees centigrade, according
      to methods of the present invention, the velocity of EM radiation
      through water at that temperature (225,319,768 m/s) would be used
      in determining therapeutic resonant frequencies. Dividing this
      velocity by the genome length: (225,319,768 m/s)/(3.09996 e-4
      meters)=7.2684734 e+11 Hz, which would be the therapeutic resonant
      frequency of Borrelia burgdorferi DNA in water at 40 degrees
      centigrade.<br>
      <br>
      [0047] To determine corresponding therapeutic resonant frequencies
      in a different electromagnetic frequency range, again in this
      instance the audio range, the resulting resonant frequency above
      is then divided by appropriate powers of 2. This gives therapeutic
      resonant frequencies in the audio range for Borrelia burgdorferi
      in a 40-degree centigrade water medium of: 676.9 Hz, 1353.9 Hz,
      2707.7 Hz, 5415.4 Hz, etc.<br>
      <br>
      [0048] In an alternative embodiment of the present invention,
      methods for determining therapeutic resonant frequencies for
      genomic material under consideration use the numerical constant
      4,526,016.44 as follows: 4,526,016.44 divided by the number of
      base pairs or bases in a chain=frequency. As such, this method
      provides an efficient means for determining frequency by
      ascertaining the number of base pairs or bases in the genomic
      material, and dividing that number into the aforementioned
      constant. For example, if there are 250 base pairs, or bases in a
      DNA chain, 4,526,016.44/250=18,104.07 hertz. For 5,000 base pairs
      or bases in a DNA chain, 4,526,016.44/5,000=905.20 hertz. For
      22,000 base pairs or bases in a DNA chain,
      4,526,016.44/22,000=205.73 hertz.<br>
      <br>
      [0049] As described above, in methods of the present invention,
      therapeutic resonant frequencies are also determined for a
      different electromagnetic range, for example in the audio range,
      by dividing (or multiplying) by appropriate powers of 2. Using the
      example of a 250-base pair DNA chain above, 18,104.07
      Hz/2=9,052.035 Hz. Repeated division of the resulting frequency by
      a factor of 2, such that 9,052.035 Hz/2=4526.017 Hz/2=2263.008
      Hz/2=1131.504 Hz/2=565.752 Hz, quickly determines frequencies in
      the range capable of generation by typical frequency-emitting
      devices. To further shorten the process, dividing 18,104.07 hz by
      32, or 2&lt;5 &gt; (2 to the power of 5), yields a frequency of
      565.752 Hz. Multiplying or dividing by an appropriate factor of 2
      (2, 4, 8, 16, 32, 64, 128, 526, etc.) will accurately convert
      therapeutic resonant frequencies to a desired range for use in
      currently available frequency emission devices. Shifting
      frequencies by factors of 2 produces a frequency event that is an
      octave-related resonant frequency and wavelength.<br>
      <br>
      [0050] As described above, many currently available
      frequency-emitting devices are not capable of producing
      therapeutic resonant frequencies in the infrared range, as that
      determined for the Borrelia burgdorferi genome. To overcome such
      limitations, methods of the present invention adjust resonant
      frequencies downward (or upward) by dividing (or multiplying) by a
      power of 2, until a frequency in the frequency-generating range of
      a device is achieved.<br>
      <br>
      [0051] Certain frequency devices emit not only a basic frequency
      (also referred to as the "fundamental" frequency), but also many
      harmonics of that frequency. A "harmonic" is defined as a positive
      integer multiple of the fundamental frequency. On this basis, in
      methods of the present invention, additional frequencies can be
      determined and programmed into a frequency-emitting device such
      that a harmonic of a frequency corresponding to a first
      therapeutic resonant frequency of a target genomic material, would
      be emitted along with the fundamental frequency. Similar
      additional frequencies can be determined by dividing the
      therapeutic resonant frequency by a positive integer, resulting in
      a "subharmonic" frequency. Subharmonic frequencies corresponding
      to a first therapeutic resonant frequency of a target genomic
      material could also be programmed into a frequency-emitting
      device, and be emitted along with the fundamental frequency. In
      this manner, a group of resonant frequencies corresponding to the
      first therapeutic resonant frequency can be emitted
      simultaneously. As a result, effectiveness of a particular device
      can be enhanced.<br>
      <br>
      [0052] As an example, one in-vivo Borrelia burgdorferi therapeutic
      resonant frequency in an audio-range octave is 636.12 Hz. When
      this therapeutic resonant frequency is divided by the positive
      integer 2, the resulting subharmonic frequency is 318.06 Hz. When
      this subharmonic frequency is programmed into a harmonic-rich
      output device and emitted, the audio-range therapeutic resonant
      frequency 636.12 Hz is emitted simultaneously, increasing the
      likelihood that a therapeutic resonant frequency will impinge a
      target Borrelia burgdorferi genome. In like manner, when dividing
      the audio-range therapeutic resonant frequency 636.12 Hz by the
      positive integer 3, the resulting subharmonic frequency is 212.04
      Hz. A harmonic-rich output device programmed with this subharmonic
      frequency would also emit the 636.12 Hz therapeutic resonant
      frequency, further increasing the likely efficacy of the
      treatment.<br>
      <br>
      [0053] The in-vivo therapeutic resonant frequency determined in
      the audio range for the Borrelia burgdorferi genome (636.12 Hz) is
      very close to a frequency (640 Hz) commonly used for lyme disease,
      which is caused by Borrelia burgdorferi. The accuracy of the
      methods of the present invention may be confirmed by comparing the
      resultant therapeutic resonant frequencies produced by these
      methods, with many known and publicly available therapeutic
      frequencies.<br>
      <br>
      [0054] In another example using a different pathogen, the Rubella
      measles RNA virus contains 9755 bases in its entire genome. (9755
      nucleotides)*(the spacing of 3.403846 e-10 meters)=3.32045 e-6
      meters total length. This length is used as the wavelength for the
      Rubella viral genome. To obtain the in-vivo therapeutic resonant
      frequency of this wavelength, formula (1) above is again used:
      (105,868,288.9 meters per second)/(3.32045 e-6 meters)=3.188371724
      e+13 Hz. A shifting of this near-infrared frequency to audio range
      by dividing by 2&lt;36&gt; , gives a frequency of 463.97 Hz. A
      known therapeutic frequency for the condition of Rubella measles
      is 459 Hz, which is another close match to the therapeutic
      resonant frequency determined by the methods of the present
      invention.<br>
      <br>
      [0055] A number of favorable responses have been reported by
      individuals using previously unknown therapeutic resonant
      frequencies determined by methods of the present invention. For
      example, one person who often experienced severe outbreaks of
      herpes simplex virus used the genome-related therapeutic resonant
      frequencies derived by the methods of the present invention for
      several strains of herpes simplex viruses. This individual
      reported a much faster healing process than what is usually
      experienced. Another example involves a person suffering from
      cancerous cervical warts. After use of previously unknown
      therapeutic resonant frequencies relating to the genome of a
      strain of papilloma virus derived by the methods of the present
      invention, this person reported disappearance of the warts. Still
      another example is a person infected with the chickenpox virus,
      who used a previously unavailable therapeutic resonant frequency
      derived by the methods of the present invention and associated
      with the varicella virus genome. This person reported rapid
      disappearance of blisters and symptoms associated with this
      disease.<br>
      <br>
      [0056] In addition, in-vitro laboratory testing demonstrated that
      exposure of a strain of Escherichia coli to a genome-related
      resonant frequency produced a statistically significant reduction
      in the number of colonies in cultures. Genes and Gene Sections<br>
      <br>
      [0057] Methods of the present invention for determining
      therapeutic resonant frequencies as described above can also be
      applied to sections of DNA and/or RNA, as in genes, for example.
      Using genetic coding information, methods of the present invention
      for determining therapeutic resonant frequencies may also be
      utilized with other sub-components of genomic material, such as
      the coding associated with enzymes, immune factors, oncogenes,
      oncogenic growth factors, and other proteins.<br>
      <br>
      [0058] In embodiments of the present invention, therapeutic
      resonant frequencies are determined using basic information about
      a protein, for example, how many amino acids are in the protein
      chain. Because an amino acid is always coded by three bases in the
      messenger RNA, the number of bases for use in determining resonant
      frequencies can be ascertained by multiplying the number of amino
      acids in a protein chain by 3. For example, if there are 100 amino
      acids in a protein chain, there would be 300 bases in the final
      messenger RNA related to that protein. Thus, according to methods
      of the present invention, a therapeutic resonant frequency can be
      easily determined: 4,526,016.44/300 bases=15,086.72 Hz. Using a
      factor of 2&lt;5 &gt; to determine a corresponding therapeutic
      resonant frequency in a lower octave within the acoustic range as
      described in the methods of the present invention above, the
      resulting therapeutic resonant frequency would be: 15,086.72
      Hz/32=471.46 Hz. which is a frequency that currently available
      frequency-emitting devices are capable of generating.<br>
      <br>
      [0059] As an example, the int-1 mammary oncogene contains 4522
      base pairs of DNA. A therapeutic resonant frequency for this
      oncogene determined by the methods of the present invention above
      is 2001.77 Hz. This therapeutic resonant frequency is very close
      to 2008 Hz, a commonly used cancer-related frequency. Furthermore,
      the messenger RNA associated with the final form of the
      transforming protein of the int-1 mammary oncogene contains 1112
      bases. A therapeutic resonant frequency for this transforming
      protein determined by the methods of the present invention above
      is 2035.08 Hz, which is also in a range of cancer-related
      frequencies currently in use.<br>
      <br>
      [0060] As another example, the messenger RNA for the
      cancer-associated enzyme human tyrosine kinase contains 3151
      bases. A therapeutic resonant frequency for this enzyme's
      messenger RNA, as determined by the methods of the present
      invention above, is 2872.7 Hz. This frequency is very close to the
      cancer-related frequency 2876 Hz, which, along with its related
      octaves, have been used throughout most of the twentieth century
      in association with certain cancer therapy modalities.<br>
      <br>
      [0061] Another example is a precursor gene for Borrelia
      burgdorferi outer surface protein A (ospA), which contains 822
      base pairs. A therapeutic resonant frequency for this gene
      determined by the methods of the present invention above, after
      being factored by powers of 2 to the audible range, is 344.13 Hz.
      A previously known frequency currently used for therapy related to
      lyme disease is 344 Hz, nearly an exact match.<br>
      <br>
      [0062] As can be seen, therapeutic resonant frequencies for genes,
      gene sections, and constituent components of genomic material can
      be determined more readily and efficiently by methods of the
      present invention than for example, by trial and error.<br>
      <br>
      [0063] Favorable responses have been reported from the use of
      previously unavailable therapeutic resonant frequencies determined
      by methods of the present invention, relating to genes, components
      of genes, and/or messenger RNA coding associated with certain
      proteins. For example, an individual diagnosed with lung cancer
      used therapeutic resonant frequencies related to certain growth
      factors and the K-ras oncogene, which is associated with his type
      of tumor. It is reported that this individual experienced
      eradication of lung tumor material. Another example is a student
      experiencing symptoms of both lyme disease and ehrlichiosis, who
      was unable to attend school for a year and half due to the
      severity of symptoms. The student used previously unavailable
      therapeutic resonant frequencies as determined by methods of the
      present invention, for certain membrane and antigenic proteins
      associated with the organism Ehrlichia chaffeensis. Within two
      weeks of beginning therapy with those therapeutic resonant
      frequencies, this student was well enough to return to school.<br>
      <br>
      [0064] While the present invention has been described with
      reference to several specific embodiments, those skilled in the
      art will be able to make various modifications to the described
      embodiments, for instance, by factoring therapeutic resonant
      frequencies to electromagnetic ranges to other than audible
      ranges, and by adjusting for various media, without departing from
      the spirit and scope of the invention. It is therefore to be
      understood that within the scope of the appended claims the
      invention may be practiced other than as specifically described
      herein.<br>
      <br>
      <hr width="100%" size="2"><a name="lawsuit"></a><br>
      <b>http://www.scribd.com/doc/116884099/Boehm-et-al</b><br>
      <br>
      <div align="center"><img alt="" src="law1.jpg" width="374"
          height="398"><br>
      </div>
      <br>
      <hr width="100%" size="2"><a name="forum"></a><br>
      <a
href="http://www.scribd.com/doc/189456824/PROSTATE-CANCER-Charlene-Boehm-DNA-Frequencies-The-Rife-ForumDec%205,%202013"
        "><b>http://www.scribd.com/doc/189456824/PROSTATE-CANCER-Charlene-Boehm-DNA-Frequencies-The-Rife-Forum</b><b><br>
        </b><b> </b><b><br>
        </b><b>Dec 5, 2013</b></a><b><br>
      </b><b><br>
      </b>
      <div align="center"><b>PROSTATE CANCER- Charlene Boehm DNA
          Frequencies - The Rife Forum </b></div>
      <br>
      <br>
      <img alt="" src="scrib1.jpg" width="799" height="422"> <img
        alt="" src="scr2.jpg" width="788" height="452"> <img alt=""
        src="scr3.jpg" width="778" height="415"> <img alt=""
        src="scr4.jpg" width="757" height="290">&nbsp;&nbsp; ...<br>
      <br>
      <hr width="100%" size="2"></blockquote>
    <div align="center"><img alt="" src="0logo.gif" width="124"
        height="82"><br>
      <br>
    </div>
    <div style="text-align: center;"><b>Your Support Maintains this
        Service -- </b><b><br>
      </b> <b><br>
      </b> <b>BUY</b><b><br>
      </b> <b><br>
      </b> <big><b>The</b><b> Rex Research Civilization Kit </b></big><b><br>
      </b> <b><br>
      </b> <b>... It's Your Best Bet &amp; Investment in Sustainable
        Humanity on Earth ... </b><b><br>
      </b><b> Ensure &amp; Enhance Your Survival &amp; Genome
        Transmission ...</b><b> </b><b><br>
      </b> <b>Everything @ rexresearch.com on a </b><b>Thumb Drive !</b><b><br>
      </b> <b><br>
      </b> <b><a href="../order.htm" ">ORDER PAGE</a></b><b><br>
      </b> </div>
    <b> </b><b><br>
    </b><b> </b>
    <hr style="width: 62%; height: 2px;">
  </body>
</html>
