<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  <head>
    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <title>Marcus PETER, et al. : si-RNA vs Cancer -- Articles &amp;
      patent application US2018251762</title>
    <!-- saved from url=http://www.rexresearch.com -->
  </head>
  <body>
    <blockquote><img src="0logo.gif" alt="logo" width="124" height="82"><br>
      <a href="1index.htm"><b>rexresearch.com</b></a><b><br>
      </b>
      <hr width="100%" size="2"><b><br>
      </b>
      <div align="center"><b><font size="+2">Marcus E. PETER</font>, et
          al.</b><br>
        <br>
        <font size="+2"><b>siRNA vs Cancer</b><br>
        </font></div>
      <br>
      <hr width="100%" size="2">
      <div align="center"><br>
        <script type="text/javascript"><!--
google_ad_client = "pub-2924514667432343";
google_ad_width = 728;
google_ad_height = 90;
google_ad_format = "728x90_as";
google_ad_type = "text_image";
//2007-01-05: http://www.rexresearch.com
google_ad_channel = "5682237150";
//--></script>
        <script type="text/javascript" src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script><br>
        <script type="text/javascript"><!--
google_ad_client = "pub-2924514667432343";
google_ad_width = 728;
google_ad_height = 90;
google_ad_format = "728x90_as";
google_ad_type = "text_image";
//2007-01-05: http://www.rexresearch.com
google_ad_channel = "5682237150";
//--></script>
        <script type="text/javascript" src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script><br>
        <script type="text/javascript"><!--
google_ad_client = "pub-2924514667432343";
google_ad_width = 728;
google_ad_height = 90;
google_ad_format = "728x90_as";
google_ad_type = "text_image";
//2007-01-05: http://www.rexresearch.com
google_ad_channel = "5682237150";
//--></script>
        <script type="text/javascript" src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script><br>
      </div>
      <hr width="100%" size="2"><br>
      <a
href="https://www.dailymail.co.uk/health/article-6330575/The-end-chemotherapy-Scientists-discover-cancerous-cells-KILL-CODE.html"
        "><b>https://www.dailymail.co.uk/health/article-6330575/The-end-chemotherapy-Scientists-discover-cancerous-cells-KILL-CODE.html</b><b><br>
        </b></a><b><br>
      </b>
      <div align="center"><b>The end of chemotherapy? Scientists
          discover all cancerous cells have a KILL CODE that can be
          triggered without the gruelling treatment</b><br>
        <br>
        <i>&nbsp;&nbsp;&nbsp; Researchers at Northwestern University
          found that our cells can kill themselves </i><i><br>
        </i><i>&nbsp;&nbsp;&nbsp; Currently, this is triggered by
          disease itself or the artificial use of chemotherapy</i><i><br>
        </i><i>&nbsp;&nbsp;&nbsp; Now, experts believe the 'kill codes'
          could be synthetically duplicated for use</i><i><br>
        </i><br>
        <b>By Peter Lloyd</b></div>
      <br>
      Every cell in the human body contains a 'kill code' which can be
      triggered to cause its own self-destruction.<br>
      <br>
      That's the discovery made by researchers at Northwestern
      University, Illinois, who believe it could be utilized for the
      future fight against cancer.<br>
      <br>
      Specifically, they predict malignant cells could be encouraged to
      'commit suicide' without toxic chemicals pumped into the body -
      which currently turns them on.<br>
      <br>
      And, in the process, it could mean an end to grueling rounds of
      chemotherapy.<br>
      <br>
      Currently, as soon as the cell's inner bodyguards sense it is
      mutating into cancer, it self-activates the kill code to
      extinguish itself.<br>
      <br>
      Embedded in ribonucleic acids, scientists estimate these evolved
      more than 800 million years ago in part to protect the body from
      diseases. <br>
      <br>
      However, while they are already somewhat successful, they can't
      always compete with aggressive tumours. Hence they they are
      artificially prompted with drugs. <br>
      <br>
      But these could be even more powerful if synthetically duplicated
      - not least because they would benefit a patient without the
      side-effects of chemo. <br>
      <br>
      ''Now that we know the kill code, we can trigger the mechanism
      without having to use chemotherapy and without messing with the
      genome,' said lead author Marcus E. Peter, the Tomas D. Spies
      Professor of Cancer Metabolism at Northwestern University Feinberg
      School of Medicine.<br>
      <br>
      'We can use these small RNAs directly, introduce them into cells
      and trigger the kill switch.' <br>
      <br>
      'My goal was not to come up with a new artificial toxic
      substance,' he added.&nbsp; 'I wanted to follow nature's lead. I
      want to utilize a mechanism that nature designed.'<br>
      <br>
      He added: 'Based on what we have learned in these two studies, we
      can now design artificial microRNAs that are much more powerful in
      killing cancer cells than even the ones developed by nature.'<br>
      <br>
      However, he stressed that a potential therapy is many years off. <br>
      <br>
      The paper describing the kill code and identifying how the
      cancer-fighting microRNAs use the code to kill tumor cells was
      published in the Nature Communications, today. <br>
      <br>
      <hr width="100%" size="2"><br>
      <a
href="https://news.northwestern.edu/stories/2017/october/suicide-molecules-kill-any-cancer-cell/"
        "><b>https://news.northwestern.edu/stories/2017/october/suicide-molecules-kill-any-cancer-cell/</b><b><br>
        </b></a><b><br>
      </b>
      <div align="center"><font size="+1"><b>Suicide molecules kill any
            cancer cell</b></font><br>
      </div>
      <br>
      <div align="center"><i>Time traveling hundreds of millions of
          years to unleash one of nature’s original kill switches</i><br>
        <br>
        <b>By Marla Paul</b><br>
      </div>
      <br>
      Small RNA molecules originally developed as a tool to study gene
      function trigger a mechanism hidden in every cell that forces the
      cell to commit suicide, reports a new Northwestern Medicine study,
      the first to identify molecules to trigger a fail-safe mechanism
      that may protect us from cancer.<br>
      <br>
      The mechanism -- RNA suicide molecules -- can potentially be
      developed into a novel form of cancer therapy, the study authors
      said.<br>
      <br>
      Cancer cells treated with the RNA molecules never become resistant
      to them because they simultaneously eliminate multiple genes that
      cancer cells need for survival.<br>
      <br>
      “It’s like committing suicide by stabbing yourself, shooting
      yourself and jumping off a building all at the same time,” said
      Northwestern scientist and lead study author Marcus Peter. “You
      cannot survive.”<br>
      <br>
      The inability of cancer cells to develop resistance to the
      molecules is a first, Peter said.<br>
      <br>
      “This could be a major breakthrough,” noted Peter, the Tom D.
      Spies Professor of Cancer Metabolism at Northwestern University
      Feinberg School of Medicine and a member of the Robert H. Lurie
      Comprehensive Cancer Center of Northwestern University. &nbsp;<br>
      <br>
      Peter and his team discovered sequences in the human genome that
      when converted into small double-stranded RNA molecules trigger
      what they believe to be an ancient kill switch in cells to prevent
      cancer. He has been searching for the phantom molecules with this
      activity for eight years.<br>
      <br>
      “We think this is how multicellular organisms eliminated cancer
      before the development of the adaptive immune system, which is
      about 500 million years old,” he said. “It could be a fail safe
      that forces rogue cells to commit suicide. We believe it is active
      in every cell protecting us from cancer.”<br>
      <br>
      This study, which will be published Oct. 24 in eLife, and two
      other new Northwestern studies in Oncotarget and Cell Cycle by the
      Peter group, describe the discovery of the assassin molecules
      present in multiple human genes and their powerful effect on
      cancer in mice.<br>
      <br>
      Looking back hundreds of millions of years<br>
      <br>
      Why are these molecules so powerful?<br>
      <br>
      “Ever since life became multicellular, which could be more than 2
      billion years ago, it had to deal with preventing or fighting
      cancer,” Peter said. “So nature must have developed a fail safe
      mechanism to prevent cancer or fight it the moment it forms.
      Otherwise, we wouldn’t still be here.”<br>
      <br>
      Thus began his search for natural molecules coded in the genome
      that kill cancer.<br>
      <br>
      “We knew they would be very hard to find,” Peter said. “The kill
      mechanism would only be active in a single cell the moment it
      becomes cancerous. It was a needle in a haystack."<br>
      <br>
      But he found them by testing a class of small RNAs, called small
      interfering (si)RNAs, scientists use to suppress gene activity.
      siRNAs are designed by taking short sequences of the gene to be
      targeted and converting them into double- stranded RNA. These
      siRNAs when introduced into cells suppress the expression of the
      gene they are derived from.<br>
      <br>
      Peter found that a large number of these small RNAs derived from
      certain genes did not, as expected, only suppress the gene they
      were designed against. They also killed all cancer cells. His team
      discovered these special sequences are distributed throughout the
      human genome, embedded in multiple genes as shown in the study in
      Cell Cycle. <br>
      <br>
      When converted to siRNAs, these sequences all act as highly
      trained super assassins. They kill the cells by simultaneously
      eliminating the genes required for cell survival. By taking out
      these survivor genes, the assassin molecule activates multiple
      death cell pathways in parallel.<br>
      <br>
      The small RNA assassin molecules trigger a mechanism Peter calls
      DISE, for Death Induced by Survival gene Elimination.<br>
      <br>
      Activating DISE in organisms with cancer might allow cancer cells
      to be eliminated. Peter’s group has evidence this form of cell
      death preferentially affects cancer cells with little effect on
      normal cells. <br>
      <br>
      To test this in a treatment situation, Peter collaborated with Dr.
      Shad Thaxton, associate professor of urology at Feinberg, to
      deliver the assassin molecules via nanoparticles to mice bearing
      human ovarian cancer. In the treated mice, the treatment strongly
      reduced the tumor growth with no toxicity to the mice, reports the
      study in Oncotarget. Importantly, the tumors did not develop
      resistance to this form of cancer treatment. Peter and Thaxton are
      now refining the treatment to increase its efficacy. <br>
      <br>
      Peter has long been frustrated with the lack of progress in solid
      cancer treatment.<br>
      <br>
      “The problem is cancer cells are so diverse that even though the
      drugs, designed to target single cancer driving genes, often
      initially are effective, they eventually stop working and patients
      succumb to the disease,” Peter said. He thinks a number of cancer
      cell subsets are never really affected by most targeted anticancer
      drugs currently used. <br>
      <br>
      Most of the advanced solid cancers such as brain, lung, pancreatic
      or ovarian cancer have not seen an improvement in survival, Peter
      said.<br>
      <br>
      “If you had an aggressive, metastasizing form of the disease 50
      years ago, you were busted back then and you are still busted
      today,” he said. “Improvements are often due to better detection
      methods and not to better treatments.”<br>
      <br>
      Cancer scientists need to listen to nature more, Peter said.
      Immune therapy has been a success, he noted, because it is aimed
      at activating an anticancer mechanism that evolution developed.
      Unfortunately, few cancers respond to immune therapy and only a
      few patients with these cancers benefit, he said.<br>
      <br>
      "Our research may be tapping into one of nature’s original kill
      switches, and we hope the impact will affect many cancers,” he
      said. “Our findings could be disruptive.”<br>
      <br>
      Northwestern co-authors include first authors William Putzbach,
      Quan Q. Gao, and Monal Patel, and coauthors Ashley Haluck-Kangas,
      Elizabeth T. Bartom, Kwang-Youn A. Kim, Denise M. Scholtens,
      Jonathan C. Zhao and Andrea E. Murmann.<br>
      <br>
      The research is funded by grants T32CA070085, T32CA009560,
      R50CA211271 and R35CA197450 from the National Cancer Institute of
      the National Institutes of Health.&nbsp;&nbsp;&nbsp; &nbsp;<br>
      <br>
      <hr width="100%" size="2"><br>
      <a
href="https://www.newsweek.com/cancer-super-assassin-huntingtons-disease-802967"
        "><b>https://www.newsweek.com/cancer-super-assassin-huntingtons-disease-802967</b></a><b><br>
      </b><b><br>
      </b>
      <div align="center"><b>Cancer 'Super Assassin': Huntington's
          Disease Molecule Can Target and Kill All Tumor Cells,
          Scientists Find</b><br>
        <b>by Katherine Hignett</b><br>
      </div>
      <br>
      Scientists have destroyed numerous types of human cancer cells
      with a toxic molecule characteristic of fatal genetic illness
      Huntington’s disease.<br>
      <br>
      The researchers hailed the molecule—which has killed both human
      and mouse ovarian, breast, prostate, liver, brain, lung, skin and
      colon cancer cell lines in mice—as a “super assassin.”<br>
      <br>
      Their results were published Monday in the journal EMBO Reports.<br>
      <br>
      Huntington’s disease is a progressive illness caused by an excess
      of a specific repeating RNA sequence in the huntington gene, which
      is present in every cell.<br>
      <br>
      The defect causes the death of brain cells, and gradually worsens
      a person’s physical and mental abilities. The disease has no cure.<br>
      <br>
      “We’ve never seen anything this powerful”<br>
      <br>
      Researchers believe that the defect may be even more powerful
      against cancer cells than nerve cells in the brain, and the team
      hopes it can be harnessed to kill cancer cells without causing
      Huntington’s symptoms.<br>
      <br>
      “This molecule is a super assassin against all tumor cells,” said
      senior author Marcus Peter, a professor of cancer metabolism at
      Northwestern University Feinberg School of Medicine, in a press
      statement. “We’ve never seen anything this powerful.”<br>
      <br>
      Peter collaborated with Feinberg colleague Shad Thaxton, associate
      professor of urology, to deliver the molecule in the form of
      nanoparticles to mice with human ovarian cancer. The targeted
      molecule decreased tumor growth with no toxicity to the mice.<br>
      <br>
      <b>"Kill switch"</b><b><br>
      </b><br>
      First author Andrea Murmann, a research assistant professor who
      discovered the cancer-killing mechanism, used the molecule to kill
      numerous other human and mouse cancer cell lines. Building on
      previous research into a cancer “kill switch,” Murmann looked to
      diseases associated with low rates of cancer and a suspected RNA
      link.<br>
      <br>
      “I thought maybe there is a situation where this kill switch is
      overactive in certain people, and where it could cause loss of
      tissues,” Murmann said in the statement. “These patients would not
      only have a disease with an RNA component, but they also had to
      have less cancer.“<br>
      <br>
      There is up to 80 percent less cancer in people with Huntington’s
      disease than the general population.<br>
      <br>
      Murmann recognized similarities between the kill switch and the
      toxic Huntington’s disease RNA sequences.<br>
      <br>
      Based on their results, the team believe the “super assassin”
      molecule could be used to fight cancer in humans. “We believe a
      short-term treatment cancer therapy for a few weeks might be
      possible, where we could treat a patient to kill the cancer cells
      without causing the neurological issues that Huntington’s patients
      suffer from,” Peter said.<br>
      <br>
      The scientists next aim to refine the molecule’s delivery method
      to improve tumor targeting, and to stabilize the nanoparticles for
      storage.<br>
      <br>
      <hr width="100%" size="2"><br>
      <a href="https://www.ncbi.nlm.nih.gov/pubmed/29063830"
        "><b>https://www.ncbi.nlm.nih.gov/pubmed/29063830</b><b><br>
        </b></a><b>Elife. 2017 Oct 24;6. </b><b><br>
      </b><b>pii: e29702. </b><b><br>
      </b><b>doi: 10.7554/eLife.29702.</b><b><br>
      </b>
      <div align="center"><font size="+1"><b>Many si/shRNAs can kill
            cancer cells by targeting multiple survival genes through an
            off-target mechanism.</b></font><br>
        <b>Putzbach W, et al.</b><br>
      </div>
      <b>Abstract</b><b><br>
      </b>Over 80% of multiple-tested siRNAs and shRNAs targeting CD95
      or CD95 ligand (CD95L) induce a form of cell death characterized
      by simultaneous activation of multiple cell death pathways
      preferentially killing transformed and cancer stem cells. We now
      show these si/shRNAs kill cancer cells through canonical RNAi by
      targeting the 3'UTR of critical survival genes in a unique form of
      off-target effect we call DISE (death induced by survival gene
      elimination). Drosha and Dicer-deficient cells, devoid of most
      miRNAs, are hypersensitive to DISE, suggesting cellular miRNAs
      protect cells from this form of cell death. By testing 4666 shRNAs
      derived from the CD95 and CD95L mRNA sequences and an unrelated
      control gene, Venus, we have identified many toxic sequences -
      most of them located in the open reading frame of CD95L. We
      propose that specific toxic RNAi-active sequences present in the
      genome can kill cancer cells.<br>
      <br>
      <hr width="100%" size="2"><br>
      <a href="https://www.nature.com/articles/s41467-018-06526-1#Abs1"
        "><b>https://www.nature.com/articles/s41467-018-06526-1#Abs1</b><b><br>
        </b></a><b>Nature Communicationsvolume 9, Article number: 4504
        (2018) </b><font size="+1"><b><br>
        </b><b><br>
        </b></font>
      <div align="center"><font size="+1"><b>6mer seed toxicity in tumor
            suppressive microRNAs</b><br>
        </font><b>Quan Q. Gao, et al.</b><br>
      </div>
      <b><br>
      </b><b>Abstract</b><b><br>
      </b>Many small-interfering (si)RNAs are toxic to cancer cells
      through a 6mer seed sequence (positions 2–7 of the guide strand).
      Here we performed an siRNA screen with all 4096 6mer seeds
      revealing a preference for guanine in positions 1 and 2 and a high
      overall G or C content in the seed of the most toxic siRNAs for
      four tested human and mouse cell lines. Toxicity of these siRNAs
      stems from targeting survival genes with C-rich 3′UTRs. The master
      tumor suppressor miRNA miR-34a-5p is toxic through such a G-rich
      6mer seed and is upregulated in cells subjected to genotoxic
      stress. An analysis of all mature miRNAs suggests that during
      evolution most miRNAs evolved to avoid guanine at the 5′ end of
      the 6mer seed sequence of the guide strand. In contrast, for
      certain tumor-suppressive miRNAs the guide strand contains a
      G-rich toxic 6mer seed, presumably to eliminate cancer cells.<br>
      <br>
      <b>Introduction</b><b><br>
      </b><br>
      RNA interference (RNAi) is a form of post-transcriptional
      regulation exerted by 19–21 nt long double-stranded RNAs that
      negatively regulate gene expression at the mRNA level. RNAi-active
      guide RNAs can come from endogenous siRNAs and micro(mi)RNAs. For
      an miRNA, the RNAi pathway begins in the nucleus with
      transcription of a primary miRNA precursor (pri-miRNA)1.
      Pri-miRNAs are first processed by the Drosha/DGCR8 microprocessor
      complex into pre-miRNAs2, which are then exported from the nucleus
      to the cytoplasm by Exportin-53. Once in the cytoplasm, Dicer
      processes them further4,5 and these mature dsRNA duplexes are then
      loaded into Argonaute (Ago) proteins to form the RNA-induced
      silencing complex (RISC)6. The sense/passenger strand is
      ejected/degraded, while the guide strand remains associated with
      the RISC7. Depending on the degree of complementarity between the
      guide strand and its target, the outcome of RNAi can either be
      target degradation—most often achieved by siRNAs with full
      complementarity to their target mRNA8—or miRNA-like
      cleavage-independent silencing, mediated by
      deadenylation/degradation or translational repression9. The latter
      mechanism can be initiated with as little as six nucleotide
      base-pairing between a guide RNA’s so-called seed sequence
      (positions 2–7) and fully complementary seed matches in the target
      RNA10,11. This seed-based targeting most often occurs in the 3′UTR
      of a target mRNA12,13.<br>
      <br>
      A number of miRNAs function either as tumor suppressors or as
      oncogenes14. Their cancer-specific activities are usually
      explained by their identified targets, being oncogenes or tumor
      suppressors, respectively14. Examples of targets of
      tumor-suppressive miRNAs are the oncogenes Bcl-2 for miR-15/1615
      and c-Myc for miR-34a16. While many miRNAs have been reported to
      have both tumor suppressive and oncogenic activities depending on
      the cancer context, examples for widely established
      tumor-promoting miRNAs are miR-221/222, miR-21, miR-155, and
      members of the miR-17~92 cluster, or its paralogues miR-106b~25
      and miR-106a~36317,18. In contrast, two of the major
      tumor-suppressive miRNA families are miR-15/16 and the p53
      regulated miR-34a/c and miR-34b19.<br>
      <br>
      We recently discovered that many si- and shRNAs can kill all
      tested cancer cell lines through RNAi by targeting the 3′UTRs of
      critical survival genes (SGs)20. We called this mechanism DISE
      (for death induced by SG elimination). Cancer cells have
      difficulty in developing resistance to this mechanism both in
      vitro and when treated in vivo21. We reported that a 6mer seed
      sequence in the toxic siRNAs is sufficient for effective
      killing20. We have now performed a strand-specific siRNA screen
      with a library of individual siRNAs representing all 4096 possible
      6mer seed sequences in a neutral RNA duplex. This screen, while
      based on siRNA biochemistry, was not designed to identify targets
      that are degraded through siRNA-mediated slicing activity but to
      identify toxicity caused by moderately targeting hundreds of genes
      required for cell survival in a mechanism similar to miRNA-induced
      silencing.<br>
      <br>
      We report that the most toxic 6mer seeds are G-rich with a G
      enrichment towards the 5′ end targeting SGs with a high C content
      in their 3′UTR in a miRNA-like manner. Many tumor-suppressive
      miRNAs such as miR-34a-5p but none of the established oncogenic
      miRNAs contain G-rich 6mer seeds and most of miR-34a-5p's toxicity
      comes from its 6mer seed sequence. Mature miRNAs from older and
      more conserved miRNAs contain less toxic seeds. We demonstrate
      that for most miRNAs the more abundant mature form corresponds to
      the arm that contains the less toxic seed. In contrast, for major
      tumor-suppressive miRNAs, the mature miRNA is derived from the arm
      that harbors the more toxic seed. Our data allow us to conclude
      that while most miRNAs have evolved to avoid targeting survival
      and housekeeping genes, certain tumor-suppressive miRNAs function
      to kill cancer cells through a toxic G-rich 6mer seed targeting
      the 3′UTR of SGs...<br>
      <br>
      <hr width="100%" size="2"><br>
      <a href="https://www.medicalnewstoday.com/articles/323506.php"
        "><b>https://www.medicalnewstoday.com/articles/323506.php</b></a><b><br>
      </b><b><br>
      </b>
      <div align="center"><b>'Triggering the kill switch' without chemo</b><br>
      </div>
      <br>
      ...In last year's paper, Prof. Peter and his team found a sequence
      of six nucleotides contained in small RNAs that made these
      molecules toxic to cancer cells. A nucleotide is "the basic
      structural unit and building block for DNA" and RNA.<br>
      <br>
      In the first recently published study, Prof. Peter found that
      about 3 percent of all the large RNAs can be "cut" into small
      pieces that then act as toxic microRNAs that can kill cancer.<br>
      <br>
      In the second recent study, Prof. Peter's team tested almost 4,100
      different possible combinations of nucleotide bases from those six
      initial nucleotides in an attempt to find the deadliest, most
      toxic combination.<br>
      <br>
      "Based on what we have learned in these two studies, we can now
      design artificial microRNAs that are much more powerful in killing
      cancer cells than even the ones developed by nature," Prof. Peter
      explains. "We absolutely need to turn this into a novel form of
      therapy."<br>
      <br>
      In the second recent study, Prof. Peter's team tested almost 4,100
      different possible combinations of nucleotide bases from those six
      initial nucleotides in an attempt to find the deadliest, most
      toxic combination.<br>
      <br>
      "Based on what we have learned in these two studies, we can now
      design artificial microRNAs that are much more powerful in killing
      cancer cells than even the ones developed by nature," Prof. Peter
      explains. "We absolutely need to turn this into a novel form of
      therapy."<br>
      <br>
      &nbsp;&nbsp;&nbsp; "Now that we know the kill code, we can trigger
      the mechanism without having to use chemotherapy and without
      messing with the genome. We can use these small RNAs directly,
      introduce them into cells and trigger the kill switch."<br>
      <br>
      <hr width="100%" size="2"><br>
      <div align="center"><b>US2018251762</b><br>
        <b>USE OF TRINUCLEOTIDE REPEAT RNAs TO TREAT CANCER<br>
          <br>
          [ <a href="US2018251762A1.pdf">PDF</a> ]<br>
          <br>
          <img src="uspa2018251762.png" alt="sirna" width="628"
            height="126"><br>
        </b> </div>
      <b><br>
      </b>Inventor: PETER MARCUS, et al.<br>
      Applicant: UNIV NORTHWESTERN [US] &nbsp;&nbsp; &nbsp;<br>
      <br>
      Disclosed are compositions and methods related to RNA interference
      (RNAi) and the use of RNAi active sequence for treating diseases
      and disorders. Particular disclosed are toxic RNAi active
      sequences such as siRNA and shRNA for killing cancer cells. The
      disclosed toxic RNAi active sequences typically include
      trinucleotide repeats and preferentially target the expression of
      multiple essential genes for cell survival and/or growth<br>
      <br>
      <b>BACKGROUND</b><b><br>
      </b><br>
      [0004] The field of the invention relates to RNA interference
      (RNAi) and the use of RNAi active sequences for treating diseases
      and disorders. In particular, the field of the invention relates
      to the use of toxic RNAi active sequences for killing cancer
      cells.<br>
      <br>
      [0005] Cancer therapy is only marginally effective and not
      curative because tumor cells will often develop resistance and
      metastasize. This resistance is driven by the enhanced mutagenesis
      rate cancer cells experience and is particularly effective in
      circumventing drugs designed to target a single molecule or
      pathway.<br>
      <br>
      [0006] We recently reported that siRNAs and shRNAs derived from
      CD95, CD95L (17) and other genes in the human genome (47) kill
      cancer cells through RNAi by targeting a network of critical
      survival genes (15,17). DISE (“death induced by survival gene
      elimination”) was found to involve simultaneous activation of
      multiple cell death pathways and cancer cells have a hard time
      developing resistance to this form of cell death (16). DISE was
      found to preferentially affect transformed cells (16) and among
      them cancer stem cells (44). We can artificially induce DISE in
      cancer cells by introducing si/shRNAs that correspond to certain
      gene transcript sequences and by using artificially designed
      siRNAs that do not exist in nature optimized to induce DISE in
      cancer cells (15). In fact, treating tumor bearing mice with
      nanoparticle coupled DISE-inducing siRNAs resulted in reduced
      tumor growth with no toxicity to the mice. We hypothesized that
      DISE is part of a natural anticancer mechanism (15). We therefore
      wondered whether this mechanism could be accidentally triggered
      resulting in human pathology. Diseases that could be caused by
      overactive DISE would have to result in loss of various tissues
      and an RNA component would have to be involved in the disease
      pathology.<br>
      <br>
      [0007] Here, we report that certain repeat sequences (CAG/CUG
      repeats) that are found to cause tissue loss in a number of
      degenerative diseases (i.e., Huntington's disease), are highly
      toxic to cancer cells by targeting multiple survival genes
      containing complementary trinucleotide repeat sequences. The
      significance of this discovery is that cancer cells would need to
      develop multiple concurring mutations to become resistant to this
      form of DISE because multiple death pathways are activated at once
      (17). Indeed, we have not found a single compound or knockdown of
      any gene that can rescue cancer cells from DISE and this DISE-like
      cell death preferentially affects cancer cells (17,44). Here we
      describe an entirely new way to kill cancer cells in vivo by
      introducing trinucleotide repeat derived siRNAs that
      preferentially target a large number of genes that are critical
      for the survival of cancer cells.<br>
      <br>
      <b>SUMMARY</b><b><br>
      </b><br>
      [0008] Disclosed are polynucleotides, compositions, and methods
      related to RNA interference (RNAi). The disclosed polynucleotides,
      compositions, and methods may be utilized for treating diseases
      and disorders through RNAi.<br>
      <br>
      [0009] Particularly disclosed are toxic RNAi active seed sequences
      and methods of using toxic RNAi active seed sequences for killing
      cancer cells. The disclosed toxic RNAi active seed sequences
      typically include trinucleotide RNA nucleotide repeats and
      preferentially target and inhibit the expression of multiple
      essential genes for cell survival and/or growth. The disclosed
      toxic RNAi active seed may be presented or administered in siRNAs,
      shRNAs, and/or vectors that express siRNAs and/or shRNAs.<br>
      <br>
      <b>BRIEF DESCRIPTION OF THE FIGURES</b><b><br>
      </b><br>
      [0010] FIGS. 1A-E. An siRNA duplex comprised of CAG and CUG
      repeats is super toxic to various cancer cell lines of human and
      mouse origin.<br>
      <br>
      [0011] FIG. 1A. Sequence of the siCAG/CUG duplex. dA,
      deoxyadenosine.<br>
      <br>
      [0012] FIG. 1B and FIG. 1C. Confluences over time of human (B) and
      mouse (C) cell lines transfected with 10 nM of either siNT, siL3,
      or siCAG/CUG. M565, 3LL and B 16F10 cells were reverse
      transfected. Values are mean −/+SEM. The 13 cell lines were tested
      between 1 and 6 times in each case with between 3 and 6 technical
      replicates.<br>
      <br>
      [0013] FIG. 1D. Confluence over time of HCT116 cells transfected
      with either siNT or siCAG/CUG at 0.1 or 0.01 nM. Values are mean
      −/+SEM. n=2 biological replicates, 6 technical replicates each.<br>
      <br>
      [0014] FIG. 1E. Viability (ATP content) of HeyA8 cells transfected
      with different concentrations of siNT, siL3 or siCAG/CUG. Values
      are mean −/+SD. n=3 biological replicates, 3 technical replicates
      each.<br>
      <br>
      [0015] FIGS. 2A-E. Identification of the most toxic TNR-based
      siRNAs.<br>
      <br>
      [0016] FIG. 2A. Confluence over time of HeyA8 (top) or A549
      (bottom) cells transfected with 1 nM of either siNT or the four
      trinucleotide repeat (TNR) based siRNAs, siCAG/CUG, siCGG/CCG,
      siGAA/UUC and siCGA/UCG. Values are mean −/+SEM. n=3/2 biological
      replicates (HeyA8/A549), 8 technical replicates.<br>
      <br>
      [0017] FIG. 2B. Results of toxicity screens of 60 TNR-based siRNAs
      in human HeyA8 (top) and mouse M565 (bottom) cells. Cells were
      reverse transfected in triplicate in 384 well plates with 1 nM of
      each siRNA. In all cases, the complementary (sense) strand was
      inactivated by a 2′-OMe modification in positions 1 and 2.
      Trinucleotide repeats were characterized as effecting: (i) no or
      less than 50% loss in viability; (ii) &gt;50% loss in viability;
      or (iii) &gt;75% loss in viability. Family 7 is the only family in
      which all 6 duplexes were super toxic. The six members of Family 5
      have the same GC content and the same nucleotide composition as
      the members of family 7. Each family contains three TNRs in the
      forward frame (for) and three TNRs that are the reverse complement
      (rev) of these TNRs. Values are mean −/+SD. 3 technical
      replicates.<br>
      <br>
      [0018] FIG. 2C. Correlation between the average of two viability
      screens performed in HeyA8 and two performed in M565 cells. The
      data points for the 6 members of TNR family 5 and family 7 are
      labeled. Pearson correlation and p-value is given.<br>
      <br>
      [0019] FIG. 2D. Results of toxicity screen of 60 TNR-based 6 mer
      seeds in a nontoxic backbone siRNA (see FIG. 10B) in HeyA8 cells.
      Values are mean −/+SD. n=3 technical replicates.<br>
      <br>
      [0020] FIG. 2E. Correlation between the average of two viability
      screens performed in HeyA8 with the 60 TNR based siRNAs and the
      screen performed with the 60 TNR based seeds. The data points for
      the 6 members of TNR family 7 and family 10 are labeled. Each
      member of family 7 effected &lt; ̃20% viability and no member of
      family 10 effected &lt; ̃75% viability. Pearson correlation and
      p-value is given.<br>
      <br>
      [0021] FIGS. 3A-E. siCAG/CUG kills cancer cells through RNAi.<br>
      <br>
      [0022] FIG. 3A. Western blot analysis of HeyA8 and A549 cells
      treated with a control SmartPool (siCtr) or an AGO2 siRNA
      SmartPool for 48 hrs.<br>
      <br>
      [0023] FIG. 3B. Confluence over time of HeyA8 and A549 cells after
      transfection with 1 nM siNT or siCAG/CUG. Cells were first
      transfected with either 25 nM of a Ctr SmartPool or the AGO2
      SmartPool and then after 24 hrs transfected with siNT or
      siCAG/CUG. Values are mean −/+SEM. n=3 biological replicates, 3
      technical replicates each.<br>
      <br>
      [0024] FIG. 3C. Left: Sequences with positions of the
      2′-O-methylation labeled in either the passenger/sense (S), the
      guide/antisense (AS) or both (A/AS) strands of siCAG/CUG. Right:
      Confluence over time of HeyA8 cells transfected with 10 nM of the
      four duplexes depicted on the left and two similarly modified
      duplexes derived from siNT. siCAG/CUG and siCAG (5-OMe) effected a
      significant reduction in % confluence. Values are mean −/+SEM. n=2
      biological replicates, 4-6 technical replicates each.<br>
      <br>
      [0025] FIG. 3D. Venn diagrams showing the overlap between 1185
      survival genes (genes identified as critical survival genes in two
      genome-wide lethality screens [45, 46], blue) and genes
      significantly downregulated (red) in cells transfected with either
      siCAG/CUG (top) or siCGA/UCG (bottom) when compared to siNT.<br>
      <br>
      [0026] FIG. 3E. Venn diagram comparing the 3466 genes
      downregulated in the siCAG/CUG treated HeyA8 cells (&gt;1.5 fold
      adj. p &lt;0.05) as determined by RNA-Seq (siCAG/CUG down) (see
      FIG. 3D) and the 1236 genes expression of which inversely
      correlate with the length of CAG repeats recently reported in HD
      patients (CAGnome down) [24].<br>
      <br>
      [0027] FIGS. 4A and 4B. The toxicity of the TNR-based siRNAs
      correlates with the presence of long complementary repeats in the
      ORFs of genes.<br>
      <br>
      [0028] FIG. 4A. Correlation between the average of the two
      viability screens performed in HeyA8 (top panels) or M565 (bottom
      panels) cells with the percentage of TNRs that are part of 6 mer
      or higher longer repeat sequences in either the ORF (left panels)
      or 3′UTR (right panel) of the genes. The data points for the 6
      members of TNR family 5 are labeled in blue and those of family 7
      are labeled in red. Pearson correlations and p-values are given.<br>
      <br>
      [0029] FIG. 4B. The amino acids that are coded by the targeted
      trinucleotide repeats in families 5 and 7.<br>
      <br>
      [0030] FIGS. 5A-G. Killing cancer cells using siCAG/CUG coupled to
      TLP nanoparticles both in vitro and in vivo.<br>
      <br>
      [0031] FIG. 5A. Left: Confluency over time of HeyA8 (Nuc-Red)
      cells treated with 10 nM of either siNT-TLP siL3-TLP, or
      siCAG/CUG-TLP. Right: Phase and red fluorescence image of HeyA8
      (Nuc-Red) cells 90 hrs after transfection with 15 nM of TLPs.
      Values are mean −/+SEM. n=3 technical replicates.<br>
      <br>
      [0032] FIG. 5B. Confluency over time of different human and mouse
      cancer cell lines treated with either 10 nM (OVCAR3, HepG2, M565)
      or 20 nM (A549, T98G) of TLPs. Values are mean −/+SEM. n=2
      biological replicates, 6 technical replicates each.<br>
      <br>
      [0033] FIG. 5C. Percent cell death (Trypan blue counting) of
      GIC-20 neurospheres derived from a patient with glioblastoma six
      days after adding the TLPs (30 nM). Values are mean −/+SD. n=3
      technical replicates. **p&lt;0.01.<br>
      <br>
      [0034] FIG. 5D. Treatment scheme.<br>
      <br>
      [0035] FIG. 5E. Tumor growth over time based on small animal
      imaging of 10&lt;5 &gt;HeyA8-Nuc-red-Luc-neo cells injected i.p.
      into NSG mice treated with either siNT-TLPs or siCAG/CUG-TLPs.
      Treatment group 1 received 18 injections over four weeks and
      treatment group 2 10 injections over three weeks. The
      bioluminescence signal of IVIS #4 and #5 for individual mice is
      shown (right panel). The experiment represents one of two similar
      experiments. Values are mean −/+SD. * p&lt;0.05; **p&lt;0.01;
      ***p&lt;0.0001, NS, not significant.<br>
      <br>
      [0036] FIG. 5F and FIG. 5G. Change in red object count (growth) of
      tumor cells from 3 mice of the siNT-TLP and the siCAG/CUG-TLP
      treatment group 1 either after transfection with 1 nM siNT or
      siCAG/CUG (F) or after incubation with 7.5 nM siNT-TLP or
      siCAG/CUG-TLP (G). 1000 cells per well were plated. Values are
      mean −/+SEM. n=3-8 technical replicates.<br>
      <br>
      [0037] FIG. 6A and 6B. Morphological changes and cell death in
      cells transfected with siCAG/CUG.<br>
      <br>
      [0038] FIG. 6A. Left: DNA fragmentation in HeyA8 and A549 cells
      120/72 hours after transfection with indicated siRNAs. n=3-4
      technical replicates. Right: Viability of human and mouse cancer
      cell lines 96 hours after transfection with the indicated siRNAs.
      Values are mean −/+SD. n=2 biological replicates, 3 technical
      replicates each.<br>
      <br>
      [0039] FIG. 6B. Human and mouse cell lines transfected with either
      siNT or siCAG/CUG. Time points after transfection of picture
      taking is given. Size bar=100 μm.<br>
      <br>
      [0040] FIG. 7. Properties of toxic TNR based siRNAs. Left:
      Sequences of the siCAG repeat in three different frames. Right:
      Confluence over time of HeyA8 cells transfected with 1 nM of
      either siNT or the three duplexes depicted on the left. Values are
      mean −/+SEM. n=3 biological replicates, 4-8 technical replicates
      each.<br>
      <br>
      [0041] FIG. 8. The toxicity of siL3 is solely based on its guide
      strand. Left: Scheme showing positions of the 2′O-methylation in
      either the passenger/sense (S) or the guide/antisense (AS) strand
      of siL3. The siL3 seed region is shown as a green box. Right:
      Confluence over time of HeyA8 cells transfected with 10 nM of the
      four duplexes depicted on the left and two similarly modified
      duplexes derived from siNT. Values are mean −/+SEM. n=3 technical
      replicates.<br>
      <br>
      [0042] FIG. 9. Reproducibility of the TNR siRNA screens. Variation
      between the two screens of the 60 TNR based siRNAs in HeyA8 (left)
      and in M565 (right) cells. The data points for the six members of
      TNR family 5 are labeled in blue those of family 7 are labeled in
      red. Pearson correlations and p-values are given.<br>
      <br>
      [0043] FIG. 10A and 10B. DISE inducing activity of siL3 is mostly
      based on its seed sequence.<br>
      <br>
      [0044] FIG. 10A. Left: Scheme showing the different mutant siNT
      and siL3 duplexes used. The four light blue boxes indicate the
      four positions that in the nontargeting siRNA (siNT) were
      identical to the same positions in siL3. In siL3 the seed sequence
      is shown as a green box, and the siNT seed sequence is shown as a
      grey box. siNT/siL3 is a chimeric duplex comprised of the seed
      sequence of siL3 and the rest of siNT. In the siL3 seed duplex,
      the four siL3 positions in siNT were replaced with the
      complementary nucleotides (i.e. an G:C was changed to a C:G).
      Right: Confluence over time of HeyA8 cells transfected with 10 nM
      of the five duplexes depicted on the left. Values are mean −/+SEM.
      n=6 technical replicates.<br>
      <br>
      [0045] FIG. 10B. Schematic showing the design of the seed siRNAs
      tested in FIG. 2D. Y and Z indicate the Watson-Crick complementary
      nucleotides of the 6 mer seed. The two red Xs indicate the
      position of the 2′O-methylation in the passenger strand.<br>
      <br>
      [0046] FIG. 11. Increased sensitivity of HCT116 Drosha&lt;−/−&gt;
      cells compared to HCT116 wild-type cells. Confluency over time of
      either HCT116 wt (left) or Drosha&lt;−/−&gt; (right) cells
      transfected with 0.1 nM of either siNT or siCAG/CUG. Transfection
      efficiency of the two cell lines was similar as assessed by uptake
      of siGLO [1]. p-value according to polynomial distribution is
      given. Values are mean −/+SEM. n=3 biological replicates, 3-4
      technical replicates each.<br>
      <br>
      [0047] FIG. 12A and 12B. siCAG/CUG induced growth reduction of
      mouse embryonic stem cells requires Ago2.<br>
      <br>
      [0048] FIG. 12A. Sensitivity of mouse embryonic stem cells to
      siCAG/CUG lacking expression of all four AGO proteins with the
      same cells in which we rendered the RISC functional by
      re-expression of human AGO2. Western blot analysis of Ago1-4 k.o.
      mouse embryonic stem cells expressing a Tet inducible AGO2
      protein. Lanes are as follows: 1: Low Dox; 2: 4 days without Dox;
      3: 3 days without Dox-1 day with high Dox; 4: 3 days without Dox-1
      day with high Dox-1 day without Dox; 5: 3 days without Dox-1 day
      with high Dox-2 days without Dox; 6: 3 days without Dox-1 day with
      high Dox-3 days without Dox; 7: 3 days without Dox-1 day with high
      Dox-4 days without Dox; 8: 3 days without Dox-1 day with high
      Dox-7 days without Dox. Low Dox=0.1 μg/ml; high Dox=2.5 μg/ml. n=2
      biological replicates.<br>
      <br>
      [0049] FIG. 12B. Confluence over time of Ago1-4 k.o. cells with
      induced AGO2 with high Dox (left) or without Dox (right) after
      transfection with 5 nM siNT or siCAG/CUG. Two-way ANOVA is given.
      Equal transfection efficiency was established by transfecting
      cells with 5 nM of either siNT or 5′Cy5 labeled siNT followed by
      FACS analysis (inserts). Values are mean −/+SEM. n=3 biological
      replicates, 4 technical replicates each. AGO2 expressing Ago1-4
      k.o. ESCs over-expressing human AGO2 showed a very low but
      reproducible susceptibility to siCAG/CUG. In contrast, Ago1-4 k.o.
      cells were completely resistant to this form of toxicity, despite
      similar transfection efficiencies (see inserts) between Ago1-4
      k.o. cells and those expressing AGO2. The low sensitivity of the
      Ago1-4 k.o./AGO2 cells could either be due to normal cells being
      less sensitive to this form of cell death or could point at a
      functional role of Ago family members other than Ago2 in this
      process.<br>
      <br>
      [0050] FIG. 13. Efficient knockdown of CUG repeat containing genes
      in cells transfected with siCAG. HeyA8 cells were transfected with
      1 nM of either siNT, siCAG/CUG, siCAG (with the CUG containing
      passenger strand modified by 2′O-methylation), or siCUG (with the
      CAG containing passenger strand modified by 2′O-methylation). RNA
      was quantified by real-time PCR. The genes are ranked according to
      their highest fold downregulation in the RNA Seq experiment.
      Values are mean −/+SD. n=2 biological replicates (for siNT and
      siCAG/CUG), 3 technical replicates each.<br>
      <br>
      [0051] FIG. 14A and 14B. Genes containing the 19 mer targeted by
      siCAG are poorly conserved between human and mouse.<br>
      <br>
      [0052] FIG. 14A. Venn diagram of human and mouse ORFs and 3′UTRs
      containing the CAG (left) or CGA (right) trinucleotide repeats.<br>
      <br>
      [0053] FIG. 14B. Venn diagram of human and mouse ORFs and 3′UTRs
      containing the 19 mer sequences completely complementary to the
      CAG (left) or CGA (right) based 19 mer.<br>
      <br>
      [0054] FIGS. 15A-C. No adverse effects in mice treated with
      siCAG/CUG-TLPs.<br>
      <br>
      [0055] FIG. 15A. Weight of the ten mice in treatment group 1 (see
      FIG. 5D) over the course of the treatment. Values are mean −/+SD.
      NS, unpaired p-value not significant.<br>
      <br>
      [0056] FIG. 15B. H&amp;E stained liver sections of two of the mice
      that were treated with either siNT-TLP or siCAG/CUG-TLP on day 27
      in the experiment shown in FIG. 5E).<br>
      <br>
      [0057] FIG. 15C. Serum analysis of the same two mice per treatment
      group. 1=Sample assay value is less than the dynamic range. For
      most assays, the dynamic range low limit is reported. 2=Sample was
      diluted for testing. Assay value for sample was below dynamic
      range, but results have been corrected for dilution. 3=Assay is a
      calculated value. Either or both assay values used in the
      calculation were below the dynamic range of the assay, therefore
      no result is reported.<br>
      <br>
      <b>DETAILED DESCRIPTION</b><b><br>
      </b><br>
      [0068] The disclosed technology relates to nucleic acid and the
      use of nucleic acid for treated diseases and disorders. The terms
      “nucleic acid” and “oligonucleotide,” as used herein, refer to
      polydeoxyribonucleotides (containing 2-deoxy-ribose),
      polyribonucleotides (containing ribose), and to any other type of
      polynucleotide that is an N glycoside of a purine or pyrimidine
      base. As used herein, the terms “A,” “T,” “C”, “G” and “U” refer
      to adenine, thymine, cytosine, guanine, uracil as a nucleotide
      base, respectively. There is no intended distinction in length
      between the terms “nucleic acid,” “oligonucleotide,” and
      “polynucleotide,” and these terms will be used interchangeably.
      These terms refer only to the primary structure of the molecule.
      Thus, these terms include double- and single-stranded DNA, as well
      as double- and single-stranded RNA. For use in the present
      invention, an oligonucleotide also can comprise nucleotide analogs
      in which the base, sugar or phosphate backbone is modified as well
      as non-purine or non-pyrimidine nucleotide analogs.<br>
      <br>
      [0069] A “fragment” of a polynucleotide is a portion of a
      polynucleotide sequence which is identical in sequence to but
      shorter in length than a reference sequence. A fragment may
      comprise up to the entire length of the reference sequence, minus
      at least one nucleotide. For example, a fragment may comprise from
      5 to 1000 contiguous nucleotides of a reference polynucleotide. In
      some embodiments, a fragment may comprise at least 5, 10, 15, 20,
      25, 30, 40, 50, 60, 70, 80, 90, 100, 150, 250, or 500 contiguous
      nucleotides of a reference polynucleotide; in other embodiments a
      fragment may comprise no more than 5, 10, 15, 20, 25, 30, 40, 50,
      60, 70, 80, 90, 100, 150, 250, or 500 contiguous nucleotides of a
      reference polynucleotide; in further embodiments a fragment may
      comprise a range of contiguous nucleotides of a reference
      polynucleotide bounded by any of the foregoing values (e.g. a
      fragment comprising 20-50 contiguous nucleotides of a reference
      polynucleotide). Fragments may be preferentially selected from
      certain regions of a molecule. The term “at least a fragment”
      encompasses the full length polynucleotide. A “variant,” “mutant,”
      or “derivative” of a reference polynucleotide sequence may include
      a fragment of the reference polynucleotide sequence.<br>
      <br>
      [0070] Regarding polynucleotide sequences, percent identity may be
      measured over the length of an entire defined polynucleotide
      sequence, for example, as defined by a particular SEQ ID number,
      or may be measured over a shorter length, for example, over the
      length of a fragment taken from a larger, defined sequence, for
      instance, a fragment of at least 20, at least 30, at least 40, at
      least 50, at least 70, at least 100, or at least 200 contiguous
      nucleotides. Such lengths are exemplary only, and it is understood
      that any fragment length supported by the sequences shown herein,
      in the tables, figures, or Sequence Listing, may be used to
      describe a length over which percentage identity may be measured.<br>
      <br>
      [0071] Regarding polynucleotide sequences, “variant,” “mutant,” or
      “derivative” may be defined as a nucleic acid sequence having at
      least 50% sequence identity to the particular nucleic acid
      sequence over a certain length of one of the nucleic acid
      sequences using blastn with the “BLAST 2 Sequences” tool available
      at the National Center for Biotechnology Information's website.
      (See Tatiana A. Tatusova, Thomas L. Madden (1999), “Blast 2
      sequences—a new tool for comparing protein and nucleotide
      sequences”, FEMS Microbiol Lett. 174:247-250). Such a pair of
      nucleic acids may show, for example, at least 20%, at least 30%,
      at least 40%, at least 50%, at least 60%, at least 70%, at least
      80%, at least 85%, at least 90%, at least 91%, at least 92%, at
      least 93%, at least 94%, at least 95%, at least 96%, at least 97%,
      at least 98%, or at least 99% or greater sequence identity over a
      certain defined length.<br>
      <br>
      [0072] A “recombinant nucleic acid” is a sequence that is not
      naturally occurring or has a sequence that is made by an
      artificial combination of two or more otherwise separated segments
      of sequence. This artificial combination is often accomplished by
      chemical synthesis or, more commonly, by the artificial
      manipulation of isolated segments of nucleic acids, e.g., by
      genetic engineering techniques known in the art. The term
      recombinant includes nucleic acids that have been altered solely
      by addition, substitution, or deletion of a portion of the nucleic
      acid. Frequently, a recombinant nucleic acid may include a nucleic
      acid sequence operably linked to a promoter sequence. Such a
      recombinant nucleic acid may be part of a vector that is used, for
      example, to transform a cell.<br>
      <br>
      [0073] The nucleic acids disclosed herein may be “substantially
      isolated or purified.” The term “substantially isolated or
      purified” refers to a nucleic acid that is removed from its
      natural environment, and is at least 60% free, preferably at least
      75% free, and more preferably at least 90% free, even more
      preferably at least 95% free from other components with which it
      is naturally associated.<br>
      <br>
      [0074] Oligonucleotides can be prepared by any suitable method,
      including direct chemical synthesis by a method such as the
      phosphotriester method of Narang et al., 1979, Meth. Enzymol.
      68:90-99; the phosphodiester method of Brown et al., 1979, Meth.
      Enzymol. 68:109-151; the diethylphosphoramidite method of Beaucage
      et al., 1981, Tetrahedron Letters 22:1859-1862; and the solid
      support method of U.S. Pat. No. 4,458,066, each incorporated
      herein by reference. A review of synthesis methods of conjugates
      of oligonucleotides and modified nucleotides is provided in
      Goodchild, 1990, Bioconjugate Chemistry 1(3): 165-187,
      incorporated herein by reference.<br>
      <br>
      [0075] The term “promoter” as used herein refers to a cis-acting
      DNA sequence that directs RNA polymerase and other trans-acting
      transcription factors to initiate RNA transcription from the DNA
      template that includes the cis-acting DNA sequence.<br>
      <br>
      [0076] As used herein, the term “complementary” in reference to a
      first polynucleotide sequence and a second polynucleotide sequence
      means that the first polynucleotide sequence will base-pair
      exactly with the second polynucleotide sequence throughout a
      stretch of nucleotides without mismatch. The term “cognate” may in
      reference to a first polynucleotide sequence and a second
      polynucleotide sequence means that the first polynucleotide
      sequence will base-pair with the second polynucleotide sequence
      throughout a stretch of nucleotides but may include one or more
      mismatches within the stretch of nucleotides. As used herein, the
      term “complementary” may refer to the ability of a first
      polynucleotide to hybridize with a second polynucleotide due to
      base-pair interactions between the nucleotide pairs of the first
      polynucleotide and the second polynucleotide (e.g., A:T, A:U, C:G,
      G:C, G:U, T:A, U:A, and U:G).<br>
      <br>
      [0077] As used herein, the term “complementarity” may refers to a
      sequence region on an anti-sense strand that is substantially
      complementary to a target sequence but not fully complementary to
      a target sequence. Where the anti-sense strand is not fully
      complementary to the target sequence, mismatches may be optionally
      present in the terminal regions of the anti-sense strand or
      elsewhere in the anti-sense strand. If mismatches are present,
      optionally the mismatches may be present in terminal region or
      regions of the anti-sense strand (e.g., within 6, 5, 4, 3, or 2
      nucleotides of the 5′ and/or 3′ terminus of the anti-sense
      strand).<br>
      <br>
      [0078] The term “hybridization,” as used herein, refers to the
      formation of a duplex structure by two single-stranded nucleic
      acids due to complementary base pairing. Hybridization can occur
      between fully complementary nucleic acid strands or between
      “substantially complementary” nucleic acid strands that contain
      minor regions of mismatch. Conditions under which hybridization of
      fully complementary nucleic acid strands is strongly preferred are
      referred to as “stringent hybridization conditions” or
      “sequence-specific hybridization conditions.” Stable duplexes of
      substantially complementary sequences can be achieved under less
      stringent hybridization conditions; the degree of mismatch
      tolerated can be controlled by suitable adjustment of the
      hybridization conditions. Those skilled in the art of nucleic acid
      technology can determine duplex stability empirically considering
      a number of variables including, for example, the length and base
      pair composition of the oligonucleotides, ionic strength, and
      incidence of mismatched base pairs, following the guidance
      provided by the art (see, e.g., Sambrook et al., 1989, Molecular
      Cloning—A Laboratory Manual, Cold Spring Harbor Laboratory, Cold
      Spring Harbor, N.Y.; Wetmur, 1991, Critical Review in Biochem. and
      Mol. Biol. 26(3/4):227-259; and Owczarzy et al., 2008,
      Biochemistry, 47: 5336-5353, which are incorporated herein by
      reference).<br>
      <br>
      [0079] As used herein, the term “double-stranded RNA” (“dsRNA”)
      refers to a complex of ribonucleic acid molecules having a duplex
      structure comprising two anti-parallel and substantially
      complementary nucleic acid strands.<br>
      <br>
      [0080] As used herein, the term “nucleotide overhang” refers to an
      unpaired nucleotide or nucleotides that extend from the 5′-end or
      3′-end of a duplex structure of a dsRNA when a 5′-end of one
      strand of the dsRNA extends beyond the 3′-end of the other strand,
      or when a 3′-end of one strand of the dsRNA extends beyond the
      5′-end of the other strand.<br>
      <br>
      [0081] As used herein, the term “blunt” refers to a dsRNA in which
      there are no unpaired nucleotides at the 5′-end and/or the 3′-end
      of the dsRNA (i.e., no nucleotide overhang at the 5′-end or the
      3′-end). A “blunt ended” dsRNA is a dsRNA that has no nucleotide
      overhang at the 5′-end or the 3′-end of the dsRNA molecule.<br>
      <br>
      [0082] As used herein, the term “anti-sense strand” refers to a
      strand of a dsRNA which includes a region that is substantially
      complementary to a target sequence (i.e., where the target
      sequence has a sequence corresponding to the sense strand).<br>
      <br>
      [0083] As used herein, the term “sense strand,” refers to the
      strand of a dsRNA that includes a region that is substantially
      complementary to a region of the anti-sense strand and that
      includes a region that substantially corresponds to a region of
      the target sequence.<br>
      <br>
      [0084] As used herein, RNAi active sequences may include “siRNA”
      and “shRNA” and dsRNA that is processed by nucleases to provide
      siRNA and/or shRNA. The term “siRNA” refers to a “small
      interfering RNA” and the term “shRNA” refers to “short hairpin
      RNA.” RNA interference (RNAi) refers to the process of
      sequence-specific post-transcriptional gene silencing in a cell or
      an animal mediated by siRNA and/or shRNA.<br>
      <br>
      [0085] As used herein, the term “siRNA targeted against mRNA”
      refers to siRNA specifically promote degradation of the targeted
      mRNA via sequence-specific complementary multiple base pairings
      (e.g., at least 6 contiguous base-pairs between the siRNA and the
      target mRNA at optionally at least 7, 8, 9, 10, 11, 12, 13, 14,
      15, 16, 17, 18, 19, 20, 21, 22, or 23 contiguous base-pairs
      between the siRNA and the target mRNA).<br>
      <br>
      [0086] The terms “target, “target sequence”, “target region”, and
      “target nucleic acid,” as used herein, are synonymous and refer to
      a region or sequence of a nucleic acid which may be selected as a
      sequence to which the anti-sense strand of siRNA or shRNA is
      substantially complementary to and hybridizes to as discussed
      herein. A target sequence may refer to a contiguous portion of a
      nucleotide sequence of an mRNA molecule of a particular gene,
      including but not limited to, genes that are essential for
      survival and/or growth of cells and in particular cancer cells.
      The target sequence of a siRNA refers to a mRNA sequence of a gene
      that is targeted by the siRNA due to complementarity between the
      anti-sense strand of the siRNA and the mRNA sequence and to which
      the anti-sense strand of the siRNA hybridizes when brought into
      contact with the mRNA sequence.<br>
      <br>
      [0087] As used herein, the term “transfecting” means “introducing
      into a cell” a molecule, which may include a polynucleotide
      molecule such as dsRNA. When referring to a dsRNA, transfecting
      means facilitating uptake or absorption into the cell, as is
      understood by the skilled person. Absorption or uptake of dsRNA
      can occur or may be facilitated through passive diffusive or
      active cellular processes, or through the use of auxiliary agents
      or devices. Transfection into a cell includes methods known in the
      art such as electroporation and lipofection. However, the meaning
      of the term “transfection” is not limited to introducing molecules
      into cells in vitro. As contemplated herein, a dsRNA also may be
      “introduced into a cell,” where the cell is part of a living
      organism. For example, for in vivo delivery, a dsRNA may be
      injected into a tissue site or may be administered systemically.<br>
      <br>
      [0088] As used herein, the terms “silencing” and “inhibiting the
      expression of” refer to at least partial suppression of the
      expression of a target gene, for example, as manifested by a
      reduction of mRNA associated with the target gene.<br>
      <br>
      [0089] As used herein, the phrase “effective amount” shall mean
      that drug dosage that provides the specific pharmacological
      response for which the drug is administered in a significant
      number of patients in need of such treatment. An effective amount
      of a drug that is administered to a particular patient in a
      particular instance will not always be effective in treating the
      conditions/diseases described herein, even though such dosage is
      deemed to be a therapeutically effective amount by those of skill
      in the art.<br>
      <br>
      [0090] As used herein, the term “pharmaceutical composition” may
      include be defined as a composition that includes a
      pharmacologically effective amount of a dsRNA and a
      pharmaceutically acceptable carrier for delivering the dsRNA to
      target cells or target tissue. As used herein, the term
      “pharmaceutically acceptable carrier” refers to a carrier for
      administration of a therapeutic agent which facilitates the
      delivery of the therapeutic agent (e.g., dsRNA) to target cells or
      target tissue. As used herein, the term “therapeutically effective
      amount” refers to that amount of a therapeutic agent that provides
      a therapeutic benefit in the treatment, prevention, or management
      of a disease or disorder (e.g., a cell proliferation disease or
      disorder such as cancer).<br>
      <br>
      [0091] In one aspect, the present inventors disclose an isolated
      double stranded short interfering ribonucleic acid (siRNA)
      molecule or small hairpin ribonucleotide acid (shRNA) molecule
      that silences expression of one or more mRNA' s of essential genes
      that are required for survival and growth of cells such as cancer
      cells. Preferably, the disclosed siRNA molecules or shRNA
      molecules silence the expression of multiple mRNA's of essential
      genes that are required for survival and growth of cells such as
      cancer cells through a process similar to the process called
      “death-induced by survival gene elimination” or “DISE.”<br>
      <br>
      [0092] The mechanism of action of siRNA and shRNA is understood by
      the skilled person. Interfering RNA (RNAi) generally refers to a
      single-stranded RNA (ssRNA) or double-stranded RNA (dsRNA). The
      dsRNA is capable of targeting specific messenger RNA (mRNA) and
      silencing (i.e., inhibiting) the expression of a target gene.
      During this process, dsRNA (which may include shRNA) is
      enzymatically processed into short-interfering RNA (siRNA)
      duplexes of ̃21-23 nucleotides in length. The anti-sense strand of
      the siRNA duplex is then incorporated into a cytoplasmic complex
      of proteins (RNA-induced silencing complex or RISC). The RISC
      complex containing the anti-sense siRNA strand also binds mRNA
      which has a sequence complementary to the anti-sense
      strand-allowing complementary base-pairing between the anti-sense
      siRNA strand and the sense mRNA molecule. The mRNA molecule is
      then specifically cleaved by an enzyme (RNase) associated with
      RISC resulting in specific gene silencing. For gene silencing or
      knock down (i.e., mRNA cleavage) to occur, anti-sense RNA (e.g.,
      siRNA) has to become incorporated into the RISC. This represents
      an efficient process that occurs in nucleated cells during
      regulation of gene expression.<br>
      <br>
      [0093] As such, siRNA-mediated RNA interference may be considered
      to involve two-steps: (i) an initiation step, and (ii) an effector
      step. In the first step, input siRNA is processed into small
      fragments, such as ̃21-23-nucleotide ‘guide sequences.’ The guide
      RNAs can be incorporated into the protein-RNA RISC complex which
      is capable of degrading mRNA. As such, the RISC complex acts in
      the second effector step to destroy mRNAs that are recognized by
      the guide RNAs through base-pairing interactions. RNA interference
      via use of siRNA may be considered to involve the introduction by
      any means of double stranded RNA into a cell which triggers events
      that cause the degradation of a target RNA, and as such siRNA may
      be considered to be a form of post-transcriptional gene silencing.
      The skilled person understands how to prepare and utilize siRNA
      molecules. (See, e.g., Hammond et al., Nature Rev Gen 2: 110-119
      (2001); and Sharp, Genes Dev 15: 485-490 (2001), the contents of
      which are incorporate herein by reference in their entireties).<br>
      <br>
      [0094] For purposes of this application, the anti-sense strand of
      the disclosed RNA molecules (e.g., siRNA molecules) may comprise a
      contiguous nucleotide sequence, where the base sequence of the
      anti-sense strand has substantial or complete sequence
      complementarity to the base sequence of a contiguous nucleotide
      sequence of corresponding length contained in an mRNA sequence of
      the targeted mRNA (e.g., in a non-coding 3′-end of an mRNA
      sequence). Substantial complementary permits some nucleotide
      mismatches (i.e., non-pairing nucleotides) and as such, the
      anti-sense strand of the siRNA need not have full complementarity.<br>
      <br>
      [0095] In some embodiments, at least a portion of an anti-sense
      strand of the disclosed RNA molecules (e.g., siRNA molecules)
      comprises or consists of a sequence that is 100% complementary to
      a target sequence or a portion thereof. In another embodiment, at
      least a portion of an anti-sense strand of an siRNA molecule
      comprises or consists of a sequence that is at least about 90%,
      95%, or 99% complementary to a target sequence or a portion
      thereof. For purposes of this application, the anti-sense strand
      of the disclosed RNA molecules (e.g., siRNA molecules) preferably
      comprises or consists of a sequence that specifically hybridizes
      to a target sequence or a portion thereof so as to inhibit
      expression of the target mRNA.<br>
      <br>
      [0096] In some embodiments, the disclosed RNAs, including siRNAs
      administered in RNAi therapy, may include repeat sequences. For
      example, in some embodiments, the disclosed RNAs may include
      trinucleotide repeats such as any of: (AAA)n, (AAC)n, (AAG)n,
      (AAU)n, (ACA)n, (ACC)n, (ACG)n, (ACU)n, (AGA)n, (AGC)n, (AGG)n,
      (AGU)n, (AUA)n, (AUC)n, (AUG)n, (AUU)n, (CAA)n, (CAC)n, (CAG)n,
      (CAU)n, (CCA)n, (CCC)n, (CCG)n, (CCU)n, (CGA)n, (CGC)n, (CGG)n,
      (CGU)n, (CUA)n, (CUC)n, (CUG)n, (CUU)n, (GAA)n, (GAC)n, (GAG)n,
      (GAU)n, (GCA)n, (GCC)n, (GCG)n, (GCU)n, (GGA)n, (GGC)n, (GGG)n,
      (GGU)n, (GUA)n, (GUC)n, (GUG)n, (GUU)n, (UAA)n, (UAC)n, (UAG)n,
      (UAU)n, (UCA)n, (UCC)n, (UCG)n, (UCU)n, (UGA)n, (UGC)n, (UGG)n,
      (UGU)n, (UUA)n, (UUC)n, (UUG)n, and (UUU)n, where n is an integer,
      typically selected from 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14,
      15, to about 100 or higher. Preferably, the disclosed RNAs may
      include trinucleotide repeats such as any of: (AGC)n, (CAG)n,
      (CUG)n, (GCA)n, (GGU)n and (UGC)n, where n is an integer,
      typically selected from 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14,
      15, to about 100 or higher.<br>
      <br>
      [0097] Methods for preparing and isolating siRNA also are known in
      the art. (See, e.g., Sambrook et al., Molecular Cloning, A
      Laboratory Manual (2.sup.nd Ed., 1989), the content of which is
      incorporated herein by reference in its entirety). The disclosed
      siRNA may be chemically synthesized, using any of a variety of
      techniques known in the art. The disclosed siRNA may include
      modifications, for example, modifications that stabilize the siRNA
      and/or protect the siRNA from degradation via endonucleases and/or
      exonucleases. In some embodiments, the disclosed siRNA may include
      nucleic acid protecting and coupling groups, such as
      dimethoxytrityl at the 5′-end and/or phosphoramidites at the
      3′-end.<br>
      <br>
      [0098] In one embodiment, the disclosed RNAs comprise a double
      stranded region of about 15 to about 30 nucleotides in length.
      Preferably, the disclosed RNAs are about 20-25 nucleotides in
      length. The disclosed RNAs of the present invention are capable of
      silencing the expression of a target sequence in vitro and in
      vivo.<br>
      <br>
      [0099] In one embodiment, the dsRNA disclosed herein comprises a
      hairpin loop structure and may be referred to as shRNA which may
      be processed to a siRNA. In another embodiment, the dsRNA or siRNA
      has an overhang on its 3′ or 5′ ends relative to the target RNA
      which is to be cleaved. The overhang may be 2-10 nucleotides long.
      In one embodiment, the dsRNA or siRNA does not have an overhang
      (i.e., the dsRNA or siRNA has blunt ends).<br>
      <br>
      [0100] In another embodiment, the disclosed RNA molecules (e.g.,
      siRNA molecules) may contain one or more modified nucleotides,
      including one or more modified nucleotides at the 5′ and/or 3′
      terminus of the RNA molecules. In yet another embodiment, the
      disclosed RNA molecules may comprise one, two, three four or more
      modified nucleotides in the double-stranded region. Exemplary
      modified nucleotides may include but are not limited to, modified
      nucleotides such as 2′-O-methyl (2′OMe) nucleotides,
      2′-deoxy-2′-fluoro (2′F) nucleotides, 2-deoxy nucleotides,
      2′-O-(2-methoxyethyl) (MOE) nucleotides, and the like. The
      preparation of modified siRNA is known by one skilled in the art.
      In some embodiments, the disclosed dsRNA molecules include one or
      more modified nucleotides at the 5′-terminus of the passenger
      strand of the dsRNA that prevent incorporation of the passenger
      strand into RISC. (See, e.g., Walton et al., Minireview:
      “Designing highly active siRNAs for therapeutic applications,” the
      FEBS Journal, 277 (2010) 4806-4813).<br>
      <br>
      [0101] In some embodiments, the disclosed RNA molecules are
      capable of silencing one or more target mRNAs and may reduce
      expression of the one or more target mRNAs by at least about 20%,
      30%, 40%, 50%, 60%, 70%, 80%, 90%, or 100% relative to a control
      RNA molecule (e.g., a molecule not exhibiting substantial
      complementarity with the target mRNA). As such, in some
      embodiments, the presently disclosed RNA molecules targeting the
      mRNA of essential genes may be used to down-regulate or inhibit
      the expression of essential genes (e.g., by at least about 20%,
      30%, 40%, 50%, 60%, 70%, 80%, 90%, or 100% relative to a control
      RNA molecule).<br>
      <br>
      [0102] The disclosed RNA molecules may conveniently be delivered
      to a target cell or a target tissue through a number of delivery
      systems. For example, RNA may be delivered via electroporation,
      lipofection, calcium phosphate precipitation, plasmids, viral
      vectors that express the RNA, viral nucleic acids, phage nucleic
      acids, phages, cosmids, nanoparticles, or via transfer of genetic
      material in cells or carriers such as cationic liposomes. In one
      embodiment, transfection of RNA may employ viral vectors, chemical
      transfectants, or physico-mechanical methods such as
      electroporation and direct diffusion of DNA.<br>
      <br>
      [0103] Also disclosed herein are pharmaceutical compositions
      (e.g., pharmaceutical compositions comprising therapeutic RNA) and
      methods of administering pharmaceutical compositions for treating
      diseases and disorders (e.g., cell proliferative diseases and
      disorders such as cancer). The pharmaceutical composition may
      comprise one or more RNAs as therapeutic agents for inhibiting the
      gene activity of one or more essential genes and a pharmaceutical
      acceptable carrier. Pharmaceutically acceptable carriers include,
      but are not limited to, excipients such as inert diluents,
      disintegrating agents, binding agents, lubricating agents,
      sweetening agents, flavoring agents, coloring agents and
      preservatives. Suitable inert diluents include sodium and calcium
      carbonate, sodium and calcium phosphate, and lactose, while corn
      starch and alginic acid are suitable disintegrating agents.
      Pharmaceutical compositions containing RNA may be administered to
      a mammal in vivo to treat cancer. In one embodiment, the
      pharmaceutical formulation includes a dosage suitable for oral
      administration. In another embodiment, the pharmaceutical
      formulation is designed to suit various means for RNA
      administration. Exemplary means include uptake of naked RNA,
      liposome fusion, intramuscular injection via a gene gun,
      endocytosis and the like.<br>
      <br>
      [0104] Toxic RNAi Active Sequence for Killing Cancer Cells<br>
      <br>
      [0105] Disclosed herein are polynucleotide sequences that may be
      utilized in therapeutic methods of killing cancer cells in a
      subject in need thereof. The disclosed polynucleotide sequences
      may be referred to as “toxic RNAi active sequences.” Particularly
      disclosed are toxic RNAi active sequences such as siRNA and shRNA
      and methods of using toxic RNAi active sequence for killing cancer
      cells. The disclosed toxic RNAi active sequences typically include
      trinucleotide repeats and preferentially target and inhibit the
      expression of multiple essential genes for cell survival and/or
      growth through a process similar to the process called
      “death-induced by survival gene elimination” or “DISE.”<br>
      <br>
      [0106] In some embodiments, the disclosed polynucleotide sequences
      include nucleotide trinucleotide repeats such as any of: (AAA)n,
      (AAC)n, (AAG)n, (AAU)n, (ACA)n, (ACC)n, (ACG)n, (ACU)n, (AGA)n,
      (AGC)n, (AGG)n, (AGU)n, (AUA)n, (AUC)n, (AUG)n, (AUU)n, (CAA)n,
      (CAC)n, (CAG)n, (CAU)n, (CCA)n, (CCC)n, (CCG)n, (CCU)n, (CGA)n,
      (CGC)n, (CGG)n, (CGU)n, (CUA)n, (CUC)n, (CUG)n, (CUU)n, (GAA)n,
      (GAC)n, (GAG)n, (GAU)n, (GCA)n, (GCC)n, (GCG)n, (GCU)n, (GGA)n,
      (GGC)n, (GGG)n, (GGU)n, (GUA)n, (GUC)n, (GUG)n, (GUU)n, (UAA)n,
      (UAC)n, (UAG)n, (UAU)n, (UCA)n, (UCC)n, (UCG)n, (UCU)n, (UGA)n,
      (UGC)n, (UGG)n, (UGU)n, (UUA)n, (UUC)n, (UUG)n, (UUU)n, where n is
      an integer, typically selected from 3, 4, 5, 6, 7, 8, 9, 10, 11,
      12, 13, 14, 15, to about 100 or higher. Preferably, the disclosed
      toxic RNAi active sequences may include trinucleotide repeats such
      as any of: (AGC)n, (CAG)n, (CUG)n, (GCA)n, (GGU)n, and (UGC)n,
      where n is an integer, typically selected from 3, 4, 5, 6, 7, 8,
      9, 10, 11, 12, 13, 14, 15, to about 100 or higher. For example,
      the disclosed toxic RNAi active sequences may include the
      nucleotide repeat (CAG)n, or (CUG)n, where n is 3, 4, 5, 6, 7, 8,
      9, 10, 11, 12, 13, 14, 15, to about 100 or higher.<br>
      <br>
      [0107] In some embodiments, the disclosed polynucleotide sequences
      comprise a passenger strand and a guide strand, and may include
      siRNAs and/or shRNAs. The disclosed polynucleotide sequences also
      optionally and preferably comprise: (i) an RNA nucleotide
      trinucleotide repeat sequence (X1X2X3)n, wherein X1, X2, and X3
      independently are selected from any ribonucleotide A, C, G, and U,
      and n is an integer from 3-10; and where optionally and preferably
      (ii) one or more modified nucleotides at the 5′-terminus of the
      passenger strand that prevents loading of the passenger strand
      into the RNA-induced silencing complex (RISC). Preferably, the RNA
      trinucleotide repeat sequence has a GC content of at least 66%.<br>
      <br>
      [0108] In some embodiments, the trinucleotide repeat sequence of
      the disclosed polynucleotide sequences is selected from the group
      consisting of (ACC)n, (ACG)n, (AGC)n, (AGG)n, (CAC)n, (CAG)n,
      (CCA)n, (CCC)n, (CCG)n, (CCU)n, (CGA)n, (CGC)n, (CGG)n, (CGU)n,
      (CUC)n, (CUG)n, (GAC)n, (GAG)n, (GCA)n, (GCC)n, (GCG)n, (GCU)n,
      (GGA)n, (GGC)n, (GGG)n, (GGU)n, (GUC)n, (GUG)n, (UCC)n, (UCG)n,
      (UGC)n, and (UGG)n, Preferably, the trinucleotide repeat sequence
      of the disclosed polynucleotide sequences is selected from the
      group consisting of (AGC)n, (CAG)n, (CUG)n, (GCA)n, (GGU)n, and
      (UGC)n, where n is an integer, typically selected from 3, 4, 5, 6,
      7, 8, 9, 10, 11, 12, 13, 14, 15, to about 100 or higher.
      Particularly, in some embodiments, the trinucleotide repeat
      sequence of the disclosed polynucleotide sequences is CAG or CUG
      and is present in the guide strand.<br>
      <br>
      [0109] Typically, the disclosed polynucleotide sequences comprise
      one or more modified nucleotides at the 5′-terminus of the
      passenger strand that prevents loading of the passenger strand
      into the RNA-induced silencing complex (RISC). In particular, the
      passenger strand may comprise at least two modified nucleotides at
      its 5′-terminus that prevents loading of the passenger strand into
      the RNA-induced silencing complex (RISC). Suitable modified
      nucleotides may include, but are not limited to 2′-O-methyl
      (2′OMe) nucleotides, 2′-deoxy-2′-fluoro (2′F) nucleotides,
      2′-deoxy nucleotides, and 2′-O-(2-methoxyethyl) (MOE) nucleotides.<br>
      <br>
      [0110] The disclosed polynucleotide sequences may include a 3′
      overhang of one or more nucleotides at the passenger strand, the
      guide strand, or both strands of the double-stranded
      polynucleotide. In some embodiments, the double-stranded
      polynucleotide comprises a 3′ overhang of one or two
      deoxyribonucleotide residues (A, C, G, or T) in the passenger
      strand (optionally one or two thymidine residues) and/or the
      double-stranded polynucleotide comprises a 3′ overhang of one or
      two deoxyribonucleotide residues (A, C, G, or T) in the guide
      strand (optionally one or two adenosine residues).<br>
      <br>
      [0111] The identified polynucleotides that exhibit toxicity to
      cancer cells may be formulated as pharmaceutical compositions, for
      example, as pharmaceutical compositions for treating cell
      proliferative diseases and disorders such as cancer. The disclosed
      pharmaceutical compositions may be administered to a subject in
      need thereof, for example, a subject having a cell proliferative
      disease or disorder such as cancer.<br>
      <br>
      Illustrative Embodiments<br>
      <br>
      [0112] The following Embodiments are illustrative and should not
      be interpreted to limit the scope of the claimed subject matter.<br>
      <br>
      [0113] Embodiment 1. A double-stranded polynucleotide comprising a
      passenger strand and a guide strand, the double-stranded
      polynucleotide optionally comprising: (i) a trinucleotide repeat
      sequence (X1X2X3)n, wherein X1, X2, and X3 independently are
      selected from any ribonucleotide A, C, G, and U, and n is an
      integer from 3-10; and optionally where (ii) one or more modified
      nucleotides at the 5′-terminus of the passenger strand that
      prevents loading of the passenger strand into the RNA-induced
      silencing complex (RISC).<br>
      <br>
      [0114] Embodiment 2. The double-stranded polynucleotide of
      embodiment 1, wherein the RNA is an siRNA or an shRNA.<br>
      <br>
      [0115] Embodiment 3. The double-stranded polynucleotide of
      embodiment 1 or 2, wherein the trinucleotide repeat sequence has a
      GC content of at least 33%.<br>
      <br>
      [0116] Embodiment 4. The double-stranded polynucleotide of any of
      the foregoing embodiments, wherein the trinucleotide repeat
      sequence has a GC content of at least 66%.<br>
      <br>
      [0117] Embodiment 5. The double-stranded polynucleotide of any of
      the foregoing embodiments, wherein the trinucleotide repeat
      sequence is selected from the group consisting of (ACC)n, (ACG)n,
      (AGC)n, (AGG)n, (CAC)n, (CAG)n, (CCA)n, (CCC)n, (CCG)n, (CCU)n,
      (CGA)n, (CGC)n, (CGG)n, (CGU)n, (CUC)n, (CUG)n, (GAC)n, (GAG)n,
      (GCA)n, (GCC)n, (GCG)n, (GCU)n, (GGA)n, (GGC)n, (GGG)n, (GGU)n,
      (GUC)n, (GUG)n, (UCC)n, (UCG)n, (UGC)n, and (UGG)n, preferably
      wherein the trinucleotide is selected from the group consisting of
      (AGC)n, (CAG)n, (CUG)n, (GCA)n, (GGU)n, and (UGC)n, where n is an
      integer, typically selected from 3, 4, 5, 6, 7, 8, 9, 10, 11, 12,
      13, 14, 15, to about 100 or higher.<br>
      <br>
      [0118] Embodiment 6. The double-stranded polynucleotide of any of
      the foregoing embodiments, wherein the trinucleotide repeat
      sequence is CAG or CUG and the trinucleotide repeat sequence is
      present in the guide strand.<br>
      <br>
      [0119] Embodiment 7. The double-stranded polynucleotide of any of
      the foregoing embodiments, wherein the passenger strand comprises
      at least two modified nucleotides at its 5′-terminus.<br>
      <br>
      [0120] Embodiment 8. The double-stranded polynucleotide of any of
      the foregoing embodiments, wherein the one or more modified
      nucleotides at the 5′-terminus of the passenger strand are
      selected from the group consisting of 2′-O-methyl (2′OMe)
      nucleotides, 2′-deoxy-2′-fluoro (2′F) nucleotides, 2′-deoxy
      nucleotides, and 2′-O-(2-methoxyethyl) (MOE) nucleotides.<br>
      <br>
      [0121] Embodiment 9. The double-stranded polynucleotide of any of
      the foregoing embodiments, wherein the double-stranded
      polynucleotide comprises a 3′ overhang of one or more nucleotides
      at the passenger strand, the guide strand, or both strands of the
      double-stranded polynucleotide.<br>
      <br>
      [0122] Embodiment 10. The double-stranded polynucleotide of any of
      the foregoing embodiments, comprising a 3′ overhang of one or two
      deoxyribonucleotide residues (A, C, G, or T) in the passenger
      strand, optionally wherein the one or two deoxyribonucleotide
      residues are thymidine residues.<br>
      <br>
      [0123] Embodiment 11. The double-stranded polynucleotide of any of
      the foregoing embodiments, comprising a 3′ overhang of one or two
      deoxyribonucleotide residues (A, C, G, or T) in the guide strand,
      optionally wherein the one or two deoxyribonucleotide residues are
      adenosine residues.<br>
      <br>
      [0124] Embodiment 12. An expression vector that expresses the
      polynucleotide of any of the foregoing embodiments or a
      single-stranded portion thereof.<br>
      <br>
      [0125] Embodiment 13. The expression vector of embodiment 12
      comprising a eukaryotic promoter operably linked to DNA encoding
      the polynucleotide or a single-stranded portion thereof.<br>
      <br>
      [0126] Embodiment 14. The expression vector of embodiment 12 or
      13, wherein the expression vector is a plasmid or a viral
      expression vector.<br>
      <br>
      [0127] Embodiment 15. A pharmaceutical composition comprising: (i)
      the double-stranded polynucleotide of any of the foregoing
      embodiments or a single-stranded portion thereof, or a vector for
      expressing the double-stranded polynucleotide of any of the
      foregoing embodiments or a single-stranded portion thereof; and
      (ii) a pharmaceutically acceptable excipient.<br>
      <br>
      [0128] Embodiment 16. A method for treating a disease or disorder
      in a subject in need thereof, the method comprising administering
      to the subject the pharmaceutical composition of embodiment 12.<br>
      <br>
      [0129] Embodiment 17. The method of embodiment 13, wherein the
      disease or disorder is a cell proliferative disease or disorder
      such as cancer.<br>
      <br>
      [0130] Embodiment 18. A method of inhibiting the growth of a cell
      or killing a cell, the method comprising introducing into the cell
      the double-stranded polynucleotide of any of embodiments 1-11 into
      the cell or a single-stranded portion thereof, or introducing a
      vector that expresses the double-stranded polynucleotide of any of
      embodiments 1-11 or a single-stranded portion thereof.<br>
      <br>
      [0131] Embodiment 19. A nanoparticle comprising the polynucleotide
      of any of embodiments 1-11 or a single-stranded portion thereof.<br>
      <br>
      [0132] Embodiment 20. The nanoparticle of embodiment 19, wherein
      the nanoparticle is a nanoparticle formed from lipoproteins and/or
      phospholipids (e.g., wherein the nanoparticle is a liposome or a
      micelle).<br>
      <br>
      [0133] Embodiment 21. The nanoparticle of embodiment 19 or 20,
      wherein the polynucleotide is a siRNA and the siRNA is coupled to
      a lipoprotein of the nanoparticle.<br>
      <br>
      [0134] Embodiment 22. An expression vector that expresses the
      polynucleotide of any of the foregoing embodiments or a
      single-stranded portion thereof.<br>
      <br>
      [0135] Embodiment 23. The expression vector of embodiment 22
      comprising a eukaryotic promoter operably linked to DNA encoding
      the polynucleotide or a single-stranded portion thereof.<br>
      <br>
      [0136] Embodiment 24. The expression vector of embodiment 22 or
      23, wherein the expression vector is a plasmid or a viral
      expression vector.<br>
      <br>
      EXAMPLES<br>
      <br>
      [0137] The following Examples are illustrative and are
      illustrative and should not be interpreted to limit the scope of
      the claimed subject matter.<br>
      <br>
      [0138] Title—Small interfering RNAs based on huntingtin
      trinucleotide repeats are highly toxic to cancer cells<br>
      <br>
      [0139] Reference is made to Murmann et al. “Small interfering RNAs
      based on huntingtin trinucleotide repeats are highly toxic to
      cancer cells,” (2018), the content of which is incorporate herein
      by reference in its entirety.<br>
      <br>
      <b>[0140] Abstract</b><b><br>
      </b><br>
      [0141] Trinucleotide repeat (TNR) expansions in the genome cause a
      number of degenerative diseases. A prominent TNR expansion
      involves the trinucleotide repeat CAG in the huntingtin (HTT) gene
      responsible for Huntington's disease (HD). Pathology is caused by
      protein and RNA generated from the TNR regions including small
      siRNA-sized repeat fragments. An inverse correlation between the
      length of the repeats in HTT and cancer incidence has been
      reported for HD patients. We now show that siRNAs based on the CAG
      TNR are toxic to cancer cells by targeting genes that contain long
      reverse complimentary TNRs in their open reading frames. Of the 60
      siRNAs based on the different TNRs, the 6 members in the CAG/CUG
      family of related TNRs are the most toxic to both human and mouse
      cancer cells. siCAG/CUG TNR-based siRNAs induce cell death in
      vitro in all tested cancer cell lines and slow down tumor growth
      in a preclinical mouse model of ovarian cancer with no signs of
      toxicity to the mice. We propose to explore TNR-based siRNAs as a
      novel form of anti-cancer reagents.<br>
      <br>
      <b>[0142] Introduction</b><b><br>
      </b><br>
      [0143] Trinucleotide repeat (TNR) expansions are the cause of a
      large number of degenerative disease syndromes characterized by
      amplification of DNA triplet motifs [1]. They include
      spinocerebellar ataxias (SCAs), spinobulbar muscular atrophy
      (SBMA), myotonic dystrophy type 1 (DM1), and Huntington's disease
      (HD) [1, 2]. HD is a dominantly inherited neurodegenerative
      disorder caused by expansion of CAG repeats in the huntingtin
      (HTT) gene. It has been shown that the resulting glutamine
      expansions (polyQ) in HTT are toxic to cells [3, 4] and that the
      length of the CAG amplifications determines severity and onset of
      the disease [2, 4]. In addition to polyQ toxicity,
      repeat-associated, non-ATG translation (RAN translation) was
      discovered as another translation-level pathogenic mechanism of
      CAG repeat-containing mRNAs [5]. More recent evidence however,
      also points toward RNA playing a role in affecting cell viability
      by poly-triplet repeats [1, 6]. Indeed, many of the repeats in
      several TNR diseases are not located in open reading frames (ORFs)
      but in introns or untranslated regions (UTRs) [4]. DM1 is the
      best-characterized disease regarding RNA toxicity. The CUG repeats
      are in the 3′UTR of the dystrophia myotonica protein kinase (DMPK)
      gene, causing most of their toxicity by forming hairpin structures
      [7]. These hairpins are believed to recruit a number of
      RNA-binding proteins to nuclear RNA foci [8]. Another mechanism by
      which CAG/CUG TNRs could be toxic at the RNA level is by
      interfering with cellular splicing. This has been shown for CUG in
      DM1 [9] and CAG in HD [10].<br>
      <br>
      [0144] Mounting evidence suggests the CAG TNR expansions are toxic
      at the RNA level. It was shown in Drosophila that the toxicity of
      the CAG repeat disease gene spinocerebellar ataxia type 3 (SCA3)
      protein ataxin-3, is in large part caused by the trinucleotide
      repeat RNA and not the polyQ protein [11]. Replacing some of the
      glutamine coding CAG repeats with the other codon coding for
      glutamine, CAA, mitigated the toxicity despite similar polyQ
      protein expression levels. Direct toxicity of mRNA with extended
      CAG repeats was also demonstrated in mice [12]. Finally, there is
      convincing evidence that CAG/CUG repeats can give rise to
      RNAi-active small RNAs. In human neuronal cells, expression of the
      CAG expanded exon 1 of HTT (above the threshold for complete
      penetrance which is &gt;40) [6] caused an increase in small CAG
      repeat-derived RNAs (sCAG) of about 21 nt in length. Above a
      certain length, CAG/CUG repeats were found to be cleaved by Dicer,
      the enzyme that generates mature miRNAs from pre-miRNAs before
      they are incorporated into the RNA induced silencing complex
      (RISC) [13]. The CAG repeat derived fragments could bind to
      complementary transcripts and downregulate their expression via an
      RNAi-based mechanism. In a mouse model of HD treatment of the mice
      with a locked nucleic acid-modified 20 mer antisense
      oligonucleotide complementary to the CAG TNR (LNA-CTG) which
      reduced the expression of sCAGs but not of HTT mRNA or protein
      reversed motor deficits [14]. This study identified sCAG as a
      disease causing agent. Since sCAGs, isolated from HD human brains,
      when transfected reduced viability of neurons [6], these sequences
      might affect cell viability through RNAi by targeting genes that
      regulate cell survival.<br>
      <br>
      [0145] We recently reported that siRNAs and shRNAs derived from
      CD95, CD95L [15], and other genes in the human genome [16] kill
      cancer cells through RNAi by targeting a network of critical
      survival genes [15]. DISE (death induced by survival gene
      elimination) was found to involve simultaneous activation of
      multiple cell death pathways, and cancer cells have a hard time
      developing resistance to this form of cell death [17]. DISE was
      found to preferentially affect transformed cells [17]. Because the
      length of the CAG repeats in different CAG repeat diseases has
      been inversely correlated with cancer incidence in various organs
      [18-21], we were wondering whether RNAi active CAG based TNRs
      might be responsible for this phenomenon and whether they could be
      used to kill cancer cells.<br>
      <br>
      [0146] We have now identified an entire family of TNR-based
      siRNAs—which contains the CAG repeat that causes HD—to be at least
      10 times more toxic to cancer cells than any tested DISE-inducing
      si/shRNA. Our data suggest this super toxicity is caused by
      targeting multiple complementary TNR expansions present in the
      open reading frames (ORFs) of multiple genes, rather than in their
      3′UTRs. As a proof of concept, we demonstrate that siCAG/CUG can
      be safely administered to mice to slow down growth of xenografted
      ovarian cancer cells with no obvious toxicity to the animals. We
      are proposing to develop super toxic TNR expansion-based siRNAs
      for cancer treatment.<br>
      <br>
      <b>[0147] Results</b><b><br>
      </b><br>
      [0148] siCAG/CUG kills all cancer cells in vitro. CAG repeats are
      the defining factor in Huntington's disease, and their complement
      CTG is amplified in myotonic dystrophy type 1 (DM1) [1]. We were
      interested in determining whether a 19 mer duplex of CAG and CUG
      repeats (siCAG/CUG) (FIG. 1A) would affect the growth of cancer
      cells. When transfecting siCAG/CUG into various human (FIG. 1B)
      and mouse (FIG. 1C) cancer cell lines at 10 nM, all cancer cells
      stopped growing within hours of transfection and eventually most
      of the cells died with no outgrowth of recovering cells (FIG. 6A).
      All cancer cells transfected with siCAG/CUG showed morphological
      changes similar to the ones we observed in cells undergoing DISE
      (FIG. 6B, [15, 17]). We found that siCAG/CUG killed HCT116 cells
      even when transfected at 10 pM (FIG. 1D). Compared to any other
      si- or shRNA we have tested siCAG/CUG is ̃10-100 times more toxic
      depending on the assay used. When monitoring cell viability (ATP
      content), the IC50 for siL3, the most toxic DISE inducing siRNA we
      have used, was determined to be 0.8 nM and for siCAG/CUG was 0.039
      nM (FIG. 1E).<br>
      <br>
      [0149] Identification of the most toxic TNR based siRNAs. The
      siCAG/CUG repeat 19 mer in all three frames showed roughly the
      same level of toxicity when transfected into HeyA8 cells (FIG. 7).
      To test whether other TNR disease-derived sequences were toxic to
      cancer cells when introduced as siRNAs, the repeats siGAA/UUC (GAA
      is amplified in Friedreich's ataxia [22]), siCGG/CCG (CGG found in
      fragile X tremor ataxia syndrome [FXTAS] and CCG found in Fragile
      XE mental retardation [FRAXE] [1]) were transfected into HeyA8
      (ovarian) and A549 (lung) cancer cells (FIG. 2A). In addition,
      siCGA/UCG was tested because it has the same base composition as
      the super toxic siCAG/CUG TNR. Interestingly, among the four
      tested TNR siRNA duplexes two were super toxic to both cell lines,
      and two showed no toxicity. Most remarkable was the observation
      that siCGA/UCG was among the nontoxic repeats. This finding
      pointed at a sequence specific mechanism behind this phenomenon
      rather than a response of the cells to dsRNA of a specific base
      composition.<br>
      <br>
      [0150] To identify the most toxic TNR sequences in an unbiased
      screen, we designed a library of 19 mer siRNAs based on the 60
      possible TNRs (that contain more than one type of nucleotide). To
      reduce passenger strand loading and determine the toxicity of each
      repeat when loaded into the RISC as a guide strand, we replaced
      positions 1 and 2 of the passenger strand with 2′-O-methylated
      (OMe) nucleotides. To confirm the effect of the OMe modification,
      we modified the toxic CD95L-derived siRNA siL3 in this way. While
      the siL3 duplex modified on the intended passenger strand (S-OMe)
      was slightly more toxic to cells than unmodified siL3, likely
      reflecting a low level of passenger strand loading of siL3,
      neither siL3 modified on the antisense strand (AS-OMe) nor on both
      strands (S/AS-OMe) showed any toxicity (FIG. 8).<br>
      <br>
      [0151] All 60 TNRs were now synthesized with the sense strand
      carrying the OMe modification in positions 1 and 2, allowing us to
      determine the toxicity of each of the 60 antisense sequences.
      HeyA8 cells were transfected with 1 nM of each of the 60 TNRs and
      viability was quantified 96 hrs after transfection. The 60 TNRs
      can be grouped into 10 families [23]. Each family is comprised of
      3 triplets shifted by one nucleotide plus its three complementary
      triplets. In total 30 (50%) of the TNRs were not toxic, 11 (18%)
      were moderately toxic (&gt;50% loss of viability, shown in
      yellow), and 19 (31.7%) were super toxic (&gt;75% loss of
      viability, shown in red) to HeyA8 cells (FIG. 2B, top panels).
      Among the nontoxic TNRs were all 6 members of family 3 (0% GC
      content) and all 6 members of family 10 (100% GC content). All
      other TNR families contained nontoxic and toxic TNRs.
      Interestingly, in some cases just shifting the TNR sequence in the
      19 mer by one nucleotide resulted in opposite effects on viability
      (i.e. AGG and GGA in family 8). In other cases, members of a
      family showed toxicity of one strand but no toxicity of its
      complement (i.e. families 1, 2, 4, 5, 6, 8 and 9). This finding
      suggests a sequence-specific and in some cases frame specific
      activity of the TNRs consistent with RNAi being involved. Due to
      the different base composition of targeted RNAs, the comparison of
      TNR families with the same GC content and base composition is most
      meaningful. Two families contain a balanced GC content of 66.7%
      and identical base composition: family 5 and 7. Remarkably, while
      family 5 contained toxic and nontoxic members, all six TNRs in
      family 7 were super toxic (boxed in red in FIG. 2B). Family 7
      stands out as it contains all permutations of both the CAG and the
      CUG repeats we identified as killing all cancer cells.<br>
      <br>
      [0152] To determine how much of these activities were conserved
      between human and mouse cancer cells, the screen was repeated with
      the mouse liver cancer cell line M565 (FIG. 2B, bottom panels).
      The results for the siRNAs in TNR families 1, 2, 4, 5, 8, and 9
      were somewhat similar to the ones obtained with the human cell
      line, but also showed clear differences. This could be due to
      differences in tissue origin, cell line, or species between the
      two cell lines. Three of the TNR families performed in an
      identical fashion between the two cell lines. Similar to HeyA8
      cells, none of the 12 TNR-derived siRNAs in families 3 or 10
      showed any toxicity in M565 cells. Most strikingly however, was
      the finding that again all 6 members of family 7, which contain
      both the CAG and the CUG repeat, were super toxic to the mouse
      cell line. Screens in both HeyA8 and M565 cells were repeated and
      results showed a high degree of congruence, especially in the
      results of family 7 (FIG. 9). When the average of the screen in
      HeyA8 cells was plotted against the averages of the two screens in
      M565 cells, a significant correlation between the screens was
      found (FIG. 2C) and again the six TNRs in family 7 were most
      consistently toxic. The data suggests the toxicity of this TNR
      family is conserved and it is independent of tissue, cell line and
      species.<br>
      <br>
      [0153] We recently reported that the 6 mer seed sequence of siL3
      was the main determinant of its toxicity [15]. We therefore
      wondered how much of the toxicity of the super toxic TNRs was due
      to complete complementarity of the siRNA and how much was
      dependent on just the 6 mer seed sequence. The data on siL3 were
      obtained by generating chimeric siRNA duplexes between a nontoxic
      control siRNA (siNT) and siL3 by replacing siL3 sequences from
      either end of the duplex with siNT sequences [15]. To generate an
      artificial nontoxic siRNA backbone in which to test all 60 TNR 6
      mer seed sequences, we first replaced 4 positions in the center of
      siNT still identical to the same position in the siL3 sequence
      with the complementary nucleotides, thereby removing any identity
      between siNT and siL3 outside the seed, while maintaining GC
      content (FIG. 10A). This siL3 seed siRNA (siL3 seed) was almost as
      toxic to HeyA8 cells as siL3, confirming that the 6 mer seed
      determined a substantial part of the toxicity of siL3. We
      therefore used the modified siNT backbone to test all possible
      TNR-derived 6 mer seed sequences (FIG. 2D, FIG. 10B). While some
      TNR derived seeds were toxic to HeyA8 cells, there was only a
      moderate level of congruence between the screen with the entire
      TNR 19 mers and one just with the 6 mers in the modified siNT
      backbone (FIG. 2E). Of the 6 super toxic TNRs in family 7 only one
      was also toxic in the 6 mer screen (FIG. 2D and 2E).
      Interestingly, most of the 6 mers in family 10 were toxic although
      no toxicity was observed in the TNR screen (FIG. 2B). We interpret
      this as the inability of these 6 TNRs with their 100% GC content
      to properly enter the RISC. Together these data suggest 19 mer TNR
      siRNAs are toxic to cancer cells by a mechanism distinct from the
      process of DISE which relies on just the seed sequences targeting
      the 3′UTRs of survival genes [15].<br>
      <br>
      [0154] Super toxic TNR-based siRNAs kill cancer cells through RNAi
      resulting in the loss of survival genes. To address the question
      whether the super toxic TNR-based siRNAs killed cancer cells
      through RNAi, we first compared the toxicity of siCAG/CUG in
      HCT116 wild-type and HCT116 Drosha&lt;−/−&gt; cells. DISE inducing
      si- and shRNAs kill Drosha&lt;−/−&gt; cells more efficiently than
      wild-type cells [15]. We had interpreted this as the RISC being
      more available in the absence of most cellular miRNAs, which rely
      on Drosha for processing. While siCAG/CUG was highly toxic to both
      cell lines at early time points, Drosha &lt;−/−&gt; cells were
      more sensitive to growth reduction induced by siCAG than their
      wild-type counterparts (FIG. 11, p=0.038, according to polynomial
      fitting model). To directly test the requirement of AGO2 in the
      siCAG/CUG induced toxicity we knocked down AGO2 in both HeyA8 and
      A549 cells (FIG. 3A) and transfected the cells with either siNT or
      siCAG/CUG (FIG. 3B). Removal of AGO2 from the cells almost
      completely prevented the toxicity of siCAG/CUG confirming
      dependence on the RISC. A dependence on Ago2 for siCAG/CUG
      toxicity was confirmed in Ago1-4 knock-out mouse embryonic
      fibroblasts with re-expressed AGO2 (FIG. 12). These data indicated
      that siCAG/CUG was negatively affecting cells through canonical
      RNAi involving the RISC complex. To confirm this, we modified the
      siCAG siRNAs with the 2′-O-methylation to selectively block
      loading of either the siCAG or the siCUG based strand into the
      RISC (FIG. 3C). When the CAG-based guide strand was modified
      (siCAG AS-OMe), the toxicity of the siCAG/CUG duplex was severely
      reduced. It was not affected when the CUG repeat containing strand
      was 2′-O-methylated (siCAG 5-OMe), confirming that most of the
      toxicity of the siCAG/CUG repeat comes from the CAG repeat strand.
      siCAG/CUG did not have any toxicity when both strands were
      modified indicating most, if not all, of its toxicity requires
      RISC loading confirming that RNAi was responsible for cell death.<br>
      <br>
      [0155] We recently reported that DISE-inducing CD95L derived sh-
      and siRNAs kill cancer cells by targeting the 3′UTR of critical
      survival genes through canonical RNAi [15]. To test whether the
      super toxic siCAG/CUG duplex also killed cancer cells through this
      mechanism, we transfected HeyA8 cells with siNT, siCAG/CUG or the
      nontoxic siCGA/UCG, and subjected the RNA 48 hours after
      transfection to a RNA-Seq analysis. Interestingly, in the cells
      transfected with siCAG/CUG, 3466 genes were down and 867 genes
      were upregulated (&gt;1.5 fold, adjusted p value&lt;0.05) (data
      not shown). A DAVID gene ontology analysis of the upregulated
      genes did not reveal any evidence of an interferon response by the
      cells induced by the transfected siRNA (data not shown). In cells
      transfected with the nontoxic siCGA/UCG, only 194 genes were found
      to be downregulated and 420 genes upregulated.<br>
      <br>
      [0156] We performed a gene set enrichment analysis for a group of
      ̃1800 survival genes and ̃400 nonsurvival genes that were
      identified in a genome-wide CRISPR lethality screen [45] after
      transfecting cells with either siCAG/CUG or siCGA/UCG using a.
      non-targeting siNT served as a control. Similar to cells
      undergoing DISE, when transfected with siCAG/CUG, the ̃1800
      critical survival genes but not the ̃400 nonsurvival control genes
      [15] were significantly enriched in the downregulated genes in
      cells transfected with siCAG/CUG but not in cells transfected with
      siCGA/UCG (data not shown). In fact, we detected a ̃12-fold
      increased percentage of survival genes compared to the
      non-survival genes among the downregulated RNAs in the siCAG/CUG
      treated cells (FIG. 3D)—a higher difference than seen in cells
      treated with DISE-inducing sh- or siRNAs (data not shown).<br>
      <br>
      [0157] We also performed a metascape analysis of 4 RNA-Seq data
      sets of cells into which were introduced siL3, two CD95L derived
      shRNAs (shL1 and shL3), a CD95 derived shRNA (shR6) ORF,
      previously described [15], and the downregulated genes in cells
      transfected with siCAG/CUG. The Metascape gene ontology analysis
      comparing the downregulated genes in cells treated with either
      CD95 or CD95L derived si- or shRNAs with the data from the
      siCAG/CUG treated cells showed a strong overlap in the GO terms
      including cell cycle, response to DNA damage, mitosis, and
      chromatin organization suggesting that cells died through a
      mechanism similar to DISE (data not shown). When the RNA-Seq data
      of cells treated with siCGA/UCG was included in the analysis, not
      a single GO cluster overlapped (data not shown).<br>
      <br>
      [0158] Interestingly, a large genome-wide comparison of
      lymphoblastoid cell lines from 107 HD patients reported an inverse
      correlation between CAG repeat length and downregulated genes.
      Biological pathways that were significantly affected were
      ribosomal process, energy metabolism and cell death pathways [24]
      all consistent with reduced cell viability. We compared the genes
      that were reported to be negatively and significantly correlated
      with the length of the CAG repeats in these patients (1236 genes,
      according to Pearson correlation) with the 3466 genes
      downregulated in the HeyA8 cells transfected with siCAG/CUG. Of
      the 1236 genes downregulated in the patients 182 (14.7%) were also
      downregulated in the siCAG/CUG treated HeyA8 cells (FIG. 3E). In a
      DAVID gene ontology analysis with these 182 genes the two most
      significantly enriched clusters were consistent with genes playing
      a role in cell division and mitosis, consistent with a major
      effect of siCAG/CUG on mitosis (data not shown). In summary, these
      data suggest that the toxicity of the CAG repeat based siRNA may
      involve loss of survival genes and that this form of cell death
      could be related to the TNR activities seen in patients with
      extended CAG repeats.<br>
      <br>
      [0159] Super toxic TNR-derived siRNAs kill cells by targeting TNR
      sequences present in the ORF of genes complementary to the toxic
      siRNA guide strand. To determine which genes and what part of the
      mRNAs could be targeted by toxic TNR-derived siRNAs, we subjected
      ranked lists of downregulated genes of cells treated with either
      siCAG/CUG or siCGA/UCG to a Sylamer analysis [25]. This method
      detects enrichment of seed matches in mRNAs that are complementary
      to the seed of the introduced siRNAs. In particular, we performed
      a 6-nucleotide Sylamer analysis of the ORFs and the 3′UTRs of
      genes deregulated in HeyA8 cells transfected with either siCAG/CUG
      or siCGA/UCG and ordered the genes from most down-regulated to
      most up-regulated.<br>
      <br>
      [0160] When the seed length was set to 6 nts, we detected a minor
      enrichment of the 6 mer TGCTGC in the 3′UTRs of the downregulated
      genes in the cells treated with siCAG/CUG. TGCTGC is the expected
      seed match (position 2-7) of the siCAG 19 mer guide strand (data
      not shown). No significant seed match enrichment was found in
      cells treated with siCGA/UCG or when the ORFs of these genes were
      analyzed.<br>
      <br>
      [0161] We also performed a 10-nucleotide Sylamer analysis of the
      ORFs and 3′UTRs of genes deregulated in HeyA8 cells transfected
      with either siCAG/CUG or siCGA/UCG and ordered the genes from most
      down-regulated to most up-regulated. The three most highly
      enriched sequences were as follows:<br>
      <br>
      &nbsp; ORF 10-mers, siCAG/CUG:<br>
      &nbsp; CTGCTGCTGC,&nbsp; (SEQ ID NO: 9)<br>
      &nbsp; TCTGAGACCA,&nbsp; (SEQ ID NO: 10)<br>
      &nbsp; TGCTGCTGCT&nbsp; (SEQ ID NO: 11)<br>
      &nbsp; ORF 10-mers, siCGA/UCG:<br>
      &nbsp; GGGGGTGGGG,&nbsp; (SEQ ID NO: 12)<br>
      &nbsp; CCTCCCTCCC,&nbsp; (SEQ ID NO: 13)<br>
      &nbsp; CCCCGCCCCC&nbsp; (SEQ ID NO: 14)<br>
      &nbsp; 3′UTR 10-mers, siCAG/CUG:<br>
      &nbsp; GGCCCTGGCC,&nbsp; (SEQ ID NO: 15<br>
      &nbsp; CACTCCCCAC,&nbsp; (SEQ ID NO: 16)<br>
      &nbsp; GGCAGGGGTG&nbsp; (SEQ ID NO: 17)<br>
      &nbsp; 3′UTR 10-mers, siCGA/UCG:<br>
      &nbsp; GGGGGTGGGG,&nbsp; (SEQ ID NO: 18)<br>
      &nbsp; CCTCCCTCCC,&nbsp; (SEQ ID NO: 19)<br>
      &nbsp; CCCCGCCCCC&nbsp; (SEQ ID NO: 20)<br>
      <br>
      [0162] When we analyzed the ORFs of cells treated with siCAG/CUG
      when setting the seed length to the maximum of 10 nts, we found a
      very profound enrichment of two 10 nt sequences (p-value
      ̃10&lt;−50&gt;) that corresponded to positions 1-10 and 2-11,
      respectively of the targeting siCAG 19 mer, which were CTGCTGCTGC
      (SEQ ID NO:9) and TGCTGCTGCT (SEQ ID NO:11). No such enrichment
      was found when the 3′UTRs of the genes were used for the analysis.
      These data suggest that in contrast to DISE-inducing si/shRNAs,
      siCAG/CUG killed cancer cells by targeting long repeat sequences
      located mainly in ORFs. Consistent with this conclusion, genes
      containing either of the two targeted lOmers in their ORFs were
      very strongly enriched among the downregulated genes in siCAG/CUG
      treated cells, while only a weak enrichment was found when the
      3′UTRs were analyzed.<br>
      <br>
      [0163] Now knowing that the toxicity of the siCAG/CUG correlated
      with the presence of targeted trinucleotide repeats found in the
      ORFs of genes, we wondered whether the toxicity across all 60 TNR
      derived siRNAs correlated with the presence of higher ordered
      reverse complementary TNRs in ORFs that could be targeted by the
      TNR siRNAs and whether this was conserved between human and mouse
      cells. We analyzed the frequencies of all triplets combined in
      mouse and human ORFs and 3′UTRs and counted the number of genes
      containing 6 mers, 10 mers, or 19 mers targeted by the 60 TNRs. We
      also performed a set enrichment analysis for genes containing a
      GCTGCTGCTGCTGCTGCTG (SEQ ID NO:21) 19 mer (targeted by the siCAG
      19 mer) in their ORFs and 3′UTRs in cells transfected with
      siCAG/CUG when compared to cells transfected with siNT. When
      counting all 60 single triplets in both the ORFs and 3′UTRs of all
      human and mouse genes they were found to be slightly more abundant
      in the ORF of genes. When separating into individual triplets of
      the 10 families most triplets are found at similar frequencies in
      both the ORFs and 3′UTRs in humans and in mice. This situation
      changed when we focused on targeted (reverse complements of the
      targeting TNRs) repeats. We plotted the results for 6 mers (as
      this is the minimum sequence required for RNAi-based targeting),
      10 mers (the maximum seed length allowed by Sylamer), and 19 mers
      (the length of the siRNAs used). In all cases, with longer
      sequences, the preference for certain triplets became clearer.
      When analyzing the 19 mers, in both human and mouse, the most
      abundant TNRs are members of the super toxic family 7 and barely
      any 19 mers were found in the related family 5. Genes containing
      the 19 mer TNR (or longer) in their ORF targeted by siCAG were
      enriched in the most downregulated genes in cells transfected with
      siCAG/CUG. We selected 5 of the 6 most highly expressed and
      downregulated genes from the RNA-Seq analysis for validation (FIG.
      13). HeyA8 cells were transfected with siRNAs at 1 nM, and the
      mRNAs levels were quantified by real time PCR 10, 20 or 40 hrs
      after transfection. When transfecting siCAG/CUG all 5 CUG
      repeat-containing mRNAs were downregulated as early as 10 hrs with
      maximal downregulation at 40 hrs. Specificity of the targeting was
      established by transfecting the cells with either siCAG or siCUG
      (in which the passenger strand was disabled by adding the
      2′-O-methylation). Only the siCAG-based siRNAs were active in
      silencing the CUG TNR containing genes. These data strongly
      suggested that the toxicity of TNR-based siRNAs in general might
      be explained by the presence of extended reverse complementary
      repeat sequences present preferentially in the ORF of targeted
      genes. This was confirmed by plotting the viability of cells
      treated with any of the 60 TNR siRNAs and the number of targeted
      TNR sequences of 6 nts or longer (FIG. 4A). The highest and most
      significant correlation was found for both human and mouse ORFs.
      Remarkably, while 80-90% of triplets targeted by the 6 members of
      the CGA containing family 5 are present as singular events (blue
      dots), between 50 and 70% of the triplets targeted by members of
      the CAG containing family 7 are found to be part of 6 mers or
      higher ordered TNRs (red dots). Interestingly, the TNRs targeted
      by the 6 toxic members of family 7 code for 5 different amino
      acids (FIG. 4B). As all the 6 members of family 7 are equally
      toxic to cancer cells, this suggests that this involves targeting
      long repeat elements (i.e. TNRs), rather than a requirement for
      poly-homo amino acid coding stretches. This view is supported by
      an analysis of species conservation (FIG. 14). Of the genes that
      contain targeted 19 mers, only 7 of the 99 genes found in either
      the mouse and the human genome overlapped and the genes that are
      targeted do not have shared functions (FIG. 14B). In summary, our
      data provide evidence that TNR-based siRNAs are toxic to cells by
      targeting a number of genes that contain high order trinucleotide
      repeats that are reverse complementary to the targeting TNR. The
      resulting cell death has features of what we recently described as
      DISE, with the main difference that DISE is the result of a
      miRNA-like targeting of short seed matches in the 3′UTRs of
      survival genes, whereas the TNR-induced cell death is an on-target
      effect affecting a larger number of genes that contain targeted
      sequences in their ORF. This now provides an explanation for why
      the most toxic TNR-based siRNAs are much more toxic than
      DISE-inducing siRNAs. Intriguingly, the CAG repeats found in HD
      are part of the most toxic family of TNRs and their reverse
      complementary 19 mers that can serve as targets are the most
      abundant TNR sequences in the ORFs of both human and mouse
      genomes.<br>
      <br>
      [0164] Super toxic CAG/CUG TNR based siRNAs slow down tumor growth
      in vivo with no toxicity to normal tissues. We were wondering
      whether the super toxic TNR-based siRNAs could be used for cancer
      therapy. We decided to deliver the siRNAs to cancer cells in vivo
      using templated lipoprotein (TLP) nanoparticles [26]. Before using
      the TLP particles loaded with the siCAG/CUG duplex (siCAG/CUG-TLP)
      in vivo, we tested their effects on tumor cells in vitro. They
      killed HeyA8 cells more efficiently than siL3-TLPs (FIG. 5A) and
      also slowed down growth of the tested human or mouse cancer cell
      lines (FIG. 5B and data not shown). They also killed neurospheres
      derived from patients with glioblastoma (FIG. 5C). To test the
      activity of the siCAG/CUG-TLPs in vivo we treated orthotopically
      xenografted HeyA8 ovarian cancer cells in mice. Mice injected with
      100,000 tumor cells were i.p. injected with nano particles 5 times
      a week for two weeks (FIG. 5D). After the tenth treatment mice
      were split into two groups, one group continued to receive
      treatment in the third week and the second group did not. This was
      done to determine whether large established tumors would still
      respond to the treatment. The large tumors in treatment group 1
      still benefited from the effect of siCAG/CUG in the third week of
      treatment (FIG. 5E, left panel). In contrast, some tumors in the
      mice in treatment group 2 grew out rapidly, while others showed
      persisting growth reduction (FIG. 5E, right panel). These results
      suggest that established tumors respond to the siCAG/CUG
      treatment. This was confirmed in another experiment in which
      10&lt;6 &gt;HeyA8 cells were injected and mice were first treated
      3 times a week and then switched to daily treatment 19 days after
      tumor cell injection (data not shown).<br>
      <br>
      [0165] To determine whether siCAG/CUG was detrimental to mice,
      mice in treatment group 1 were treated a few more times with the
      siRNA and were analyzed just when the control treated mice were
      moribund at around day 27. We did not see any signs of toxicity in
      any of the mice. They were feeding well (not shown), did not lose
      weight (FIG. 15A), had normal liver histology (FIG. 15B), and
      showed no increase in liver enzymes in the serum (FIG. 15C). These
      data demonstrated that super toxic CAG/CUG TNR-based siRNAs
      delayed tumor growth in vivo 5-6 days with no gross toxicity to
      normal cells and that they could be safely administered using TLP
      nanoparticles.<br>
      <br>
      [0166] To determine whether tumor cells acquired resistance to the
      treatment, we tested tumors from three mice in treatment group 1
      ex vivo. Three tumors of mice treated with siNT-TLP and three
      tumors of mice treated with siCAG/CUG-TLP were transfected with
      the same siRNAs in vitro a day after tumor isolation (FIG. 5F). In
      parallel, these tumor cells were also treated with the
      nanoparticles again (FIG. 5G). In all cases, the tumors from the
      mice that had received the toxic siRNA were as sensitive to the
      toxic effects of siCAG/CUG in vitro as the tumors from mice
      treated with siNT suggesting that cancer cells cannot become
      resistant to cell death induced by the toxic TNR-based siRNAs, at
      least not in the classical sense observed after targeted therapy,
      and that preferentially tumor cells were responding as there were
      no signs of toxicity in the mice.<br>
      <br>
      <b>[0167] Discussion</b><b><br>
      </b><br>
      [0168] The siRNA we used for cancer treatment was a duplex between
      the basic TNR module found in HD (CAG) and the fully complementary
      strand CUG found in Myotonic Dystrophy (DM1). In a screen of all
      TNR-derived siRNAs both CAG and CUG were part of a family that
      contains 6 members all of which were highly toxic to cancer cells
      of both human and mouse origin. This hybrid duplex between the two
      disease molecules was also recently tested in a well-established
      Drosophila model of DM1 [27]. Expression of the two transcripts
      led to the generation of Dicer-2 (dcr-2) and ago2-dependent 21-nt
      TNR-derived siRNAs resulting in high toxicity to the cells. In a
      separate study, it was shown that expression of these
      complementary repeat RNAs leads to dcr-2-dependent
      neurodegeneration [28]. These results suggest that co-expression
      of CAG and CUG repeat-derived sequences may dramatically enhance
      toxicity in human repeat expansion diseases in which anti-sense
      transcription occurs. Antisense transcription was reported to
      occur in SCA8 in two genes encompassing the repeats are expressed:
      ATXN8 (CAG repeat), on the sense strand and ATXN8OS (CTG repeat)
      on the antisense strand [29].<br>
      <br>
      [0169] One could argue that a cancer therapy based on delivering
      siCAG/CUG could be detrimental to patients as TNR expansion
      patients suffer from various pathologies. However, similar to many
      other genes with amplified CAG repeats, HTT is ubiquitously
      expressed throughout the body with somewhat higher expression in
      the brain and in testis [30]. The disease is characterized by
      neurodegeneration affecting the cerebral cortex and neuropathology
      in the striatum, but it also affects other tissues [31]. So, if
      sCAGs are produced in multiple tissues the effects on most normal
      tissues seems to be moderate. Even in the brain, while detrimental
      to HD patients long term, most patients do not have major symptoms
      before the age of 40 [4, 31]. Short-term exposure to toxic sCAGs
      for cancer therapy, as suggested by our mouse experiments, may not
      have a dramatic effect on normal tissues but may be enough to kill
      cancer cells. If a CAG based siRNA were to produce side effects
      particularly in the brain it may be possible to protect the brain
      through local administration of neuroprotecting LNA-CTGs as
      described [14].<br>
      <br>
      [0170] What could be the mechanism of the relative resistance of
      normal versus tumor cells to the toxic siRNAs? Our recent data
      suggested that miRNAs inhibit DISE and in fact may protect normal
      cells from it [15]. Both Drosha and Dicer k.o. HCT116 cells were
      found to be hypersensitive to a DISE inducing siRNA. This is
      entirely consistent with reported activities of CAG repeats which
      may also act though RNAi. Pathogenic Ataxin-3 with amplified CAG
      repeats showed strongly enhanced toxicity in HeLa cells after
      knockdown of Dicer [32]. In addition, it was previously shown in
      Drosophila that impairing miRNA processing dramatically enhanced
      neurodegeneration caused by the CAG repeat gene Ataxin-3. Two fly
      mutants were tested, one with a deficiency in dcr-1, the
      Drosophila Dicer ortholog that is required for miRNA biogenesis,
      and another with a deficiency in R3D1, a gene required for dcr-1
      to function [33]. The authors concluded that “miRNA pathways
      normally play a protective role in polyQ-induced
      neurodegeneration”. In the light of our data it is possible that
      miRNAs might actually protect cells from the toxic effects of TNR
      based siRNAs.<br>
      <br>
      [0171] While HD is the best known disease caused by CAG repeats,
      one of the two diseases first discovered to be caused by TNR
      repeat expansions is the neurodegenerative disorder SBMA/Kennedy
      disease [34] wherein the pathogenic CAG repeat is found in exon 1b
      of the androgen receptor (AR). If patients with amplified CAG
      repeats produce sCAGs, which according to our work may be
      detrimental to cancer cells, one would have to expect that such
      patients have a reduced cancer incidence due to toxic siRNA
      expression in many tissues. Indeed reduced cancer incidence was
      reported for HD and SBMA patient populations in Sweden [18], and
      HD patients in France [19], Denmark [20], and England [21].<br>
      <br>
      [0172] Possibly the clearest connection to cancer has been
      reported for the CAG repeats in the AR gene and prostate cancer
      (PCa). The CAG repeat length in the AR has been inversely linked
      to PCa. While longer repeats (&gt;20 CAGs) confer a protective
      effect among the PCa patients 45 years or older [35], shorter CAG
      repeats have been shown to result in a two-fold increased cancer
      risk [36], a more aggressive disease, and a high risk of distant
      metastases [37-39]. Shortening of CAG repeat length was found in
      in situ lesions of PCa and its possible precursors [40],
      suggesting that PCa avoids longer CAG repeats. This is consistent
      with our finding of super toxicity of CAG based siRNAs.<br>
      <br>
      [0173] There are two observations that suggest the targeted TNRs
      present in the ORFs of certain genes are not there because these
      proteins require stretches of the same amino acid for their
      function which would presumably be conserved between human and
      mouse: first, all six members of the TNR family 7 were super toxic
      targeting 6 different reverse complementary TNRs that code for 5
      different amino acids (FIG. 4B), and second, the genes with the
      longest repeats with complete complementarity to the most toxic
      siCAG/CUG of 19 nts showed little overlap between human and mouse
      (Fig S 14B) and the different targeted genes do not share similar
      functions based on a Metascape analysis (data not shown).
      Interestingly, both the AR and the HTT genes contain some of the
      longest CAG repeats in the human genome, however, those are not
      found in the mouse orthologs at the same positions in the ORF.<br>
      <br>
      [0174] So, if there was no pressure to maintain these TNRs in
      specific genes but rather anywhere in the genome, could there be
      an evolutionary link between TNRs and cancer? A hint may come from
      the way the repeat expansion are generated. It is believed that
      among other mechanisms DNA replication slippage and/or defective
      base excision repair causes expansion of TNRs [41]. Therefore CAG
      repeats could be part of a mechanism used during evolution to
      maintain genome integrity and, in the context of multicellular
      organisms, to prevent cancer formation by producing toxic siRNAs.
      This would occur whenever too many mutations start accumulating in
      cells, one property all cancers have in common.<br>
      <br>
      [0175] While the treatment with siCAG/CUG requires optimization,
      our data on the toxicity of CAG TNR based siRNAs for cancer cells
      but not normal cells when administered in vivo and the reported
      decreased incidence rate for different types of cancer in patients
      with CAG expansions suggest that TNR-based siRNAs may be useful
      for cancer therapy.<br>
      <br>
      <b>[0176] Materials and Methods</b><b><br>
      </b><br>
      [0177] Cell lines and tissue culture. All cells were grown in an
      atmosphere of 5% carbon dioxide (CO2) at 37° C. Unless indicated
      otherwise base media were supplemented with 10% heat-inactivated
      fetal bovine serum (FBS; Sigma-Aldrich) and 1%
      penicillin/streptomycin and L-Glutamine (Mediatech Inc.). Cells
      were dissociated with 0.25% (w/v) Trypsin-0.53 mM EDTA solution
      (Mediatech Inc.). The following cell lines were cultured in
      supplemented RPMI1640 Medium (Mediatech Inc.): Ovarian cancer cell
      lines HeyA8 (RRID:CVCL_8878), OVCAR3 and OVCAR4 (both from Tumor
      Biology Core, Northwestern University), and lung cancer cells A549
      (ATCC CRM-CCL-185) and H460 (ATCC HTB-177). The GBM cell line T98G
      (ATCC CRL1690) was cultured in Eagle's Minimum Essential Medium
      (EMEM) (ATCC). Melanoma B16F10 cells (ATCC CRL-6475) and 293T
      cells (RRID:CVCL0063) were cultured in DMEM (Cellgro). HepG2 (ATCC
      HB-80645) was cultured in EMEM (ATCC). IDB, a mouse ovarian cancer
      cell line, was cultured in DMEM supplemented with 4% FBS, and 10
      mg/1 Insulin, 5.5 mg/1 Transferrin, 6.7 μg/ml Selenium (ITS,
      Mediatech, Inc., 1:10 diluted). and 3LL Lewis lung cancer cells
      (ATCC CRL-1642) were cultured in DMEM. FOSE2 cells are
      spontaneously immortalized ovarian surface epithelial cells, and
      M565 cells are from a spontaneously formed liver cancer in a
      female and male mouse, respectively, both isolated from mice
      carrying a floxed Fas allele [17]. Both were cultured in DMEM/F12
      (Gibco #11330), 1% ITS (Insulin-Transferrin-Selenium Gibco
      51300-044). M565 cells were dissociated with Accutase detachment
      reagent (Fisher Sci.). HCT116 Drosha&lt;−/−&gt; were generated by
      Narry Kim [42]. HCT116 parental (cat#HC19023, RRID:CVCL_0291) and
      the Drosha&lt;−/−&gt; clone (clone #40, cat#HC19020) were
      purchased from Korean Collection for Type Cultures (KCTC). All
      HCT116 cells were cultured in McCoy's medium (ATCC, cat#30-2007).
      Mouse Ago1-4 k.o. embryonic stem cells inducibly expressing human
      FLAG-HA-AGO2 were described in [43]. CELLSTAR tissue culture
      dishes (Greiner Bio-One, cat#664160, cat#639160) were coated with
      0.1% Gelatin solution (Sigma, cat#ES-006-B) for 10 to 30 minutes
      before use. Cells were cultured in DMEM media (Gibco,
      cat#12430054) supplemented with 15% Fetal Bovine Serum (Sigma,
      cat#F2442), 1% NEAA solution (HyClone, cat#SH3023801), 1% GlutaMAX
      100× (Gibco, cat#35050061), 0.0007% 2-Mercaptoethanol (Fisher,
      cat#BP176100), and 10&lt;6 &gt;units/L LIF (Sigma, #ESG1107). The
      cell culture media was refreshed daily. FLAG-HA-AGO2 is under the
      control of a TRE-Tight (TT) doxycycline (Dox)-inducible promoter.
      100 ng/ml doxycycline (Sigma, cat#D9891) was added to the media in
      order to induce moderate level of Ago2 expression to maintain
      normal cell growth. To deplete Ago2 expression in cells,
      doxycycline was withdrawn from media for 4 days. To induce wild
      type level of hAgo2 expression, 2.5 μg/ml doxycycline was added to
      the media. The human GBM derived neurosphere cell line GIC-20
      (infected with pLV-Tomato-IRES-Luciferase) was obtained from Dr.
      Alexander Stegh. Cells were grown as neurospheres in DMEM/F12
      50:50 with L-glutamine (Corning), supplemented with 1% PenStrep,
      B27 (Invitrogen), N2 (Invitrogen), human-Epidermal Growth Factor
      (hEGF; Shenandoah Biotech), Fibroblast Growth Factor (FGF;
      Shenandoah Biotech), Leukemia Inhibitory Factor (LIF; Shenandoah
      Biotech), and GlutaMAX (Life Technologies). HeyA8 xenografted
      tumors nodules were dissected from mice, cut, washed in sterile
      PBS, and dissociated with dissociated with 0.25% (w/v)
      Trypsin-0.53 mM EDTA solution for 20 minutes at 37° C. The
      digestions was stopped by adding full RPMI-1640 medium. After
      centrifugation, the trypsin solution mix was removed, and the
      tumor cells were resuspended in fresh full medium, and strained
      through 70 micron cell strainer. The tumor cell suspension was
      plated over night on 10 cm tissue culture dishes. The following
      day, cells were harvested, counted, and plated on 96-well plates
      for further experiments. Cells were transfected with siRNAs after
      cells had adhered or incubated with siRNA TLPs and then plated.<br>
      <br>
      [0178] Western blot analysis. Primary antibodies for Western blot:
      anti-β-actin antibody (Santa Cruz #sc-47778, RRID:AB_626632),
      anti-human AGO2 (Abcam #AB186733, RRID:AB_2713978). Secondary
      antibodies for Western blot: Goat anti-rabbit; IgG-HRP (Southern
      Biotech #SB-4030-05, RRID:AB_2687483). Reagents used: propidium
      iodide (Sigma-Aldrich #P4864), puromycin (Sigma-Aldrich #P9620)
      and Lipofectamine RNAiMAX (ThermoFisher Scientific #13778150).
      Western blot analysis was performed as recently described [15]<br>
      <br>
      [0179] Transfection with short oligonucleotides. For transfection
      of cancer cells with siRNAs, RNAiMAX was used at a concentration
      optimized for each cell line, following the instructions of the
      vendor. Cell lines were either transfected after cells had adhered
      (forward transfection), or during plating (reverse transfection).
      For an IncuCyte experiment cells were typically plated in 200 μl
      antibiotic free medium, and 50 μl transfection mix with RNAiMAX
      and siRNAs were added. During growth curve acquisitions the medium
      was not exchanged to avoid perturbations. All individual siRNA
      oligonucleotides were ordered from Integrated DNA Technologies
      (IDT). Individual RNA oligos were ordered for the sense and
      antisense oligo; the sense strand had 2 Ts added to the 3′ end;
      antisense strand had 2 deoxy As at the 3′ end. When indicated the
      first two positions at the 5′-end were 2′-O-methylated. Sense and
      antisense oligos were mixed with nuclease free Duplex buffer (IDT,
      Cat.No# 11-01-03-01; 100 mM Potassium Acetate, 30 mM HEPES, pH
      7.5) to 20 μM (working solution), heated up for 2 minutes at 94°
      C., and then the oligos were allowed to cool down to room
      temperature for 30 minutes. siRNA solutions were aliquoted and
      stored at −80° C. The cells were transfected with siRNAs at a
      final concentration of 0.01 nM -10 nM. The following siRNA
      sequences were used: siNT (siNT#2): UGGUUUACAUGUUGUGUGA (SEQ ID
      NO:1) (non targeting in mammalian cells), siNT1:
      UGGUUUACAUGUCGACUAA (SEQ ID NO:2) (non targeting in mammalian
      cells), siL3: GCCCUUCAAUUACCCAUAU (SEQ ID NO:3) (human CD95L exon
      1), siNT/siL3: UGGUUUACAUGUCCCAUAA (SEQ ID NO:4). siNT seed:
      UGGUAAACUAGUUGUCUGA (SEQ ID NO:5), siL3 seed: UGGUAAACUAGUCCCAUAA
      (SEQ ID NO:6). All TNR based 19 mer siRNAs were designed as
      follows: The TNR based siRNA was named according to its
      antisense/guide strand: 2 nt 3′ overhangs were added as described
      above. All TNR based siRNAs were fully complementary 19 mers. For
      instance, the siCAG/CUG sequences are: S: CAGCAGCAGCAGCAGCAGCdAdA
      (SEQ ID NO:7), AS: GCUGCUGCUGCUGCUGCUGTT (SEQ ID NO:8). In all
      siRNAs used in screens the sense/passenger strand was disabled by
      2′-O-methylation in positions 1 and 2 of the sense strand.<br>
      <br>
      [0180] For transfecting Ago1-4 k.o. mouse ESC, cells were cultured
      without doxycycline for three days. Half of the cells were then
      cultured for one more day without doxycycline before transfection.
      The other half was cultured in media containing 2.5 μg/ml of
      doxycycline for one day to induce WT level AGO2 expression before
      transfection. For IncuCyte experiments, The ESCs were transfected
      with either 5 nM siNT or siCAG/CUG using reverse transfection
      method in a 96 well plate coated with 0.1% gelatin. 5000
      cells/well and 0.2 μl RNAiMAX/well were used. One day after
      transfection, 100 μl of media (with or without 2.5 μg/ml
      doxycycline) was added to each well. After that, media was
      refreshed every 2-3 days until the cells grew confluent. For flow
      cytometry experiments, cells were transfected with either 5 nM
      unlabeled siNT or siNT labeled with Cy5 on the 5′ end of the
      antisense strand using reverse transfection method in a 12 well
      plate coated with gelatin in triplicates. 300,000 cells/well and 1
      μl RNAiMAX/well were used. Flow cytometry measurements were
      conducted 24 hours after transfection. For AGO2 knockdown
      experiment, 100,000 cells/well HeyA8 or 200,000 cells/well A549
      cells were reverse transfected in 6-well plate with either
      non-targeting (Dharmacon, cat#D-001810-10-05) or an AGO2 targeting
      siRNA SMARTpool (Dharmacon, cat#L004639-00-005) at 25 nM. 1 μl
      RNAiMAX per well was used for HeyA8 cells and 6 μl RNAiMAX per
      well was used for A549 cells. 24 hours after transfection with the
      SMARTpools, cells were reversed transfected in 96-well plate with
      either siNT or siCAG/CUG at 1 nM and monitored in the IncuCyte.
      0.1 μl/well RNAiMAX was used for HeyA8 cells and 0.6 μl/well
      RNAiMAX was used for A549 cells.<br>
      <br>
      [0181] Total RNA isolation and RNA-seq analysis. HeyA8 cells were
      transfected in 6-wells with siNT or either siCAG/CUG or siCGA/UCG
      oligonucleotides at 1 nM. The transfection mix was removed after 9
      hours. Total RNA was isolated 48 hours after transfection using
      the miRNeasy Mini Kit (Qiagen, Cat.No. 74004) following the
      manufacturers instructions. An on-column digestion step using the
      RNAse-free DNAse Set (Qiagen, Cat.No.: 79254) was included. NGS
      RNA-SEQ library construction and sequencing was performed by the
      University of Chicago Genomics Facility. The quality and quantity
      of RNA samples was assessed using an Agilent bio-analyzer. RNA-SEQ
      libraries were generated using Illumina Stranded TotalRNA TruSeq
      kits according to the Illumina provided protocol and sequencing
      was performed using the Illumina HiSEQ4000 according to Illumina
      provided protocols and reagents. The resulting paired end reads
      were aligned to the hg38 assembly of the human genome with
      Tophat2. HTseq was used to associate the aligned reads with genes,
      and EdgeR was used to identify genes significantly differentially
      expressed between treatments, all as recently described [15]. The
      accession number for the RNA-Seq and expression data reported in
      this work are GSE104552.<br>
      <br>
      [0182] Real-time PCR. Real-time PCR was performed a described
      recently [15] using the following primers: GAPDH (Hs00266705_g1),
      RPL14 (Hs03004339_g1), LRRC59 (Hs00372611_m1), CNPY3
      (Hs01047697_m1), CTSA (Hs00264902_m1), and LRP8 (Hs00182998_m1).<br>
      <br>
      [0183] Monitoring growth over time and quantification of cell
      death. To monitor cell growth over time, cells were seeded between
      125 and 4000 per well in a 96-well plate in triplicates. The plate
      was then scanned using the IncuCyte ZOOM live cell imaging system
      (Essen BioScience). Images were captured at regular intervals, at
      the indicated time points, using a 10× objective. Cell confluence
      was calculated using the IncuCyte ZOOM software (version 2015A).
      IC50 values for siL3 and siCAG/CUG were determined using GraphPad
      Prism 6 software (by logarithm normalized sigmoidal dose curve
      fitting). Quantification of DNA fragmentation (subG1 DNA) was done
      as previously described [15].<br>
      <br>
      [0184] siRNA screens and cell viability assay. HeyA8 or M565 cells
      were expanded and frozen down at the same passage. One week before
      transfection, cells were thawed and cultured in RPMI1640 medium,
      10% FBS and 1% pen/strep. Cells were split three times during the
      week and each time seeded at 4×10&lt;6 &gt;cells total in one T75
      flask. On the day of the transfection, RNA duplexes were first
      diluted with Opti-MEM to make 30 μl solution of 10 nM (for the
      duplexes with the 6 mer seeds) or 1 nM (for the TNR-based
      duplexes) as final concentration in a 384-well plate by Multidrop
      Combi. Lipofectamine RNAiMAX (Invitrogen) was diluted in Opti-MEM
      (6 μl lipid+994 μl of Opti-MEM for HeyA8 and 15.2 μl lipid+984.8
      μl of Opti-MEM for M565 cells). After incubating at room
      temperature for 5 to 10 minutes, 30 μl of the diluted lipid was
      dispensed into each well of the plate that contains RNA duplexes.
      The mixture was pipetted up and down three times by PerkinElmer
      EP3, incubated at room temperature for at least 20 minutes, and
      then the mixture was mixed again by PerkinElmer EP3. 15 μl of the
      mixture was then transferred into wells of three new plates
      (triplicates) using the PerkinElmer EP3. 50 μl with 320 HeyA8 or
      820 M565 cells was then added to each well containing the duplex
      and lipid mix, which results in a final volume of 65 μl. Plates
      were left at room temperature for 30 minutes then moved to a 37°
      C. incubator. 96 hours post transfection, cell viability was
      assayed using CellTiter-Glo (Promega) quantifying cellular ATP
      content. 35 μl medium was removed from each well and 30 μl
      CellTiter-Glo cell viability reagent was added. The plates were
      shaken for 5 minutes and incubated at room temperature for 15
      minutes. Luminescence was then read on the BioTek Synergy NEO2.<br>
      <br>
      [0185] Treatment of xenografted ovarian cancer cells in vivo with
      templated lipoprotein particles (TLP) loaded with siRNAs.
      Synthesis of TLPs and production of siRNA-TLPs was done exactly as
      recently described [44]. 10&lt;5 &gt;HeyA8 cells (infected with a
      luciferase lentivirus and a NucRed lentivirus (Essen Bioscience))
      were injected i.p. into 6-week-old female NSG mice [44] following
      the Northwestern University Institutional Animal Care and Use
      Committee (IACUC)-approved protocol. The growth of tumor cells in
      the mice over time was monitored non-invasively using the IVIS®
      Spectrum in vivo imaging system as recently described [44]. Each
      mouse of a treatment group was injected with 150 μl of either
      siNT-TLP or siCAG/CUG-TLP (1 μM stock).<br>
      <br>
      [0186] Data analyses. To determine the number of triplets, 6 mer,
      10 mer or 19 mer repeat sequences in the ORFs or 3′UTR of human or
      mouse genes, all ORF and 3′UTRs were extracted from the Homo
      sapiens (GRCh38.p7) or Mus musculus (GRCm38.p5) gene dataset of
      the Ensembl database using the Ensembl Biomart data mining tool.
      For each gene, only the longest deposited ORF or 3′UTR was
      considered. Custom perl scripts were used to identify whether each
      3′UTR or ORF contained an identical match to a particular triplet,
      6 mer, 10 mer or 19 mer.<br>
      <br>
      [0187] GSEA was performed using the GSEA v2.2.4 software from the
      Broad Institute (www.http://software.broadinstitute.org/gsea);
      1000 permutations were used. Two list of 1846 survival and 418
      nonsurvival genes were used as recently described [15, 45]. They
      were set as custom gene sets to determine enrichment of survival
      genes versus the nonsurvival control genes in downregulated genes
      from the RNA-seq data. Log(Fold Change) was used as the ranking
      metric. p-values below 0.05 were considered significantly
      enriched. The GO enrichment analysis shown was performed using all
      genes that after alignment and normalization were found to be at
      least 1.5 fold downregulated with an adjusted p values of&lt;0.05,
      using the software available on www.Metascape.org and default
      running parameters. The data sets of HeyA8 cells with introduced
      siL3, shL1, shL3 or shR6 were recently described [15].<br>
      <br>
      [0188] Sylamer analyses [25] were performed using the RNA-seq
      datasets from the HeyA8 cells transfected with siNT, siCAG/CUG or
      siCGA/UCG as recently described [15]. The analyses were performed
      using default settings. Enriched 6 or lOmer motifs were analyzed
      using either ORFs or the 3′UTRs sequences. Sylamer (version
      12-342) was run with the Markov correction parameter set to 4.
      DAVID gene ontology analysis was performed using the tool at
      https://david.ncifcrf.gov/home.jsp and default settings.<br>
      <br>
      [0189] Statistical analyses. Two-way analysis of variances (ANOVA)
      were performed using the Stata 14 software to compare growth
      curves. One-tail student t-test was performed in the software
      package R to compare tumor load between treatment groups. Wilcoxon
      Rank Sum test was performed in R to compare IVIS signal between
      treatment groups. The effects of treatment on wild-type versus
      Drosha&lt;−/−&gt; cells were statistically assessed by fitting
      regression models that included linear and quadratic terms for
      value over time, main effects for treatment and cell type, and
      two- and three-way interactions for treatment, cell-type and time.
      The three-way interaction on the polynomial terms with treatment
      and cell type was evaluated for statistical significance since
      this represents the difference in treatment effects over the
      course of the experiment for the varying cell types. All
      statistical analyses were conducted in Stata 14 (RRID:SCR_012763)
      or R 3.3.1 in Rstudio (RRID:SCR_000432) except for Pearson
      correlation analyses, which were performed using StatPlus 6.2.2.<br>
      <br>
      <b>REFERENCES</b><b><br>
      </b><br>
      [0190] 1. Nalavade R, Griesche N, Ryan D P, Hildebrand S, Krauss S
      (2013) Mechanisms of RNA-induced toxicity in CAG repeat disorders.
      Cell Death Dis 4: e752<br>
      <br>
      [0191] 2. Gatchel J R, Zoghbi H Y (2005) Diseases of unstable
      repeat expansion: mechanisms and common principles. Nat Rev Genet
      6: 743-55<br>
      <br>
      [0192] 3. Ross C A (2002) Polyglutamine pathogenesis: emergence of
      unifying mechanisms for Huntington's disease and related
      disorders. Neuron 35: 819-22<br>
      <br>
      [0193] 4. Orr H T, Zoghbi H Y (2007) Trinucleotide repeat
      disorders. Annu Rev Neurosci 30: 575-621<br>
      <br>
      [0194] 5. Cleary J D, Ranum L P (2017) New developments in RAN
      translation: insights from multiple diseases. Curr Opin Genet Dev
      44: 125-134<br>
      <br>
      [0195] 6. Banez-Coronel M, Porta S, Kagerbauer B, Mateu-Huertas E,
      Pantano L, Ferrer I, Guzman M, Estivill X, Marti E (2012) A
      pathogenic mechanism in Huntington's disease involves small
      CAG-repeated RNAs with neurotoxic activity. PLoS Genet 8: e1002481<br>
      <br>
      [0196] 7. Napierala M, Krzyzosiak W J (1997) CUG repeats present
      in myotonin kinase RNA form metastable “slippery” hairpins. J Biol
      Chem 272: 31079-85<br>
      <br>
      [0197] 8. Wojciechowska M, Krzyzosiak WJ (2011) Cellular toxicity
      of expanded RNA repeats: focus on RNA foci. Hum Mol Genet 20:
      3811-21<br>
      <br>
      [0198] 9. Ho T H, Savkur R S, Poulos M G, Mancini M A, Swanson M
      S, Cooper T A (2005) Colocalization of muscleblind with RNA foci
      is separable from mis-regulation of alternative splicing in
      myotonic dystrophy. J Cell Sci 118: 2923-33<br>
      <br>
      [0199] 10. Lin L, Park J W, Ramachandran S, Zhang Y, Tseng Y T,
      Shen S, Waldvogel H J, Curtis M A, Faull R L, Troncoso J C, et al.
      (2016) Transcriptome sequencing reveals aberrant alternative
      splicing in Huntington's disease. Hum Mol Genet 25: 3454-3466<br>
      <br>
      [0200] 11. Li LB, Yu Z, Teng X, Bonini N M (2008) RNA toxicity is
      a component of ataxin-3 degeneration in Drosophila. Nature 453:
      1107-11<br>
      <br>
      [0201] 12. Hsu R J, Hsiao K M, Lin M J, Li C Y, Wang LC, Chen L K,
      Pan H (2011) Long tract of untranslated CAG repeats is deleterious
      in transgenic mice. PLoS One 6: e16417<br>
      <br>
      [0202] 13. Krol J, Fiszer A, Mykowska A, Sobczak K, de Mezer M,
      Krzyzosiak W J (2007) Ribonuclease dicer cleaves triplet repeat
      hairpins into shorter repeats that silence specific targets. Mol
      Cell 25: 575-86<br>
      <br>
      [0203] 14. Rue L, Banez-Coronel M, Creus-Muncunill J, Giralt A,
      Alcala-Vida R, Mentxaka G, Kagerbauer B, Zomeno-Abellan M T,
      Aranda Z, Venturi V, et al. (2016)<br>
      <br>
      [0204] Targeting CAG repeat RNAs reduces Huntington's disease
      phenotype independently of huntingtin levels. J Clin Invest 126:
      4319-4330<br>
      <br>
      [0205] 15. Putzbach W, Gao QQ, Patel M, van Dongen S,
      Haluck-Kangas A, Sarshad A A, Bartom E, Kim K Y, Scholtens D M,
      Hafner M, et al. (2017) Many si/shRNAs can kill cancer cells by
      targeting multiple survival genes through an off-target mechanism.
      eLife 6: e29702<br>
      <br>
      [0206] 16. Patel M, Peter M E (2017) Identification of
      DISE-inducing shRNAs by monitoring cellular responses. Cell Cycle:
      doi: 10.1080/15384101.2017.1383576<br>
      <br>
      [0207] 17. Hadji A, Ceppi P, Murmann A E, Brockway S, Pattanayak
      A, Bhinder B, Hau A, De Chant S, Parimi V, Kolesza P, et al.
      (2014) Death induced by CD95 or CD95 ligand elimination. Cell
      Reports 10: 208-222<br>
      <br>
      [0208] 18. Ji J, Sundquist K, Sundquist J (2012) Cancer incidence
      in patients with polyglutamine diseases: a population-based study
      in Sweden. Lancet Oncol 13: 642-8<br>
      <br>
      [0209] 19. Coarelli G, Diallo A, Thion M S, Rinaldi D, Calvas F,
      Boukbiza O L, Tataru A, Charles P, Tranchant C, Marelli C, et al.
      (2017) Low cancer prevalence in polyglutamine expansion diseases.
      Neurology 88: 1114-1119<br>
      <br>
      [0210] 20. Sorensen S A, Fenger K, Olsen J H (1999) Significantly
      lower incidence of cancer among patients with Huntington disease:
      An apoptotic effect of an expanded polyglutamine tract? Cancer 86:
      1342-6<br>
      <br>
      [0211] 21. Turner M R, Goldacre R, Goldacre M J (2013) Reduced
      cancer incidence in Huntington's disease: record linkage study
      clue to an evolutionary trade-off? Clin Genet 83: 588-90<br>
      <br>
      [0212] 22. Clark R M, Dalgliesh G L, Endres D, Gomez M, Taylor J,
      Bidichandani S I (2004) Expansion of GAA triplet repeats in the
      human genome: unique origin of the FRDA mutation at the center of
      an Alu. Genomics 83: 373-83<br>
      <br>
      [0213] 23. Kozlowski P, de Mezer M, Krzyzosiak W J (2010)
      Trinucleotide repeats in human genome and exome. Nucleic Acids Res
      38: 4027-39<br>
      <br>
      [0214] 24. Lee J M, Galkina E I, Levantovsky R M, Fossale E, Anne
      Anderson M, Gillis T, Srinidhi Mysore J, Coser K R, Shioda T,
      Zhang B, et al. (2013) Dominant effects of the Huntington's
      disease HTT CAG repeat length are captured in gene-expression data
      sets by a continuous analysis mathematical modeling strategy. Hum
      Mol Genet 22: 3227-38<br>
      <br>
      [0215] 25. van Dongen S, Abreu-Goodger C, Enright A J (2008)
      Detecting microRNA binding and siRNA off-target effects from
      expression data. Nat Methods 5: 1023-5<br>
      <br>
      [0216] 26. McMahon K M, Plebanek M P, Thaxton C S (2016)
      Properties of native high-density lipoproteins inspire synthesis
      of actively targeted in vivo siRNA delivery vehicles. Adv Funct
      Mater 26: 7824-7835<br>
      <br>
      [0217] 27. Yu Z, Teng X, Bonini N M (2011) Triplet repeat-derived
      siRNAs enhance RNA-mediated toxicity in a Drosophila model for
      myotonic dystrophy. PLoS Genet 7: e1001340<br>
      <br>
      [0218] 28. Lawlor K T, O'Keefe L V, Samaraweera S E, van Eyk C L,
      McLeod C J, Maloney C A, Dang T H, Suter C M, Richards R I (2011)
      Double-stranded RNA is pathogenic in Drosophila models of expanded
      repeat neurodegenerative diseases. Hum Mol Genet 20: 3757-68<br>
      <br>
      [0219] 29. Moseley M L, Zu T, Ikeda Y, Gao W, Mosemiller A K,
      Daughters R S, Chen G, Weatherspoon M R, Clark H B, Ebner T J, et
      al. (2006) Bidirectional expression of CUG and CAG expansion
      transcripts and intranuclear polyglutamine inclusions in
      spinocerebellar ataxia type 8. Nat Genet 38: 758-69<br>
      <br>
      [0220] 30. Sharp A H, Loev S J, Schilling G, Li S H, Li X J, Bao
      J, Wagster M V, Kotzuk J A, Steiner J P, Lo A, et al. (1995)
      Widespread expression of Huntington's disease gene (IT15) protein
      product. Neuron 14: 1065-74<br>
      <br>
      [0221] 31. Vonsattel J P, DiFiglia M (1998) Huntington disease. J
      Neuropathol Exp Neurol 57: 369-84<br>
      <br>
      [0222] 32. Bilen J, Liu N, Burnett B G, Pittman R N, Bonini N M
      (2006) MicroRNA pathways modulate polyglutamine-induced
      neurodegeneration. Mol Cell 24: 157-63<br>
      <br>
      [0223] 33. Jiang F, Ye X, Liu X, Fincher L, McKearin D, Liu Q
      (2005) Dicer-1 and R3D1-L catalyze microRNA maturation in
      Drosophila. Genes Dev 19: 1674-9<br>
      <br>
      [0224] 34. La Spada A R, Wilson E M, Lubahn D B, Harding A E,
      Fischbeck K H (1991) Androgen receptor gene mutations in X-linked
      spinal and bulbar muscular atrophy. Nature 352: 77-9<br>
      <br>
      [0225] 35. Gu M, Dong X, Zhang X, Niu W (2012) The CAG repeat
      polymorphism of androgen receptor gene and prostate cancer: a
      meta-analysis. Molecular biology reports 39: 2615-24<br>
      <br>
      [0226] 36. Ingles S A, Ross R K, Yu M C, Irvine R A, La Pera G,
      Haile R W, Coetzee G A (1997) Association of prostate cancer risk
      with genetic polymorphisms in vitamin D receptor and androgen
      receptor. J Natl Cancer Inst 89: 166-70<br>
      <br>
      [0227] 37. Giovannucci E, Stampfer M J, Krithivas K, Brown M, Dahl
      D, Brufsky A, Talcott J, Hennekens C H, Kantoff P W (1997) The CAG
      repeat within the androgen receptor gene and its relationship to
      prostate cancer. Proc Natl Acad Sci U S A 94: 3320-3<br>
      <br>
      [0228] 38. Irvine R A, Yu M C, Ross R K, Coetzee G A (1995) The
      CAG and GGC microsatellites of the androgen receptor gene are in
      linkage disequilibrium in men with prostate cancer. Cancer Res 55:
      1937-40<br>
      <br>
      [0229] 39. Hakimi J M, Schoenberg M P, Rondinelli R H, Piantadosi
      S, Barrack E R (1997) Androgen receptor variants with short
      glutamine or glycine repeats may identify unique subpopulations of
      men with prostate cancer. Clin Cancer Res 3: 1599-608<br>
      <br>
      [0230] 40. Tsujimoto Y, Takakuwa T, Takayama H, Nishimura K,
      Okuyama A, Aozasa K, Nonomura N (2004) In situ shortening of CAG
      repeat length within the androgen receptor gene in prostatic
      cancer and its possible precursors. Prostate 58: 283-90<br>
      <br>
      [0231] 41. Liu Y, Wilson S H (2012) DNA base excision repair: a
      mechanism of trinucleotide repeat expansion. Trends Biochem Sci
      37: 162-72<br>
      <br>
      [0232] 42. Kim Y K, Kim B, Kim V N (2016) Re-evaluation of the
      roles of DROSHA, Export in 5, and DICER in microRNA biogenesis.
      Proc Natl Acad Sci U S A 113: E1881-9<br>
      <br>
      [0233] 43. Zamudio J R, Kelly T J, Sharp P A (2014)
      Argonaute-bound small RNAs from promoter-proximal RNA polymerase
      II. Cell 156: 920-34<br>
      <br>
      [0234] 44. Murmann A E, McMahon K M, Halluck-Kangas A, Ravindran
      N, Patel M, Law C, Brockway S, Wei J J, Thaxton C S, Peter M E
      (2017) Induction of DISE in ovarian cancer cells in vivo.
      Oncotarget 8: 84643-84658<br>
      <br>
      [0235] 45. Wang T, Birsoy K, Hughes N W, Krupczak K M, Post Y, Wei
      J J, Lander E S, Sabatini DM (2015) Identification and
      characterization of essential genes in the human genome. Science
      350: 1096-101<br>
      <br>
      [0236] 46. Blomen V A, Majek P, Jae L T, Bigenzahn J W,
      Nieuwenhuis J, Staring J, Sacco R, van Diemen F R, Olk N, Stukalov
      A, et al. (2015) Gene essentiality and synthetic lethality in
      haploid human cells. Science 350: 1092-6<br>
      <br>
      [0237] 47. Ceppi P, Hadji A, Kohlhapp F, Pattanayak A, Hau A, Xia
      L, Liu H, Murmann A E, Peter M E. (2014). CD95 and CD95L promote
      and protect cancer stem cells. Nature Commun. 5:5238...<br>
      <br>
      <hr width="100%" size="2"><br>
      <br>
    </blockquote>
    <div align="center"><img alt="" src="0logo.gif" width="124"
        height="82"><br>
      <br>
    </div>
    <div style="text-align: center;"><b>Your Support Maintains this
        Service -- </b><b><br>
      </b> <b><br>
      </b> <b>BUY</b><b><br>
      </b> <b><br>
      </b> <big><b>The</b><b> Rex Research Civilization Kit </b></big><b><br>
      </b> <b><br>
      </b> <b>... It's Your Best Bet &amp; Investment in Sustainable
        Humanity on Earth ... </b><b><br>
      </b><b> Ensure &amp; Enhance Your Survival &amp; Genome
        Transmission ...</b><b> </b><b><br>
      </b> <b>Everything @ rexresearch.com ( plus Bonus Files ) on a
        Thumb Drive ( 8 GB / USB 2.0 ) ! </b><b><br>
      </b> <b><br>
      </b> <b><a href="order.htm"
          ">ORDER PAGE</a></b><b><br>
      </b> <b><br>
      </b> </div>
    <b> </b>
    <hr style="width: 62%; height: 2px;">
    <script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
    <script type="text/javascript">
try{
var pageTracker = _gat._getTracker("UA-xxxxxx-x");
pageTracker._trackPageview();
} catch(err) {}
</script>
    <script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
    <script type="text/javascript">
try{
var pageTracker = _gat._getTracker("UA-xxxxxx-x");
pageTracker._trackPageview();
} catch(err) {}
</script>&gt;
  </body>
</html>
