<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
  <head>
    <meta http-equiv="content-type" content="text/html; charset=UTF-8">
    <title>Shosuke Yoshida , et al. -- Ideonella sakaiensis vs Polyester</title>
    <!-- saved from url=http://www.rexresearch.com -->
  </head>
  <body>
    <blockquote><b><a href="../index.htm"><img src="0logo.gif"
            alt="logo" width="124" height="82" border="0"></a><br>
        <a href="../index.htm">rexresearch.com</a><br>
      </b>
      <hr width="100%" size="2"><b><br>
      </b>
      <div align="center"><b><font size="+2">Shosuke Yoshida</font><i> ,
            et al.</i></b><br>
        <b> </b><b> </b><font size="+2"><b><i>Ideonella sakaiensis</i>
            vs Polyester</b><br>
        </font></div>
      <br>
      <hr width="100%" size="2"><br>
      <a
href="https://www.theguardian.com/environment/2016/mar/10/could-a-new-plastic-eating-bacteria-help-combat-this-pollution-scourge"
        "><b>https://www.theguardian.com/environment/2016/mar/10/could-a-new-plastic-eating-bacteria-help-combat-this-pollution-scourge</b></a><b><br>
      </b><b><br>
      </b>
      <div align="center"><b>Could a new plastic-eating bacteria help
          combat this pollution scourge?</b><br>
      </div>
      <div align="center"><i>Scientists have discovered a species of
          bacteria capable of breaking down commonly used PET plastic
          but remain unsure of its potential applications</i><br>
        <b>Karl Mathiesen</b><br>
        <br>
      </div>
      PET makes up almost one sixth of the world’s annual plastic
      production but only about half is ever collected for recycling.<br>
      <br>
      Nature has begun to fight back against the vast piles of filth
      dumped into its soils, rivers and oceans by evolving a
      plastic-eating bacteria – the first known to science.<br>
      <br>
      In a report published in the journal Science, a team of Japanese
      researchers described a species of bacteria that can break the
      molecular bonds of one of the world’s most-used plastics -
      polyethylene terephthalate, also known as PET or polyester.<br>
      <br>
      The Japanese research team sifted through hundreds of samples of
      PET pollution before finding a colony of organisms using the
      plastic as a food source.<br>
      <br>
      Further tests found the bacteria almost completely degraded
      low-quality plastic within six weeks. This was voracious when
      compared to other biological agents; including a related bacteria,
      leaf compost and a fungus enzyme recently found to have an
      appetite for PET.<br>
      <br>
      “This is the first rigorous study – it appears to be very
      carefully done – that I have seen that shows plastic being
      hydrolyzed [broken down] by bacteria,” said Dr Tracy Mincer, a
      researcher at Woods Hole Oceanographic Institution.<br>
      <br>
      The molecules that form PET are bonded very strongly, said Prof
      Uwe Bornscheuer in an accompanying comment piece in Science.
      “Until recently, no organisms were known to be able to decompose
      it.”<br>
      <br>
      In a Gaian twist, initial genetic examination revealed the
      bacteria, named Ideonella sakaiensis 201-F6, may have evolved
      enzymes specifically capable of breaking down PET in response to
      the accumulation of the plastic in the environment in the past 70
      years.<br>
      <br>
      Such rapid evolution was possible, said Enzo Palombo, a professor
      of microbiology at Swinburne University, given that microbes have
      an extraordinary ability to adapt to their surroundings. “If you
      put a bacteria in a situation where they’ve only got one food
      source to consume, over time they will adapt to do that,” he said.<br>
      <br>
      “I think we are seeing how nature can surprise us and in the end
      the resiliency of nature itself,” added Mincer.<br>
      <br>
      The bacteria took longer to eat away highly crystallised PET,
      which is used in plastic bottles. That means the enzymes and
      processes would need refinement before they could be useful for
      industrial recycling or pollution clean-up.<br>
      <br>
      “It’s difficult to break down highly crystallised PET,” said Prof
      Kenji Miyamoto from Keio University, one of the authors of the
      study. “Our research results are just the initiation for the
      application. We have to work on so many issues needed for various
      applications. It takes a long time,” he said.<br>
      <br>
      A third of all plastics end up in the environment and 8m tonnes
      end up in the ocean every year, creating vast accumulations of
      life-choking rubbish.<br>
      <br>
      PET makes up almost one-sixth of the world’s annual plastic
      production of 311m tons. Despite PET being one of the more
      commonly recycled plastics, the World Economic Forum (WEF) reports
      that only just over half is ever collected for recycling and far
      less actually ends up being reused.<br>
      <br>
      Advances in biodegradable plastics and recycling offer hope for
      the future, said Bornscheuer, “but [this] does not help to get rid
      of the plastics already in the environment”.<br>
      <br>
      However the potential applications of the discovery remain
      unclear. The most obvious use would be as a biological agent in
      nature, said Palombo. Bacteria could be sprayed on the huge
      floating trash heaps building up in the oceans. This method is
      most notably employed to combat oil spills.<br>
      <br>
      This particular bacteria would not be useful for this process as
      it only consumes PET, which is too dense to float on water. But
      Bornscheuer said the discovery could open the door to the
      discovery or manufacture of biological agents able to break down
      other plastics.<br>
      <br>
      Palombo said the discovery suggested that other bacteria may have
      already evolved to do this job and simply needed to be found.<br>
      <br>
      “I would not be surprised if samples of ocean plastics contained
      microbes that are happily growing on this material and could be
      isolated in the same manner,” he said.<br>
      <br>
      But Mincer said breaking down ocean rubbish came with dangers of
      its own. Plastics often contain additives that can be toxic when
      released. WEF estimates that the 150m tonnes of plastic currently
      in the ocean contain roughly 23m tonnes of additives.<br>
      <br>
      “Plastic debris may have been less toxic in the whole unhydrolyzed
      form where it would ultimately have been buried in the sediments
      on a geological timescale,” said Mincer.<br>
      <br>
      Beyond dealing with the plastic already fouling up the
      environment, the bacteria could potentially be used in industrial
      recycling processes.<br>
      <br>
      “Certainly, the use of these microbes or enzymes could play a role
      in remediation of plastic in a controlled reactor,” said Mincer.<br>
      <br>
      Miyamoto’s team suggested that the environmentally-benign
      constituents left behind by the bacteria could be the same ones
      from which the plastic is formed. If this were true and a process
      could be developed to isolate them, Bornscheuer said: “This could
      provide huge savings in the production of new polymer without the
      need for petrol-based starting materials.” According to the WEF,
      6% of global oil production is devoted to the production of
      plastics.<br>
      <br>
      But the plastics industry said the potential for a new biological
      process to replace or augment the current mechanical recycling
      process was very small.<br>
      <br>
      “PET is 100% recyclable,” said Mike Neal, the chairman of the
      Committee of PET Manufacturers in Europe. “I expect that a
      biodegradation system would require a similar engineering process
      to chemical depolymerisation and as such is unlikely to be
      economically viable,” he said.<br>
      <br>
      <hr width="100%" size="2"><br>
      <a href="http://science.sciencemag.org/content/351/6278/1196"
        "><b>http://science.sciencemag.org/content/351/6278/1196</b><b><br>
        </b><b> </b></a><b>Science&nbsp; 11 Mar 2016: Vol. 351, Issue
        6278, pp. 1196-1199</b><b><br>
      </b><b>DOI: 10.1126/science.aad6359</b><b><br>
      </b><br>
      <div align="center"><b>A bacterium that degrades and assimilates
          poly(ethylene terephthalate)</b><br>
        <b>Shosuke Yoshida, <i>et al.</i></b><br>
        <br>
      </div>
      <b> </b>Bacteria isolated from outside a bottle-recycling
      facility can break down and metabolize plastic. The proliferation
      of plastics in consumer products, from bottles to clothing, has
      resulted in the release of countless tons of plastics into the
      environment. Yoshida et al. show how the biodegradation of
      plastics by specialized bacteria could be a viable bioremediation
      strategy (see the Perspective by Bornscheuer). The new species,
      Ideonella sakaiensis, breaks down the plastic by using two enzymes
      to hydrolyze PET and a primary reaction intermediate, eventually
      yielding basic building blocks for growth.<br>
      <br>
      <b>Abstract</b><br>
      Poly(ethylene terephthalate) (PET) is used extensively worldwide
      in plastic products, and its accumulation in the environment has
      become a global concern. Because the ability to enzymatically
      degrade PET has been thought to be limited to a few fungal
      species, biodegradation is not yet a viable remediation or
      recycling strategy. By screening natural microbial communities
      exposed to PET in the environment, we isolated a novel bacterium,
      Ideonella sakaiensis 201-F6, that is able to use PET as its major
      energy and carbon source. When grown on PET, this strain produces
      two enzymes capable of hydrolyzing PET and the reaction
      intermediate, mono(2-hydroxyethyl) terephthalic acid. Both enzymes
      are required to enzymatically convert PET efficiently into its two
      environmentally benign monomers, terephthalic acid and ethylene
      glycol.<br>
      <br>
      <hr width="100%" size="2"><br>
      <div align="center"><b>WO2015025861</b><br>
        <b>AROMATIC POLYESTER DECOMPOSITION ENZYME AND METHOD FOR
          DECOMPOSING AROMATIC POLYESTER USING SAID ENZYME</b><br>
        <b>&nbsp;&nbsp;&nbsp; [ <a href="WO2015025861A1.pdf">PDF</a> ]</b><br>
      </div>
      <b>Inventor: MIYAMOTO KENJI,&nbsp; et al.</b><br>
      <br>
      Provided is an enzyme for hydrolyzing an aromatic polyester resin
      such as PET resin, and provided is a method for decomposing an
      aromatic polyester resin such as PET resin using said enzyme. An
      aromatic polyester such as polyethylene terephthalate (PET) can be
      decomposed by an aromatic polyester decomposition enzyme composed
      of an amino acid sequence represented by sequence nos. 2 or 4 of
      the sequence listing. Monohydryoxy ethyl terephthalate (MHET)
      produced by enzymatic decomposition of an aromatic polyester such
      as polyethylene terephthalate (PET) can be furthermore decomposed
      to completely form a monomer using an enzyme having MHET
      hydrolytic activity composed of the polyester decomposition enzyme
      and an amino acid sequence represented by sequence nos. 10 or 12
      of the sequence listing.<br>
      <br>
      <hr width="100%" size="2"><br>
      <a href="https://www.ncbi.nlm.nih.gov/pubmed/26291558"
        "><b>https://www.ncbi.nlm.nih.gov/pubmed/26291558</b></a><b><br>
      </b>
      <div align="center">
        <div align="left"><b>Biochim Biophys Acta. 2015
            Nov;1850(11):2308-17. <br>
            doi: 10.1016/j.bbagen.2015.08.009. <br>
            <br>
          </b></div>
        <b>Structural and functional studies of a <i>Fusarium oxysporum</i>
          cutinase with polyethylene terephthalate modification
          potential.</b><br>
        <b>Dimarogona M.,<i>&nbsp; et al.</i></b><br>
      </div>
      <b><br>
      </b><b>Abstract</b><b><br>
      </b><b>BACKGROUND:</b><br>
      Cutinases are serine hydrolases that degrade cutin, a polyester of
      fatty acids that is the main component of plant cuticle. These
      biocatalysts have recently attracted increased biotechnological
      interest due to their potential to modify and degrade polyethylene
      terephthalate (PET), as well as other synthetic polymers.<br>
      <br>
      <b>METHODS:</b><br>
      A cutinase from the mesophilic fungus Fusarium oxysporum, named
      FoCut5a, was expressed either in the cytoplasm or periplasm of
      Escherichia coli BL21. Its X-ray structure was determined to 1.9Å
      resolution using molecular replacement. The activity of the
      recombinant enzyme was tested on a variety of synthetic esters and
      polyester analogues.<br>
      <br>
      <b>RESULTS:</b><br>
      The highest production of recombinant FoCut5a was achieved using
      periplasmic expression at 16°C. Its crystal structure is highly
      similar to previously determined Fusarium solani cutinase
      structure. However, a more detailed comparison of the surface
      properties and amino acid interactions revealed differences with
      potential impact on the biochemical properties of the two enzymes.
      FoCut5a showed maximum activity at 40°C and pH 8.0, while it was
      active on three p-nitrophenyl synthetic esters of aliphatic acids
      (C(2), C(4), C(12)), with the highest catalytic efficiency for the
      hydrolysis of the butyl ester. The recombinant cutinase was also
      found capable of hydrolyzing PET model substrates and synthetic
      polymers.<br>
      <br>
      <b>CONCLUSIONS:</b><br>
      The present work is the first reported expression and crystal
      structure determination of a functional cutinase from the
      mesophilic fungus F. oxysporum with potential application in
      surface modification of PET synthetic polymers.<br>
      <br>
      <b>GENERAL SIGNIFICANCE:</b><br>
      FoCut5a could be used as a biocatalyst in industrial applications
      for the environmentally-friendly treatment of synthetic polymers.<br>
      <br>
      <hr width="100%" size="2"><br>
      <b><a href="https://en.wikipedia.org/wiki/Ideonella_sakaiensis"
          ">https://en.wikipedia.org/wiki/Ideonella_sakaiensis</a><br>
        <br>
      </b>
      <div align="center"><i><b>Ideonella sakaiensis<br>
          </b></i></div>
      <b><br>
      </b>Ideonella sakaiensis is a bacterium from the genus Ideonella
      and family Comamonadaceae capable of breaking down PET plastic
      which was isolated from outside a plastic bottle recycling
      facility.[2]<br>
      <b><br>
        Discovery<br>
      </b>Ideonella sakaiensis was identified in 2016 by a team of
      researchers from Kyoto Institute of Technology and Keio University
      after collecting samples of PET debris in search for bacteria that
      relied on the plastic for carbon growth. The bacterium first uses
      PETase, an enzyme that works with water, to break down the PET
      plastic. It then breaks it down further using MHETase,[2] another
      enzyme that further reacts with water to break down the plastics
      into terephthalic acid and ethylene glycol.[3][2]<b><br>
        <br>
        Impact<br>
      </b>The discovery of Ideonella sakaiensis has potential importance
      for the recycling process of PET plastics. Prior to its discovery,
      the only known consumers of PET were a small number of fungi
      including Pestalotiopsis microspora, and knowledge of the new
      species has spurred discussion about biodegradation as a method of
      recycling.[4] The bacterium can currently break down a thin film
      of PET in a little over six weeks, so it is thought that any
      prospective applications in mass recycling programs will have to
      be preceded by enhancement of its abilities through genetic
      modification.[5]<br>
      <b><br>
        Characterization<br>
      </b>I. sakaiensis is Gram-negative, aerobic, and rod-shaped. It
      does not form spores. The individual cells of the organism are
      motile and have a single flagellum. I. sakaiensis tests positive
      for oxidase and catalase. The bacterium grows at a pH range of 5.5
      to 9.0 (optimally 7 to 7.5) and a temperature of 15-42 °C
      (optimally at 30-37 °C). Through phylogenetic analysis, the
      species was shown to be affiliated to the genus Ideonella, and
      also related to Ideonella dechloratans and Ideonella azotifigens,
      justifying its scientific classification.[6]<br>
      <br>
      <b>Morphology</b><b><br>
      </b><b> </b>Colonies of I. sakaiensis are colorless, smooth, and
      circular.[6]<b><br>
        <br>
      </b>
      <hr width="100%" size="2"><b><br>
        <a href="https://en.wikipedia.org/wiki/PETase" ">https://en.wikipedia.org/wiki/PETase</a><br>
        <br>
      </b>
      <div align="center"><b>PETase</b><br>
      </div>
      <b>&nbsp;<br>
      </b>
      <div align="center"><b><img src="PETase.png" alt="PETase"
            width="220" height="201"></b><br>
      </div>
      <b>Identifiers<br>
      </b>EC number &nbsp;&nbsp;&nbsp; 3.1.1.101<br>
      Alt. names &nbsp;&nbsp;&nbsp; PET hydrolase, poly(ethylene
      terephthalate) hydrolase<br>
      Databases<br>
      IntEnz &nbsp;&nbsp;&nbsp; IntEnz view<br>
      BRENDA &nbsp;&nbsp;&nbsp; BRENDA entry<br>
      ExPASy &nbsp;&nbsp;&nbsp; NiceZyme view<br>
      KEGG &nbsp;&nbsp;&nbsp; KEGG entry<br>
      MetaCyc &nbsp;&nbsp;&nbsp; metabolic pathway<br>
      PRIAM &nbsp;&nbsp;&nbsp; profile<br>
      PDB structures &nbsp;&nbsp;&nbsp; RCSB PDB PDBe PDBsum<b><br>
        &nbsp;<br>
      </b>PETase is an enzyme discovered in 2016 from a Japanese rubbish
      dump from Ideonella sakaiensis bacteria.[1][2] PETase breaks down
      PET-plastic to monomeric mono-2-hydroxyethyl terephthalate (MHET)
      molecules. MHET is broken down in these bacteria to hydroxyethyl
      terephthalate with the help of MHETase enzyme. Hydroxyethyl
      terephthalate breaks down in water to terephthalic acid and
      ethylene glycol, which are environmentally harmless as they are
      broken down further to produce carbon dioxide and water.[1]<br>
      <b><br>
      </b>The reaction catalyzed by PETase is (n is the number of
      monomers in the PET polymer):[3]...<br>
      <br>
      <b>Genetic engineering<br>
      </b>In 2018, John McGeehan co-led an international team (the
      University of Portsmouth and the U.S. Department of Energy's
      National Renewable Energy Laboratory) that managed to genetically
      alter PETase making it 20% more efficient, and raising the
      possibility of further efficiency gains.[4][5][6] Their aim is to
      use PETase to recycle the coloured polyethylene terephthalate
      (PET) plastic used in soft drink bottles and turn it back into
      easily reusable clear plastic.[5]<br>
      <br>
      Professor John McGeehan, Professor of Structural Biology at the
      University of Plymouth and one of the scientists leading the
      study, made the following remarks on the enzyme improvement:<br>
      <br>
      "Although the improvement is modest, this unanticipated discovery
      suggests that there is room to further improve these enzymes,
      moving us closer to a recycling solution for the ever-growing
      mountain of discarded plastics."[7]<b><br>
        <br>
        References<br>
      </b>Yoshida, Shosuke; Hiraga, Kazumi; Takehana, Toshihiko;
      Taniguchi, Ikuo; Yamaji, Hironao; Maeda, Yasuhito; Toyohara,
      Kiyotsuna; Miyamoto, Kenji; Kimura, Yoshiharu (2016-03-11). "A
      bacterium that degrades and assimilates poly(ethylene
      terephthalate)". Science. 351 (6278): 1196–1199.
      doi:10.1126/science.aad6359. ISSN 0036-8075. PMID 26965627.<br>
      <br>
      Tanasupawat, Somboon; Takehana, Toshihiko; Yoshida, Shosuke;
      Hiraga, Kazumi; Oda, Kohei (August 2016). "Ideonella sakaiensis
      sp. nov., isolated from a microbial consortium that degrades
      poly(ethylene terephthalate)". International Journal of Systematic
      and Evolutionary Microbiology. 66 (8): 2813–2818.
      doi:10.1099/ijsem.0.001058. ISSN 1466-5034. PMID 27045688.<br>
      <br>
      "BRENDA - Information on EC 3.1.1.101 - poly(ethylene
      terephthalate) hydrolase". www.brenda-enzymes.org.&nbsp; <br>
      <br>
      Austin, Harry P.; Allen, Mark D.; Donohoe, Bryon S.; Rorrer,
      Nicholas A.; Kearns, Fiona L.; Silveira, Rodrigo L.; Pollard,
      Benjamin C.; Dominick, Graham; Duman, Ramona (2018-04-17).
      "Characterization and engineering of a plastic-degrading aromatic
      polyesterase". Proceedings of the National Academy of Sciences.
      doi:10.1073/pnas.1718804115. ISSN 0027-8424. PMID 29666242.<br>
      <br>
      Carrington, Damian (16 April 2018). "Scientists accidentally
      create mutant enzyme that eats plastic bottles". the Guardian. <br>
      <br>
      Editorial, Reuters. "Plastic-eating enzyme holds promise in
      fighting pollution - scientists". reuters.com.&nbsp; <br>
      "Scientists accidentally discovered a mutant enzyme that could
      help the world eliminate plastic waste".<b><br>
        <br>
      </b>
      <hr width="100%" size="2"><br>
      <a
href="http://www.microbiologyresearch.org/docserver/fulltext/ijsem/66/8/2813_ijsem001058.pdf?expires=1524372986&amp;id=id&amp;accname=guest&amp;checksum=BBF1EEA2EA1EC7FB5C19D60FF74DAF92"
        "><b>http://www.microbiologyresearch.org/docserver/fulltext/ijsem/66/8/2813_ijsem001058.pdf?expires=1524372986&amp;id=id&amp;accname=guest&amp;checksum=BBF1EEA2EA1EC7FB5C19D60FF74DAF92</b></a><b><br>
      </b><b><br>
      </b>
      <div align="center"><b>Ideonella sakaiensis sp. nov., isolated
          from a microbial consortium that degrades poly(ethylene
          terephthalate)</b><br>
        <b>Somboon Tanasupawat,<i> et al<br>
          </i>[ <a href="intjsysevmicrobiol.pdf">PDF</a> ]<i><br>
          </i></b> </div>
      &nbsp;<br>
      <hr width="100%" size="2"><br>
      <b>Related:</b><b><br>
      </b>
      <div align="center"><b>US7341857</b><br>
        <b>Novel, esterase, fungus capable of producing the same and
          method for producing the same</b><br>
      </div>
      <b><br>
      </b><b>Inventor(s): SHINOHARA MAKOTO, et al.</b><b><br>
      </b><b><br>
      </b><b>Abstract </b><br>
      The present invention provides a novel esterase derived from
      Ideonella sp. 0-0013 strain (FERM BP-08660) having the following
      properties: (1) function, substrate specificity: hydrolyzes methyl
      3-hydroxypalmitate to generate 3-hydroxypalmitic acid and
      methanol; (2) optimal temperature for functioning: 37 DEG C.; (3)
      optimal pH and stable pH range: pH 7 or more to pH 10 or less; (4)
      temperature stability: 97% of the enzyme is stable at 43 DEG C.;
      (5) inhibition, activation, and stabilization: activated by sodium
      ion and potassium ion, and inhibited by strontium ion, iron ion
      (divalent), and methyl palmitate; (6) molecular weight: about
      46,500 Da (by SDS-PAGE), about 41,000 Da (by a gel filtration
      method); and (7) isoelectric point: pI 4 (by polyacrylamide gel
      isoelectric focusing method); a microorganism producing the
      enzyme; and a method of producing the enzyme.<br>
      <br>
      <hr width="100%" size="2"><br>
      <a
        href="https://www.sciencedaily.com/releases/2018/01/180131133404.htm"
        "><b>https://www.sciencedaily.com/releases/2018/01/180131133404.htm</b></a><b><br>
      </b><b><br>
      </b>
      <div align="center"><b>Structural insight into molecular mechanism
          of PET degradation</b><br>
      </div>
      <b><br>
      </b>
      <div align="center"><i>KAIST team newly suggests a molecular
          mechanism showing superior degradability of PET</i><br>
      </div>
      <br>
      A KAIST metabolic engineering research team has newly suggested a
      molecular mechanism showing superior degradability of poly
      ethylene terephthalate (PET).<br>
      <br>
      This is the first report to simultaneously determine the 3D
      crystal structure of Ideonella sakaiensis PETase and develop the
      new variant with enhanced PET degradation.<br>
      <br>
      Recently, diverse research projects are working to address the
      non-degradability of materials. A poly ethylene terephthalate
      (PET)-degrading bacterium called Ideonella sakaiensis was recently
      identified for the possible degradation and recycling of PET by
      Japanese team in Science journal (Yoshida et al., 2016). However,
      the detailed molecular mechanism of PET degradation has not been
      yet identified.<br>
      <br>
      The team under Distinguished Professor Sang Yup Lee of the
      Department of Chemical and Biomolecular Engineering and the team
      under Professor Kyung-Jin Kim of the Department of Biotechnology
      at Kyungpook National University conducted this research. The
      findings were published in Nature Communications on January 26.<br>
      <br>
      This research predicts a special molecular mechanism based on the
      docking simulation between PETase and a PET alternative mimic
      substrate. Furthermore, they succeeded in constructing the variant
      for IsPETase with enhanced PET-degrading activity using
      structural-based protein engineering.<br>
      <br>
      It is expected that the new approaches taken in this research can
      be background for further study of other enzymes capable of
      degrading not only PET but other plastics as well.<br>
      <br>
      PET is very important source in our daily lives. However, PET
      after use causes tremendous contamination issues to our
      environment due to its non-biodegradability, which has been a
      major advantage of PET. Conventionally, PET is disposed of in
      landfills, using incineration, and sometimes recycling using
      chemical methods, which induces additional environmental
      pollution. Therefore, a new development for highly-efficient PET
      degrading enzymes is essential to degrade PET using bio-based
      eco-friendly methods.<br>
      <br>
      Recently, a new bacterial species, Ideonella sakaiensis, which can
      use PET as a carbon source, was isolated. The PETase of I.
      sakaiensis (IsPETase) can degrade PET with relatively higher
      success than other PET-degrading enzymes. However, the detailed
      enzyme mechanism has not been elucidated, hindering further
      studies.<br>
      <br>
      The research teams investigated how the substrate binds to the
      enzyme and which differences in enzyme structure result in
      significantly higher PET degrading activity compared with other
      cutinases and esterases, which make IsPETase highly attractive for
      industrial applications toward PET waste recycling.<br>
      <br>
      Based on the 3D structure and related biochemical studies, they
      successfully predicted the reasons for extraordinary PET degrading
      activity of IsPETase and suggested other enzymes that can degrade
      PET with a newly-classified phylogenetic tree. The team proposed
      that 4 MHET moieties are the most properly matched substrates due
      to a cleft on structure even with the 10-20-mers for PET. This is
      meaningful in that it is the first docking simulation between
      PETase and PET, not its monomer.<br>
      <br>
      Furthermore, they succeeded in developing a new variant with much
      higher PET-degrading activity using a crystal structure of this
      variant to show that the changed structure is better to
      accommodate PET substrates than wild type PETase, which will lead
      to developing further superior enzymes and constructing platforms
      for microbial plastic recycling.<br>
      <br>
      Professor Lee said, "Environmental pollution from plastics remains
      one of the greatest challenges worldwide with the increasing
      consumption of plastics. We successfully constructed a new
      superior PET-degrading variant with the determination of a crystal
      structure of PETase and its degrading molecular mechanism. This
      novel technology will help further studies to engineer more
      superior enzymes with high efficiency in degrading. This will be
      the subject of our team's ongoing research projects to address the
      global environmental pollution problem for next generation."<br>
      <br>
      <hr width="100%" size="2"><br>
      <a
href="https://phys.org/news/2018-02-insight-molecular-mechanism-pet-degradation.html"
        "><b>https://phys.org/news/2018-02-insight-molecular-mechanism-pet-degradation.html</b></a><br>
      <a href="https://www.nature.com/articles/s41467-018-02881-1"
        "><b>https://www.nature.com/articles/s41467-018-02881-1</b></a><b><br>
      </b><b>doi:10.1038/s41467-018-02881-1</b><b><br>
      </b><b>Nature Communicationsvolume 9, Article number: 382 (2018) </b><b><br>
      </b><b><br>
      </b>
      <div align="center"><b>Structural insight into molecular mechanism
          of poly(ethylene terephthalate) degradation</b><br>
        <b>Seongjoon Joo,<i> et al.</i></b><br>
      </div>
      <b><br>
      </b><b>Abstract</b><br>
      Plastics, including poly(ethylene terephthalate) (PET), possess
      many desirable characteristics and thus are widely used in daily
      life. However, non-biodegradability, once thought to be an
      advantage offered by plastics, is causing major environmental
      problem. Recently, a PET-degrading bacterium, Ideonella
      sakaiensis, was identified and suggested for possible use in
      degradation and/or recycling of PET. However, the molecular
      mechanism of PET degradation is not known. Here we report the
      crystal structure of I. sakaiensis PETase (IsPETase) at 1.5 Å
      resolution. IsPETase has a Ser–His-Asp catalytic triad at its
      active site and contains an optimal substrate binding site to
      accommodate four monohydroxyethyl terephthalate (MHET) moieties of
      PET. Based on structural and site-directed mutagenesis
      experiments, the detailed process of PET degradation into MHET,
      terephthalic acid, and ethylene glycol is suggested. Moreover,
      other PETase candidates potentially having high PET-degrading
      activities are suggested based on phylogenetic tree analysis of 69
      PETase-like proteins.<br>
      <div align="center"><b><img src="structuralin.jpg" alt="PETase"
            width="492" height="295"><br>
          [ Click to enlarge ]<br>
        </b></div>
      <br>
      <hr width="100%" size="2"></blockquote>
    <div align="center"><img alt="" src="0logo.gif" width="124"
        height="82"><br>
      <br>
    </div>
    <div style="text-align: center;"><b>Your Support Maintains this
        Service -- </b><b><br>
      </b> <b><br>
      </b> <b>BUY</b><b><br>
      </b> <b><br>
      </b> <big><b>The</b><b> Rex Research Civilization Kit </b></big><b><br>
      </b> <b><br>
      </b> <b>... It's Your Best Bet &amp; Investment in Sustainable
        Humanity on Earth ... </b><b><br>
      </b><b> Ensure &amp; Enhance Your Survival &amp; Genome
        Transmission ...</b><b> </b><b><br>
      </b> <b>Everything @ rexresearch.com ( plus Bonus Files ) on a
        Flash Drive ! </b><b><br>
      </b> <b><br>
      </b> <b><a href="order.htm"
          ">ORDER PAGE</a></b><b><br>
      </b> <b><br>
      </b> </div>
    <b> </b>
    <hr style="width: 62%; height: 2px;">
    <script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
    <script type="text/javascript">
try{
var pageTracker = _gat._getTracker("UA-xxxxxx-x");
pageTracker._trackPageview();
} catch(err) {}
</script>
    <script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
    <script type="text/javascript">
try{
var pageTracker = _gat._getTracker("UA-xxxxxx-x");
pageTracker._trackPageview();
} catch(err) {}
</script>&gt;
  </body>
</html>
